<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Downloading Genesets • hypeR</title>
<!-- favicons --><link rel="icon" type="image/png" sizes="16x16" href="../../favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="../../favicon-32x32.png">
<link rel="apple-touch-icon" type="image/png" sizes="180x180" href="../../apple-touch-icon.png">
<link rel="apple-touch-icon" type="image/png" sizes="120x120" href="../../apple-touch-icon-120x120.png">
<link rel="apple-touch-icon" type="image/png" sizes="76x76" href="../../apple-touch-icon-76x76.png">
<link rel="apple-touch-icon" type="image/png" sizes="60x60" href="../../apple-touch-icon-60x60.png">
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script><!-- Bootstrap --><link href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.3.7/cosmo/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha256-U5ZEeKfGNOja007MMD3YBI0A3OSZOQbeG6z2f2Y0hu8=" crossorigin="anonymous"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.7.1/css/all.min.css" integrity="sha256-nAmazAk6vS34Xqo0BSrTb+abbtFlgsFK7NKSi6o7Y78=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.7.1/css/v4-shims.min.css" integrity="sha256-6qHlizsOWFskGlwVOKuns+D1nB6ssZrHQrNj1wGplHc=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.4/clipboard.min.js" integrity="sha256-FiZwavyI2V6+EXO1U+xzLG3IKldpiTFf3153ea9zikQ=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.9.4/headroom.min.js" integrity="sha256-DJFC1kqIhelURkuza0AvYal5RxMtpzLjFhsnVIeuk+U=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.9.4/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../../pkgdown.css" rel="stylesheet">
<script src="../../pkgdown.js"></script><meta property="og:title" content="Downloading Genesets">
<meta property="og:description" content="">
<meta name="twitter:card" content="summary">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body>
    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../../index.html">hypeR</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">1.3.1</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../../index.html">Home</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Docs
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../../articles/docs/installation.html">Installation</a>
    </li>
    <li>
      <a href="../../articles/docs/basics.html">Basic Usage</a>
    </li>
    <li>
      <a href="../../articles/docs/visualize.html">Visualize Results</a>
    </li>
    <li>
      <a href="../../articles/docs/data.html">Downloading Genesets</a>
    </li>
    <li>
      <a href="../../articles/docs/multihyp.html">Multiple Signatures</a>
    </li>
    <li>
      <a href="../../articles/docs/reporting.html">Generating Reports</a>
    </li>
    <li>
      <a href="../../articles/docs/shiny.html">Shiny Applications</a>
    </li>
  </ul>
</li>
<li>
  <a href="../../reference/index.html">Reference</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="../../news/index.html">Change Log</a>
</li>
<li>
  <a href="https://github.com/montilab/hypeR">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><script src="data_files/core-js-2.5.3/shim.min.js"></script><script src="data_files/react-16.8.6/react.min.js"></script><script src="data_files/react-16.8.6/react-dom.min.js"></script><script src="data_files/reactwidget-1.0.0/react-tools.js"></script><script src="data_files/htmlwidgets-1.5.1/htmlwidgets.js"></script><script src="data_files/reactable-binding-0.1.0/reactable.js"></script><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1>Downloading Genesets</h1>
            
      
      <small class="dont-index">Source: <a href="https://github.com/montilab/hypeR/blob/master/vignettes/docs/data.Rmd"><code>vignettes/docs/data.Rmd</code></a></small>
      <div class="hidden name"><code>data.Rmd</code></div>

    </div>

    
    
<div id="object-oriented-genesets" class="section level1">
<h1 class="hasAnchor">
<a href="#object-oriented-genesets" class="anchor"></a>Object Oriented Genesets</h1>
<p>Genesets are simply a named list of character vectors which can be directly passed to <code>hyper()</code>. Alternatively, one can pass a <code>gsets</code> object, which can retain the name and version of the genesets one uses. This versioning will be included when exporting results or generating reports, which will ensure your results are reproducible.</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1"></a>genesets &lt;-<span class="st"> </span><span class="kw"><a href="https://rdrr.io/r/base/list.html">list</a></span>(<span class="st">"GSET1"</span> =<span class="st"> </span><span class="kw"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="st">"GENE1"</span>, <span class="st">"GENE2"</span>, <span class="st">"GENE3"</span>),</span>
<span id="cb1-2"><a href="#cb1-2"></a>                 <span class="st">"GSET2"</span> =<span class="st"> </span><span class="kw"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="st">"GENE4"</span>, <span class="st">"GENE6"</span>),</span>
<span id="cb1-3"><a href="#cb1-3"></a>                 <span class="st">"GSET3"</span> =<span class="st"> </span><span class="kw"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="st">"GENE7"</span>, <span class="st">"GENE8"</span>, <span class="st">"GENE9"</span>))</span></code></pre></div>
<p>Creating a <code>gsets</code> object is easy…</p>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb2-1"><a href="#cb2-1"></a>genesets &lt;-<span class="st"> </span>gsets<span class="op">$</span><span class="kw">new</span>(genesets, <span class="dt">name=</span><span class="st">"Example Genesets"</span>, <span class="dt">version=</span><span class="st">"v1.0"</span>)</span>
<span id="cb2-2"><a href="#cb2-2"></a><span class="kw"><a href="https://rdrr.io/r/base/print.html">print</a></span>(genesets)</span></code></pre></div>
<pre><code>Example Genesets v1.0 
GSET1 (3)
GSET2 (2)
GSET3 (3)</code></pre>
<p>And can be passed directly to <code>hyper()</code>…</p>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb4-1"><a href="#cb4-1"></a><span class="kw"><a href="../../reference/hypeR.html">hypeR</a></span>(signature, genesets)</span></code></pre></div>
<p>To aid in workflow efficiency, <strong>hypeR</strong> enables users to download genesets, wrapped as <code>gsets</code> objects, from multiple data sources.</p>
</div>
<div id="downloading-msigdb-genesets" class="section level1">
<h1 class="hasAnchor">
<a href="#downloading-msigdb-genesets" class="anchor"></a>Downloading <strong>msigdb</strong> Genesets</h1>
<p>Most researchers will find the genesets hosted by <a href="https://software.broadinstitute.org/gsea/msigdb/collections.jsp">msigdb</a> are adequate to perform geneset enrichment analysis. There are various types of genesets available across multiple species.</p>
<div class="sourceCode" id="cb5"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb5-1"><a href="#cb5-1"></a><span class="kw"><a href="../../reference/msigdb_info.html">msigdb_info</a></span>()</span></code></pre></div>
<pre><code>|------------------------------------------------------------------|
| Via: R package msigdbr                                    v7.0.1 |
|------------------------------------------------------------------|
| Available Species                                                |
|------------------------------------------------------------------|
| Homo sapiens                                                     |
| Mus musculus                                                     |
| Drosophila melanogaster                                          |
| Gallus gallus                                                    |
| Saccharomyces cerevisiae                                         |
| Bos taurus                                                       |
| Caenorhabditis elegans                                           |
| Canis lupus familiaris                                           |
| Danio rerio                                                      |
| Rattus norvegicus                                                |
| Sus scrofa                                                       |
|------------------------------------------------------------------|
| Available Genesets                                               |
|------------------------------------------------------------------|
| Category Subcategory | Description                               |
|------------------------------------------------------------------|
| C1                   | Positional                                |
| C2 CGP               | Chemical and Genetic Perturbations        |
| C2 CP                | Canonical Pathways                        |
| C2 CP:BIOCARTA       | Canonical BIOCARTA                        |
| C2 CP:KEGG           | Canonical KEGG                            |
| C2 CP:PID            | Canonical PID                             |
| C2 CP:REACTOME       | Canonical REACTOME                        |
| C3 MIR               | Motif miRNA Targets                       |
| C3 TFT               | Motif Transcription Factor Targets        |
| C4 CGN               | Cancer Gene Neighborhoods                 |
| C4 CM                | Cancer Modules                            |
| C5 BP                | GO Biological Process                     |
| C5 CC                | GO Cellular Component                     |
| C5 MF                | GO Molecular Function                     |
| C6                   | Oncogenic Signatures                      |
| C7                   | Immunologic Signatures                    |
| H                    | Hallmark                                  |
|------------------------------------------------------------------|
| Source: http://software.broadinstitute.org/gsea/msigdb           |
|------------------------------------------------------------------|</code></pre>
<p>Here we download the Hallmarks genesets…</p>
<div class="sourceCode" id="cb7"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb7-1"><a href="#cb7-1"></a>HALLMARK &lt;-<span class="st"> </span><span class="kw"><a href="../../reference/msigdb_gsets.html">msigdb_gsets</a></span>(<span class="dt">species=</span><span class="st">"Homo sapiens"</span>, <span class="dt">category=</span><span class="st">"H"</span>)</span>
<span id="cb7-2"><a href="#cb7-2"></a><span class="kw"><a href="https://rdrr.io/r/base/print.html">print</a></span>(HALLMARK)</span></code></pre></div>
<pre><code>H v7.0.1 
HALLMARK_ADIPOGENESIS (200)
HALLMARK_ALLOGRAFT_REJECTION (200)
HALLMARK_ANDROGEN_RESPONSE (100)
HALLMARK_ANGIOGENESIS (36)
HALLMARK_APICAL_JUNCTION (200)
HALLMARK_APICAL_SURFACE (44)</code></pre>
<p>This can be passed directly to <code><a href="../../reference/hypeR.html">hypeR()</a></code>…</p>
<div class="sourceCode" id="cb9"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb9-1"><a href="#cb9-1"></a><span class="kw"><a href="../../reference/hypeR.html">hypeR</a></span>(signature, <span class="dt">genesets=</span>HALLMARK)</span></code></pre></div>
<p>Other commonly used genesets include Biocarta, Kegg, and Reactome…</p>
<div class="sourceCode" id="cb10"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb10-1"><a href="#cb10-1"></a>BIOCARTA &lt;-<span class="st"> </span><span class="kw"><a href="../../reference/msigdb_gsets.html">msigdb_gsets</a></span>(<span class="dt">species=</span><span class="st">"Homo sapiens"</span>, <span class="dt">category=</span><span class="st">"C2"</span>, <span class="dt">subcategory=</span><span class="st">"CP:BIOCARTA"</span>)</span>
<span id="cb10-2"><a href="#cb10-2"></a>KEGG     &lt;-<span class="st"> </span><span class="kw"><a href="../../reference/msigdb_gsets.html">msigdb_gsets</a></span>(<span class="dt">species=</span><span class="st">"Homo sapiens"</span>, <span class="dt">category=</span><span class="st">"C2"</span>, <span class="dt">subcategory=</span><span class="st">"CP:KEGG"</span>)</span>
<span id="cb10-3"><a href="#cb10-3"></a>REACTOME &lt;-<span class="st"> </span><span class="kw"><a href="../../reference/msigdb_gsets.html">msigdb_gsets</a></span>(<span class="dt">species=</span><span class="st">"Homo sapiens"</span>, <span class="dt">category=</span><span class="st">"C2"</span>, <span class="dt">subcategory=</span><span class="st">"CP:REACTOME"</span>)</span></code></pre></div>
</div>
<div id="downloading-enrichr-genesets" class="section level1">
<h1 class="hasAnchor">
<a href="#downloading-enrichr-genesets" class="anchor"></a>Downloading <strong>enrichr</strong> Genesets</h1>
<p>If msigdb genesets are not sufficient, we have also provided another set of functions for downloading and loading other publicly available genesets. This is facilitated by interfacing with the publicly available <a href="https://amp.pharm.mssm.edu/Enrichr/#stats">libraries</a> hosted by <a href="https://amp.pharm.mssm.edu/Enrichr/">enrichr</a>.</p>
<div class="sourceCode" id="cb11"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb11-1"><a href="#cb11-1"></a>available &lt;-<span class="st"> </span><span class="kw"><a href="../../reference/enrichr_available.html">enrichr_available</a></span>()</span>
<span id="cb11-2"><a href="#cb11-2"></a><span class="kw">reactable</span>(available)</span></code></pre></div>
<div id="htmlwidget-6460c314dd95b72fbea9" class="reactable html-widget" style="width:auto;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-6460c314dd95b72fbea9">{"x":{"tag":{"name":"Reactable","attribs":{"data":{"Geneset":["Genome_Browser_PWMs","TRANSFAC_and_JASPAR_PWMs","Transcription_Factor_PPIs","ChEA_2013","Drug_Perturbations_from_GEO_2014","ENCODE_TF_ChIP-seq_2014","BioCarta_2013","Reactome_2013","WikiPathways_2013","Disease_Signatures_from_GEO_up_2014","KEGG_2013","TF-LOF_Expression_from_GEO","TargetScan_microRNA","PPI_Hub_Proteins","GO_Molecular_Function_2015","GeneSigDB","Chromosome_Location","Human_Gene_Atlas","Mouse_Gene_Atlas","GO_Cellular_Component_2015","GO_Biological_Process_2015","Human_Phenotype_Ontology","Epigenomics_Roadmap_HM_ChIP-seq","KEA_2013","NURSA_Human_Endogenous_Complexome","CORUM","SILAC_Phosphoproteomics","MGI_Mammalian_Phenotype_Level_3","MGI_Mammalian_Phenotype_Level_4","Old_CMAP_up","Old_CMAP_down","OMIM_Disease","OMIM_Expanded","VirusMINT","MSigDB_Computational","MSigDB_Oncogenic_Signatures","Disease_Signatures_from_GEO_down_2014","Virus_Perturbations_from_GEO_up","Virus_Perturbations_from_GEO_down","Cancer_Cell_Line_Encyclopedia","NCI-60_Cancer_Cell_Lines","Tissue_Protein_Expression_from_ProteomicsDB","Tissue_Protein_Expression_from_Human_Proteome_Map","HMDB_Metabolites","Pfam_InterPro_Domains","GO_Biological_Process_2013","GO_Cellular_Component_2013","GO_Molecular_Function_2013","Allen_Brain_Atlas_up","ENCODE_TF_ChIP-seq_2015","ENCODE_Histone_Modifications_2015","Phosphatase_Substrates_from_DEPOD","Allen_Brain_Atlas_down","ENCODE_Histone_Modifications_2013","Achilles_fitness_increase","Achilles_fitness_decrease","MGI_Mammalian_Phenotype_2013","BioCarta_2015","HumanCyc_2015","KEGG_2015","NCI-Nature_2015","Panther_2015","WikiPathways_2015","Reactome_2015","ESCAPE","HomoloGene","Disease_Perturbations_from_GEO_down","Disease_Perturbations_from_GEO_up","Drug_Perturbations_from_GEO_down","Genes_Associated_with_NIH_Grants","Drug_Perturbations_from_GEO_up","KEA_2015","Gene_Perturbations_from_GEO_up","Gene_Perturbations_from_GEO_down","ChEA_2015","dbGaP","LINCS_L1000_Chem_Pert_up","LINCS_L1000_Chem_Pert_down","GTEx_Tissue_Sample_Gene_Expression_Profiles_down","GTEx_Tissue_Sample_Gene_Expression_Profiles_up","Ligand_Perturbations_from_GEO_down","Aging_Perturbations_from_GEO_down","Aging_Perturbations_from_GEO_up","Ligand_Perturbations_from_GEO_up","MCF7_Perturbations_from_GEO_down","MCF7_Perturbations_from_GEO_up","Microbe_Perturbations_from_GEO_down","Microbe_Perturbations_from_GEO_up","LINCS_L1000_Ligand_Perturbations_down","LINCS_L1000_Ligand_Perturbations_up","LINCS_L1000_Kinase_Perturbations_down","LINCS_L1000_Kinase_Perturbations_up","Reactome_2016","KEGG_2016","WikiPathways_2016","ENCODE_and_ChEA_Consensus_TFs_from_ChIP-X","Kinase_Perturbations_from_GEO_down","Kinase_Perturbations_from_GEO_up","BioCarta_2016","HumanCyc_2016","NCI-Nature_2016","Panther_2016","DrugMatrix","ChEA_2016","huMAP","Jensen_TISSUES","RNA-Seq_Disease_Gene_and_Drug_Signatures_from_GEO","MGI_Mammalian_Phenotype_2017","Jensen_COMPARTMENTS","Jensen_DISEASES","BioPlex_2017","GO_Cellular_Component_2017","GO_Molecular_Function_2017","GO_Biological_Process_2017","GO_Cellular_Component_2017b","GO_Molecular_Function_2017b","GO_Biological_Process_2017b","ARCHS4_Tissues","ARCHS4_Cell-lines","ARCHS4_IDG_Coexp","ARCHS4_Kinases_Coexp","ARCHS4_TFs_Coexp","SysMyo_Muscle_Gene_Sets","miRTarBase_2017","TargetScan_microRNA_2017","Enrichr_Libraries_Most_Popular_Genes","Enrichr_Submissions_TF-Gene_Coocurrence","Data_Acquisition_Method_Most_Popular_Genes","DSigDB","GO_Biological_Process_2018","GO_Cellular_Component_2018","GO_Molecular_Function_2018","TF_Perturbations_Followed_by_Expression","Chromosome_Location_hg19","NIH_Funded_PIs_2017_Human_GeneRIF","NIH_Funded_PIs_2017_Human_AutoRIF","Rare_Diseases_AutoRIF_ARCHS4_Predictions","Rare_Diseases_GeneRIF_ARCHS4_Predictions","NIH_Funded_PIs_2017_AutoRIF_ARCHS4_Predictions","NIH_Funded_PIs_2017_GeneRIF_ARCHS4_Predictions","Rare_Diseases_GeneRIF_Gene_Lists","Rare_Diseases_AutoRIF_Gene_Lists","SubCell_BarCode","GWAS_Catalog_2019","WikiPathways_2019_Human","WikiPathways_2019_Mouse","TRRUST_Transcription_Factors_2019","KEGG_2019_Human","KEGG_2019_Mouse","InterPro_Domains_2019","Pfam_Domains_2019","DepMap_WG_CRISPR_Screens_Broad_CellLines_2019","DepMap_WG_CRISPR_Screens_Sanger_CellLines_2019","MGI_Mammalian_Phenotype_Level_4_2019","UK_Biobank_GWAS_v1","BioPlanet_2019","ClinVar_2019","PheWeb_2019","DisGeNET","HMS_LINCS_KinomeScan"],"Set Number":[615,326,290,353,701,498,249,78,199,142,200,269,222,385,1136,2139,386,84,96,641,5192,1779,383,474,1796,1658,84,71,476,6100,6100,90,187,85,858,189,142,323,323,967,93,207,30,3906,311,941,205,402,2192,816,412,59,2192,109,216,216,476,239,125,179,209,104,404,1389,315,12,839,839,906,32876,906,428,2460,2460,395,345,33132,33132,2918,2918,261,286,286,261,401,401,312,312,96,96,3644,3644,1530,293,437,104,285,285,237,152,209,112,7876,645,995,1842,1302,5231,2283,1811,3915,636,972,3166,816,3271,10125,108,125,352,498,1724,1135,3240,683,121,1722,12,4026,5103,446,1151,1958,36,5687,12558,3725,2244,12558,5684,2244,3725,104,1737,472,176,571,308,303,1071,608,558,325,5261,857,1510,182,1161,9828,148],"Coverage":[13362,27884,6002,47172,47107,21493,1295,3185,2854,15057,4128,34061,7504,16399,12753,23726,32740,13373,19270,13236,14264,3096,22288,4533,10231,2741,5655,10406,10493,11251,8695,1759,2178,851,10061,11250,15406,17711,17576,15797,12232,13572,6454,3723,7588,7682,7324,8469,13121,26382,29065,280,13877,15852,4320,4271,10496,1678,756,3800,2541,1918,5863,6768,25651,19129,23939,23561,23877,15886,24350,3102,31132,30832,48230,5613,9559,9448,16725,19249,15090,16129,15309,15103,15022,15676,15854,15015,3788,3357,12668,12638,8973,7010,5966,15562,17850,17660,1348,934,2541,2041,5209,49238,2243,19586,22440,8184,18329,15755,10271,10427,10601,13822,8002,10089,13247,21809,23601,20883,19612,25983,19500,14893,17598,5902,12486,1073,19513,14433,8655,11459,19741,27360,13072,13464,13787,13929,16964,17258,10352,10471,12419,19378,6201,4558,3264,7802,8551,12444,9000,7744,6204,13420,14148,9813,1397,9116,17464,394],"Genes Per Set":[275,1284,77,1370,509,3713,18,73,34,300,48,641,155,247,57,127,85,258,388,82,58,31,4368,37,158,5,342,715,200,100,100,25,89,15,106,166,300,300,300,176,343,301,301,47,35,78,172,122,305,1811,2123,9,304,912,129,128,201,21,12,48,39,39,51,47,807,1594,293,307,302,9,299,25,298,302,1429,36,73,63,1443,1443,282,292,308,318,290,310,279,321,159,153,300,300,64,87,51,887,300,300,19,13,39,42,300,1550,19,545,505,24,161,28,22,38,25,21,143,45,49,2316,2395,299,299,299,137,128,1208,109,299,100,117,36,61,39,270,802,26,45,200,200,200,200,58,76,491,37,45,54,22,92,98,23,20,363,387,32,122,49,13,22,63,73],"Source":["http://hgdownload.cse.ucsc.edu/goldenPath/hg18/database/","http://jaspar.genereg.net/html/DOWNLOAD/","","http://amp.pharm.mssm.edu/lib/cheadownload.jsp","http://www.ncbi.nlm.nih.gov/geo/","http://genome.ucsc.edu/ENCODE/downloads.html","https://cgap.nci.nih.gov/Pathways/BioCarta_Pathways","http://www.reactome.org/download/index.html","http://www.wikipathways.org/index.php/Download_Pathways","http://www.ncbi.nlm.nih.gov/geo/","http://www.kegg.jp/kegg/download/","http://www.ncbi.nlm.nih.gov/geo/","http://www.targetscan.org/cgi-bin/targetscan/data_download.cgi?db=vert_61","http://amp.pharm.mssm.edu/X2K","http://www.geneontology.org/GO.downloads.annotations.shtml","http://genesigdb.org/genesigdb/downloadall.jsp","http://software.broadinstitute.org/gsea/msigdb/index.jsp","http://biogps.org/downloads/","http://biogps.org/downloads/","http://www.geneontology.org/GO.downloads.annotations.shtml","http://www.geneontology.org/GO.downloads.annotations.shtml","http://www.human-phenotype-ontology.org/","http://www.roadmapepigenomics.org/","http://amp.pharm.mssm.edu/lib/keacommandline.jsp","https://www.nursa.org/nursa/index.jsf","http://mips.helmholtz-muenchen.de/genre/proj/corum/","http://amp.pharm.mssm.edu/lib/keacommandline.jsp","http://www.informatics.jax.org/","http://www.informatics.jax.org/","http://www.broadinstitute.org/cmap/","http://www.broadinstitute.org/cmap/","http://www.omim.org/downloads","http://www.omim.org/downloads","http://mint.bio.uniroma2.it/download.html","http://www.broadinstitute.org/gsea/msigdb/collections.jsp","http://www.broadinstitute.org/gsea/msigdb/collections.jsp","http://www.ncbi.nlm.nih.gov/geo/","http://www.ncbi.nlm.nih.gov/geo/","http://www.ncbi.nlm.nih.gov/geo/","https://portals.broadinstitute.org/ccle/home\n","http://biogps.org/downloads/","https://www.proteomicsdb.org/","http://www.humanproteomemap.org/index.php","http://www.hmdb.ca/downloads","ftp://ftp.ebi.ac.uk/pub/databases/interpro/","http://www.geneontology.org/GO.downloads.annotations.shtml","http://www.geneontology.org/GO.downloads.annotations.shtml","http://www.geneontology.org/GO.downloads.annotations.shtml","http://www.brain-map.org/","http://genome.ucsc.edu/ENCODE/downloads.html","http://genome.ucsc.edu/ENCODE/downloads.html","http://www.koehn.embl.de/depod/","http://www.brain-map.org/","http://genome.ucsc.edu/ENCODE/downloads.html","http://www.broadinstitute.org/achilles","http://www.broadinstitute.org/achilles","http://www.informatics.jax.org/","https://cgap.nci.nih.gov/Pathways/BioCarta_Pathways","http://humancyc.org/","http://www.kegg.jp/kegg/download/","http://pid.nci.nih.gov/","http://www.pantherdb.org/","http://www.wikipathways.org/index.php/Download_Pathways","http://www.reactome.org/download/index.html","http://www.maayanlab.net/ESCAPE/","http://www.ncbi.nlm.nih.gov/homologene","http://www.ncbi.nlm.nih.gov/geo/","http://www.ncbi.nlm.nih.gov/geo/","http://www.ncbi.nlm.nih.gov/geo/","https://grants.nih.gov/grants/oer.htm\n","http://www.ncbi.nlm.nih.gov/geo/","http://amp.pharm.mssm.edu/Enrichr","http://www.ncbi.nlm.nih.gov/geo/","http://www.ncbi.nlm.nih.gov/geo/","http://amp.pharm.mssm.edu/Enrichr","http://www.ncbi.nlm.nih.gov/gap","https://clue.io/","https://clue.io/","http://www.gtexportal.org/","http://www.gtexportal.org/","http://www.ncbi.nlm.nih.gov/geo/","http://www.ncbi.nlm.nih.gov/geo/","http://www.ncbi.nlm.nih.gov/geo/","http://www.ncbi.nlm.nih.gov/geo/","http://www.ncbi.nlm.nih.gov/geo/","http://www.ncbi.nlm.nih.gov/geo/","http://www.ncbi.nlm.nih.gov/geo/","http://www.ncbi.nlm.nih.gov/geo/","https://clue.io/","https://clue.io/","https://clue.io/","https://clue.io/","http://www.reactome.org/download/index.html","http://www.kegg.jp/kegg/download/","http://www.wikipathways.org/index.php/Download_Pathways","","http://www.ncbi.nlm.nih.gov/geo/","http://www.ncbi.nlm.nih.gov/geo/","http://cgap.nci.nih.gov/Pathways/BioCarta_Pathways","http://humancyc.org/","http://pid.nci.nih.gov/","http://www.pantherdb.org/pathway/","https://ntp.niehs.nih.gov/drugmatrix/","http://amp.pharm.mssm.edu/Enrichr","http://proteincomplexes.org/","http://tissues.jensenlab.org/","http://www.ncbi.nlm.nih.gov/geo/","http://www.informatics.jax.org/","http://compartments.jensenlab.org/","http://diseases.jensenlab.org/","http://bioplex.hms.harvard.edu/","http://www.geneontology.org/","http://www.geneontology.org/","http://www.geneontology.org/","http://www.geneontology.org/","http://www.geneontology.org/","http://www.geneontology.org/","http://amp.pharm.mssm.edu/archs4","http://amp.pharm.mssm.edu/archs4","http://amp.pharm.mssm.edu/archs4","http://amp.pharm.mssm.edu/archs4","http://amp.pharm.mssm.edu/archs4","http://sys-myo.rhcloud.com/","http://mirtarbase.mbc.nctu.edu.tw/","http://www.targetscan.org/","http://amp.pharm.mssm.edu/Enrichr","http://amp.pharm.mssm.edu/Enrichr","http://amp.pharm.mssm.edu/Enrichr","http://tanlab.ucdenver.edu/DSigDB/DSigDBv1.0/","http://www.geneontology.org/","http://www.geneontology.org/","http://www.geneontology.org/","http://www.ncbi.nlm.nih.gov/geo/","http://hgdownload.cse.ucsc.edu/downloads.html","https://www.ncbi.nlm.nih.gov/pubmed/","https://www.ncbi.nlm.nih.gov/pubmed/","https://amp.pharm.mssm.edu/geneshot/","https://www.ncbi.nlm.nih.gov/gene/about-generif","https://www.ncbi.nlm.nih.gov/pubmed/","https://www.ncbi.nlm.nih.gov/pubmed/","https://www.ncbi.nlm.nih.gov/gene/about-generif","https://amp.pharm.mssm.edu/geneshot/","http://www.subcellbarcode.org/","https://www.ebi.ac.uk/gwas","https://www.wikipathways.org/","https://www.wikipathways.org/","https://www.grnpedia.org/trrust/","https://www.kegg.jp/","https://www.kegg.jp/","https://www.ebi.ac.uk/interpro/","https://pfam.xfam.org/","https://depmap.org/","https://depmap.org/","http://www.informatics.jax.org/","https://www.ukbiobank.ac.uk/tag/gwas/","https://tripod.nih.gov/bioplanet/","https://www.ncbi.nlm.nih.gov/clinvar/","http://pheweb.sph.umich.edu/","https://www.disgenet.org","http://lincs.hms.harvard.edu/kinomescan/"]},"columns":[{"accessor":"Geneset","name":"Geneset","type":"factor"},{"accessor":"Set Number","name":"Set Number","type":"numeric"},{"accessor":"Coverage","name":"Coverage","type":"numeric"},{"accessor":"Genes Per Set","name":"Genes Per Set","type":"numeric"},{"accessor":"Source","name":"Source","type":"factor"}],"defaultPageSize":10,"paginationType":"numbers","showPageInfo":true,"minRows":1,"dataKey":"d7fdb5929c50dcc96cf46fd85f4f42b9"},"children":[]},"class":"reactR_markup"},"evals":[],"jsHooks":[]}</script><div class="sourceCode" id="cb12"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb12-1"><a href="#cb12-1"></a>ATLAS &lt;-<span class="st"> </span><span class="kw"><a href="../../reference/enrichr_gsets.html">enrichr_gsets</a></span>(<span class="st">"Human_Gene_Atlas"</span>)</span>
<span id="cb12-2"><a href="#cb12-2"></a><span class="kw"><a href="https://rdrr.io/r/base/print.html">print</a></span>(ATLAS)</span></code></pre></div>
<pre><code>Human_Gene_Atlas Downloaded: 2020-01-30 
Thyroid (388)
CD33+ Myeloid (679)
Testis (384)
Salivarygland (57)
CD14+ Monocytes (383)
TestisLeydigCell (107)</code></pre>
<p>Note: These libraries do not have a systematic versioning scheme, however the date downloaded will be recorded.</p>
</div>
<div id="relational-genesets" class="section level1">
<h1 class="hasAnchor">
<a href="#relational-genesets" class="anchor"></a>Relational Genesets</h1>
<p>When dealing with hundreds of genesets, it’s often useful to understand the relationships between them. This allows researchers to summarize many enriched pathways as more general biological processes. To do this, we rely on curated relationships defined between them. For example, <a href="https://reactome.org/">Reactome</a> conveniently defines their genesets in a <a href="https://reactome.org/PathwayBrowser/">hiearchy of pathways</a>. This data can be formatted into a relational genesets object called <code>rgsets</code>.</p>
<p>We currently curate some relational genesets for use with <strong>hypeR</strong> and plan to add more continuously.</p>
<div class="sourceCode" id="cb14"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb14-1"><a href="#cb14-1"></a><span class="kw"><a href="../../reference/hyperdb_info.html">hyperdb_info</a></span>()</span></code></pre></div>
<pre><code>REACTOME
Versions: 70.0
KEGG
Versions: 92.0</code></pre>
<p>Downloading relational genesets is easy…</p>
<div class="sourceCode" id="cb16"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb16-1"><a href="#cb16-1"></a>genesets &lt;-<span class="st"> </span><span class="kw"><a href="../../reference/hyperdb_rgsets.html">hyperdb_rgsets</a></span>(<span class="st">"REACTOME"</span>, <span class="st">"70.0"</span>)</span></code></pre></div>
<p>And can be passed directly to <code>hyper()</code>…</p>
<div class="sourceCode" id="cb17"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb17-1"><a href="#cb17-1"></a><span class="kw"><a href="../../reference/hypeR.html">hypeR</a></span>(signature, genesets)</span></code></pre></div>
<div id="creating-relational-genesets" class="section level3">
<h3 class="hasAnchor">
<a href="#creating-relational-genesets" class="anchor"></a>Creating Relational Genesets</h3>
<p>We try to provide relational genesets for popular databases that include hierarchical information. For users who want to create their own, we provide this example.</p>
<p>Raw data for gsets, nodes, and edges can be directly downloaded.</p>
<div class="sourceCode" id="cb18"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb18-1"><a href="#cb18-1"></a>genesets.url &lt;-<span class="st"> "https://reactome.org/download/current/ReactomePathways.gmt.zip"</span></span>
<span id="cb18-2"><a href="#cb18-2"></a>nodes.url &lt;-<span class="st"> "https://reactome.org/download/current/ReactomePathways.txt"</span></span>
<span id="cb18-3"><a href="#cb18-3"></a>edges.url &lt;-<span class="st"> "https://reactome.org/download/current/ReactomePathwaysRelation.txt"</span></span></code></pre></div>
<div id="loading-data" class="section level4">
<h4 class="hasAnchor">
<a href="#loading-data" class="anchor"></a>Loading Data</h4>
<div class="sourceCode" id="cb19"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb19-1"><a href="#cb19-1"></a><span class="co"># Genesets</span></span>
<span id="cb19-2"><a href="#cb19-2"></a>genesets.tmp &lt;-<span class="st"> </span><span class="kw"><a href="https://rdrr.io/r/base/tempfile.html">tempfile</a></span>(<span class="dt">fileext=</span><span class="st">".gmt.zip"</span>)</span>
<span id="cb19-3"><a href="#cb19-3"></a><span class="kw"><a href="https://rdrr.io/r/utils/download.file.html">download.file</a></span>(genesets.url, <span class="dt">destfile =</span> genesets.tmp, <span class="dt">mode =</span> <span class="st">"wb"</span>)</span>
<span id="cb19-4"><a href="#cb19-4"></a>genesets.raw &lt;-<span class="st"> </span>genesets.tmp <span class="op">%&gt;%</span></span>
<span id="cb19-5"><a href="#cb19-5"></a><span class="st">                </span><span class="kw"><a href="https://rdrr.io/r/utils/unzip.html">unzip</a></span>() <span class="op">%&gt;%</span></span>
<span id="cb19-6"><a href="#cb19-6"></a><span class="st">                </span><span class="kw">read.gmt</span>() <span class="op">%&gt;%</span></span>
<span id="cb19-7"><a href="#cb19-7"></a><span class="st">                </span><span class="kw"><a href="https://rdrr.io/r/base/lapply.html">lapply</a></span>(<span class="cf">function</span>(x) {</span>
<span id="cb19-8"><a href="#cb19-8"></a>                    <span class="kw"><a href="https://rdrr.io/r/base/chartr.html">toupper</a></span>(x[x <span class="op">!=</span><span class="st"> "Reactome Pathway"</span>])</span>
<span id="cb19-9"><a href="#cb19-9"></a>                })</span>
<span id="cb19-10"><a href="#cb19-10"></a><span class="co"># Nodes</span></span>
<span id="cb19-11"><a href="#cb19-11"></a>nodes.raw &lt;-<span class="st"> </span>nodes.url <span class="op">%&gt;%</span></span>
<span id="cb19-12"><a href="#cb19-12"></a><span class="st">             </span><span class="kw"><a href="https://rdrr.io/r/utils/read.table.html">read.delim</a></span>(<span class="dt">sep=</span><span class="st">"</span><span class="ch">\t</span><span class="st">"</span>, </span>
<span id="cb19-13"><a href="#cb19-13"></a>                        <span class="dt">header=</span><span class="ot">FALSE</span>, </span>
<span id="cb19-14"><a href="#cb19-14"></a>                        <span class="dt">fill=</span><span class="ot">TRUE</span>, </span>
<span id="cb19-15"><a href="#cb19-15"></a>                        <span class="dt">col.names=</span><span class="kw"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="st">"id"</span>, <span class="st">"label"</span>, <span class="st">"species"</span>), </span>
<span id="cb19-16"><a href="#cb19-16"></a>                        <span class="dt">stringsAsFactors=</span><span class="ot">FALSE</span>)</span>
<span id="cb19-17"><a href="#cb19-17"></a><span class="co"># Edges</span></span>
<span id="cb19-18"><a href="#cb19-18"></a>edges.raw &lt;-<span class="st"> </span>edges.url <span class="op">%&gt;%</span></span>
<span id="cb19-19"><a href="#cb19-19"></a><span class="st">             </span><span class="kw"><a href="https://rdrr.io/r/utils/read.table.html">read.delim</a></span>(<span class="dt">sep=</span><span class="st">"</span><span class="ch">\t</span><span class="st">"</span>, </span>
<span id="cb19-20"><a href="#cb19-20"></a>                        <span class="dt">header=</span><span class="ot">FALSE</span>, </span>
<span id="cb19-21"><a href="#cb19-21"></a>                        <span class="dt">fill=</span><span class="ot">TRUE</span>, </span>
<span id="cb19-22"><a href="#cb19-22"></a>                        <span class="dt">col.names=</span><span class="kw"><a href="https://rdrr.io/r/base/c.html">c</a></span>(<span class="st">"from"</span>, <span class="st">"to"</span>),</span>
<span id="cb19-23"><a href="#cb19-23"></a>                        <span class="dt">stringsAsFactors=</span><span class="ot">FALSE</span>)</span></code></pre></div>
</div>
<div id="organizing-a-hierarchy" class="section level4">
<h4 class="hasAnchor">
<a href="#organizing-a-hierarchy" class="anchor"></a>Organizing a Hierarchy</h4>
<div class="sourceCode" id="cb20"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb20-1"><a href="#cb20-1"></a><span class="co"># Species-specific nodes</span></span>
<span id="cb20-2"><a href="#cb20-2"></a>nodes &lt;-<span class="st"> </span>nodes.raw <span class="op">%&gt;%</span></span>
<span id="cb20-3"><a href="#cb20-3"></a><span class="st">         </span><span class="kw"><a href="https://rdrr.io/r/stats/filter.html">filter</a></span>( label <span class="op">%in%</span><span class="st"> </span><span class="kw"><a href="https://rdrr.io/r/base/names.html">names</a></span>(genesets.raw) ) <span class="op">%&gt;%</span></span>
<span id="cb20-4"><a href="#cb20-4"></a><span class="st">         </span><span class="kw"><a href="https://rdrr.io/r/stats/filter.html">filter</a></span>( species <span class="op">==</span><span class="st"> "Homo sapiens"</span> ) <span class="op">%&gt;%</span></span>
<span id="cb20-5"><a href="#cb20-5"></a><span class="st">         </span><span class="kw"><a href="https://rdrr.io/r/stats/filter.html">filter</a></span>(<span class="op">!</span><span class="st"> </span><span class="kw"><a href="https://rdrr.io/r/base/duplicated.html">duplicated</a></span>(id) ) <span class="op">%&gt;%</span></span>
<span id="cb20-6"><a href="#cb20-6"></a><span class="st">         </span><span class="kw">set_rownames</span>( .<span class="op">$</span>id ) <span class="op">%&gt;%</span></span>
<span id="cb20-7"><a href="#cb20-7"></a><span class="st">         </span>{ .[, <span class="st">"label"</span>, drop=<span class="ot">FALSE</span>] }</span>
<span id="cb20-8"><a href="#cb20-8"></a></span>
<span id="cb20-9"><a href="#cb20-9"></a><span class="co"># Species-specific edges</span></span>
<span id="cb20-10"><a href="#cb20-10"></a>edges &lt;-<span class="st"> </span>edges.raw <span class="op">%&gt;%</span></span>
<span id="cb20-11"><a href="#cb20-11"></a><span class="st">         </span><span class="kw"><a href="https://rdrr.io/r/stats/filter.html">filter</a></span>( from <span class="op">%in%</span><span class="st"> </span><span class="kw"><a href="https://rdrr.io/r/base/colnames.html">rownames</a></span>(nodes) ) <span class="op">%&gt;%</span></span>
<span id="cb20-12"><a href="#cb20-12"></a><span class="st">         </span><span class="kw"><a href="https://rdrr.io/r/stats/filter.html">filter</a></span>( to <span class="op">%in%</span><span class="st"> </span><span class="kw"><a href="https://rdrr.io/r/base/colnames.html">rownames</a></span>(nodes) )</span>
<span id="cb20-13"><a href="#cb20-13"></a></span>
<span id="cb20-14"><a href="#cb20-14"></a><span class="co"># Leaf genesets</span></span>
<span id="cb20-15"><a href="#cb20-15"></a>genesets &lt;-<span class="st"> </span>nodes <span class="op">%&gt;%</span></span>
<span id="cb20-16"><a href="#cb20-16"></a><span class="st">            </span><span class="kw"><a href="https://rdrr.io/r/base/colnames.html">rownames</a></span>() <span class="op">%&gt;%</span></span>
<span id="cb20-17"><a href="#cb20-17"></a><span class="st">            </span>.[<span class="op">!</span><span class="st"> </span>. <span class="op">%in%</span><span class="st"> </span>edges<span class="op">$</span>from] <span class="op">%&gt;%</span></span>
<span id="cb20-18"><a href="#cb20-18"></a><span class="st">            </span><span class="kw"><a href="https://rdrr.io/r/base/lapply.html">sapply</a></span>( <span class="cf">function</span>(x) nodes[x, <span class="st">"label"</span>] ) <span class="op">%&gt;%</span></span>
<span id="cb20-19"><a href="#cb20-19"></a><span class="st">            </span>genesets.raw[.]</span></code></pre></div>
<div id="nodes" class="section level5">
<h5 class="hasAnchor">
<a href="#nodes" class="anchor"></a>nodes</h5>
<p>A single-column data frame of labels where the rownames are unique identifiers. Leaf node labels should have an associated geneset, while internal nodes do not have to. The only genesets tested, will be those in the list of genesets.</p>
<div class="sourceCode" id="cb21"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb21-1"><a href="#cb21-1"></a><span class="kw"><a href="https://rdrr.io/r/utils/head.html">head</a></span>(nodes)</span></code></pre></div>
<pre><code>                                                                        label
R-HSA-164843                                           2-LTR circle formation
R-HSA-73843                        5-Phosphoribose 1-diphosphate biosynthesis
R-HSA-1971475 A tetrasaccharide linker sequence is required for GAG synthesis
R-HSA-5619084                                       ABC transporter disorders
R-HSA-1369062                           ABC transporters in lipid homeostasis
R-HSA-382556                           ABC-family proteins mediated transport</code></pre>
</div>
<div id="edges" class="section level5">
<h5 class="hasAnchor">
<a href="#edges" class="anchor"></a>edges</h5>
<p>A dataframe with two columns of identifiers, indicating directed edges between nodes in the hierarchy.</p>
<div class="sourceCode" id="cb23"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb23-1"><a href="#cb23-1"></a><span class="kw"><a href="https://rdrr.io/r/utils/head.html">head</a></span>(edges)</span></code></pre></div>
<pre><code>          from            to
1 R-HSA-109581  R-HSA-109606
2 R-HSA-109581  R-HSA-169911
3 R-HSA-109581 R-HSA-5357769
4 R-HSA-109581   R-HSA-75153
5 R-HSA-109582  R-HSA-140877
6 R-HSA-109582  R-HSA-202733</code></pre>
</div>
<div id="genesets" class="section level5">
<h5 class="hasAnchor">
<a href="#genesets" class="anchor"></a>genesets</h5>
<p>A list of character vectors, named by the geneset labels. Typically, genesets will be at the leaves of the hierarchy, while not required.</p>
<div class="sourceCode" id="cb25"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb25-1"><a href="#cb25-1"></a><span class="kw"><a href="https://rdrr.io/r/utils/head.html">head</a></span>(<span class="kw"><a href="https://rdrr.io/r/base/names.html">names</a></span>(genesets))</span></code></pre></div>
<pre><code>[1] "2-LTR circle formation"                                         
[2] "5-Phosphoribose 1-diphosphate biosynthesis"                     
[3] "A tetrasaccharide linker sequence is required for GAG synthesis"
[4] "ABC transporters in lipid homeostasis"                          
[5] "ABO blood group biosynthesis"                                   
[6] "ADP signalling through P2Y purinoceptor 1"                      </code></pre>
</div>
</div>
<div id="the-rgsets-object" class="section level4">
<h4 class="hasAnchor">
<a href="#the-rgsets-object" class="anchor"></a>The <code>rgsets</code> Object</h4>
<div class="sourceCode" id="cb27"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb27-1"><a href="#cb27-1"></a>genesets &lt;-<span class="st"> </span>rgsets<span class="op">$</span><span class="kw">new</span>(genesets, nodes, edges, <span class="dt">name=</span><span class="st">"REACTOME"</span>, <span class="dt">version=</span><span class="st">"v70.0"</span>)</span>
<span id="cb27-2"><a href="#cb27-2"></a><span class="kw"><a href="https://rdrr.io/r/base/print.html">print</a></span>(genesets)</span></code></pre></div>
<pre><code>REACTOME v70.0 

Genesets

2-LTR circle formation (13)
5-Phosphoribose 1-diphosphate biosynthesis (3)
A tetrasaccharide linker sequence is required for GAG synthesis (26)
ABC transporters in lipid homeostasis (18)
ABO blood group biosynthesis (3)
ADP signalling through P2Y purinoceptor 1 (25)

Nodes

                                                                        label
R-HSA-164843                                           2-LTR circle formation
R-HSA-73843                        5-Phosphoribose 1-diphosphate biosynthesis
R-HSA-1971475 A tetrasaccharide linker sequence is required for GAG synthesis
R-HSA-5619084                                       ABC transporter disorders
R-HSA-1369062                           ABC transporters in lipid homeostasis
R-HSA-382556                           ABC-family proteins mediated transport
                         id length
R-HSA-164843   R-HSA-164843     13
R-HSA-73843     R-HSA-73843      3
R-HSA-1971475 R-HSA-1971475     26
R-HSA-5619084 R-HSA-5619084     77
R-HSA-1369062 R-HSA-1369062     18
R-HSA-382556   R-HSA-382556     22

Edges

          from            to
1 R-HSA-109581  R-HSA-109606
2 R-HSA-109581  R-HSA-169911
3 R-HSA-109581 R-HSA-5357769
4 R-HSA-109581   R-HSA-75153
5 R-HSA-109582  R-HSA-140877
6 R-HSA-109582  R-HSA-202733</code></pre>
</div>
</div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="sidebar">

        <div id="tocnav">
      <h2 class="hasAnchor">
<a href="#tocnav" class="anchor"></a>Contents</h2>
      <ul class="nav nav-pills nav-stacked">
<li><a href="#object-oriented-genesets">Object Oriented Genesets</a></li>
      <li><a href="#downloading-msigdb-genesets">Downloading <strong>msigdb</strong> Genesets</a></li>
      <li><a href="#downloading-enrichr-genesets">Downloading <strong>enrichr</strong> Genesets</a></li>
      <li><a href="#relational-genesets">Relational Genesets</a></li>
      </ul>
</div>
      </div>

</div>



      <footer><div class="copyright">
  <p>Developed by <a href="https://github.com/anfederico">Anthony Federico</a>, <a href="https://github.com/tetomonti">Stefano Monti</a>.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.4.1.</p>
</div>

      </footer>
</div>

  


  </body>
</html>
