<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Workflow for Co-Expression Modules • hypeR</title>
<!-- favicons --><link rel="icon" type="image/png" sizes="16x16" href="../../favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="../../favicon-32x32.png">
<link rel="apple-touch-icon" type="image/png" sizes="180x180" href="../../apple-touch-icon.png">
<link rel="apple-touch-icon" type="image/png" sizes="120x120" href="../../apple-touch-icon-120x120.png">
<link rel="apple-touch-icon" type="image/png" sizes="76x76" href="../../apple-touch-icon-76x76.png">
<link rel="apple-touch-icon" type="image/png" sizes="60x60" href="../../apple-touch-icon-60x60.png">
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script><!-- Bootstrap --><link href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.3.7/cosmo/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha256-U5ZEeKfGNOja007MMD3YBI0A3OSZOQbeG6z2f2Y0hu8=" crossorigin="anonymous"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" integrity="sha256-eZrrJcwDc/3uDhsdt61sL2oOBY362qM3lon1gyExkL0=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.4/clipboard.min.js" integrity="sha256-FiZwavyI2V6+EXO1U+xzLG3IKldpiTFf3153ea9zikQ=" crossorigin="anonymous"></script><!-- sticky kit --><script src="https://cdnjs.cloudflare.com/ajax/libs/sticky-kit/1.1.3/sticky-kit.min.js" integrity="sha256-c4Rlo1ZozqTPE2RLuvbusY3+SU1pQaJC0TjuhygMipw=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../../pkgdown.css" rel="stylesheet">
<script src="../../pkgdown.js"></script><meta property="og:title" content="Workflow for Co-Expression Modules">
<meta property="og:description" content="">
<meta name="twitter:card" content="summary">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body>
    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../../index.html">hypeR</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">1.1.9</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../../index.html">Home</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Docs
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../../articles/docs/installation.html">Installation</a>
    </li>
    <li>
      <a href="../../articles/docs/basics.html">Basic Usage</a>
    </li>
    <li>
      <a href="../../articles/docs/data.html">Downloading Data</a>
    </li>
    <li>
      <a href="../../articles/docs/visualize.html">Visualize Results</a>
    </li>
    <li>
      <a href="../../articles/docs/rgsets.html">Constructing Relational Genesets</a>
    </li>
    <li>
      <a href="../../articles/docs/multihyp.html">Multiple Signatures</a>
    </li>
    <li>
      <a href="../../articles/docs/reporting.html">Generating Reports</a>
    </li>
    <li>
      <a href="../../articles/docs/shiny.html">Shiny Applications</a>
    </li>
    <li>
      <a href="../../articles/docs/extending.html">Extending hypeR</a>
    </li>
  </ul>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Examples
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../../articles/examples/workflow.html">Co-Expression Modules</a>
    </li>
  </ul>
</li>
<li>
  <a href="../../reference/index.html">Reference</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/montilab/hypeR">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      
      </header><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1>Workflow for Co-Expression Modules</h1>
            
      
      <small class="dont-index">Source: <a href="https://github.com/montilab/hypeR/blob/master/vignettes/examples/workflow.Rmd"><code>vignettes/examples/workflow.Rmd</code></a></small>
      <div class="hidden name"><code>workflow.Rmd</code></div>

    </div>

    
    
<div id="introduction" class="section level2">
<h2 class="hasAnchor">
<a href="#introduction" class="anchor"></a>Introduction</h2>
<p>Often computational biologists must process, interpret, and share large amounts of biological data. A common example is interpreting gene co-expression modules across one or more phenotype, resulting in potentially hundreds of signatures to annotate. In this workflow example, we’ll generate gene co-expression modules for each of four breast cancer subtypes and use <strong>hypeR</strong> to annotate the results.</p>
</div>
<div id="requirements" class="section level2">
<h2 class="hasAnchor">
<a href="#requirements" class="anchor"></a>Requirements</h2>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1"></a><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/library">library</a></span>(hypeR)</span>
<span id="cb1-2"><a href="#cb1-2"></a><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/library">library</a></span>(WGCNA)</span>
<span id="cb1-3"><a href="#cb1-3"></a><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/library">library</a></span>(flashClust)</span>
<span id="cb1-4"><a href="#cb1-4"></a><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/library">library</a></span>(doParallel)</span>
<span id="cb1-5"><a href="#cb1-5"></a><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/library">library</a></span>(Biobase)</span>
<span id="cb1-6"><a href="#cb1-6"></a><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/library">library</a></span>(dplyr)</span>
<span id="cb1-7"><a href="#cb1-7"></a><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/library">library</a></span>(magrittr)</span></code></pre></div>
</div>
<div id="data" class="section level2">
<h2 class="hasAnchor">
<a href="#data" class="anchor"></a>Data</h2>
<p>We’re using publicly available RNA-seq data from breast cancer samples from <a href="https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga">The Cancer Genome Atlas</a>. The data has been normalized with DESeq2, log-transformed, and subtyped using <a href="https://bioconductor.org/packages/release/bioc/html/genefu.html">Genefu signatures</a>.</p>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb2-1"><a href="#cb2-1"></a>brca &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/readRDS">readRDS</a></span>(<span class="st">"data/TCGA-BRCA-RNA-Seq-Expression-Set.rds"</span>)</span></code></pre></div>
<p>Isolate tumor samples with subtypes LumA, LumB, Her2, and Basal.</p>
<div class="sourceCode" id="cb3"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb3-1"><a href="#cb3-1"></a>subtypes &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/c">c</a></span>(<span class="st">"LA"</span>, <span class="st">"LB"</span>, <span class="st">"H2"</span>, <span class="st">"BL"</span>)</span>
<span id="cb3-2"><a href="#cb3-2"></a>brca.tumors &lt;-<span class="st"> </span>brca[, brca<span class="op">$</span>subtype <span class="op">%in%</span><span class="st"> </span>subtypes]</span></code></pre></div>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb4-1"><a href="#cb4-1"></a><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/table">table</a></span>(brca.tumors<span class="op">$</span>subtype)</span></code></pre></div>
<pre><code>
 BL  H2  LA  LB 
169 209 510 198 </code></pre>
<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb6-1"><a href="#cb6-1"></a><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/dim">dim</a></span>(brca.tumors)</span></code></pre></div>
<pre><code>Features  Samples 
   36812     1086 </code></pre>
</div>
<div id="filtering" class="section level2">
<h2 class="hasAnchor">
<a href="#filtering" class="anchor"></a>Filtering</h2>
<p>We’ll filter genes based on three conditions:</p>
<div id="annotated-from-experiment-or-a-direct-assay" class="section level4">
<h4 class="hasAnchor">
<a href="#annotated-from-experiment-or-a-direct-assay" class="anchor"></a>Annotated from experiment or a direct assay</h4>
<p>The following dataset for human and other species can be downloaded from <a href="%22http://current.geneontology.org/products/pages/downloads.html%22">Gene Ontology</a>.</p>
<div class="sourceCode" id="cb8"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb8-1"><a href="#cb8-1"></a>annotations &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/utils/topics/read.table">read.delim</a></span>(<span class="dt">file=</span><span class="st">"data/Gene-Ontology-Human-Annotations.gaf"</span>, </span>
<span id="cb8-2"><a href="#cb8-2"></a>                          <span class="dt">skip=</span><span class="dv">31</span>, </span>
<span id="cb8-3"><a href="#cb8-3"></a>                          <span class="dt">header=</span><span class="ot">FALSE</span>, </span>
<span id="cb8-4"><a href="#cb8-4"></a>                          <span class="dt">stringsAsFactors=</span><span class="ot">FALSE</span>)</span>
<span id="cb8-5"><a href="#cb8-5"></a></span>
<span id="cb8-6"><a href="#cb8-6"></a>genes<span class="fl">.0</span> &lt;-<span class="st"> </span>annotations <span class="op">%&gt;%</span></span>
<span id="cb8-7"><a href="#cb8-7"></a><span class="st">           </span><span class="kw"><a href="https://dplyr.tidyverse.org/reference/filter.html">filter</a></span>(V7 <span class="op">%in%</span><span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/c">c</a></span>(<span class="st">"EXP"</span>, <span class="st">"IDA"</span>)) <span class="op">%&gt;%</span></span>
<span id="cb8-8"><a href="#cb8-8"></a><span class="st">           </span><span class="kw"><a href="https://www.rdocumentation.org/packages/magrittr/topics/aliases">use_series</a></span>(V3) <span class="op">%&gt;%</span></span>
<span id="cb8-9"><a href="#cb8-9"></a><span class="st">           </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/unique">unique</a></span>()</span></code></pre></div>
</div>
<div id="non-zero-expression-in-15-of-samples-of-each-subtype" class="section level4">
<h4 class="hasAnchor">
<a href="#non-zero-expression-in-15-of-samples-of-each-subtype" class="anchor"></a>Non-zero expression in 1/5 of samples of each subtype</h4>
<div class="sourceCode" id="cb9"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb9-1"><a href="#cb9-1"></a>variable.genes &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/list">list</a></span>()</span>
<span id="cb9-2"><a href="#cb9-2"></a><span class="cf">for</span> (i <span class="cf">in</span> subtypes) {</span>
<span id="cb9-3"><a href="#cb9-3"></a>    brca.tumors.sub &lt;-<span class="st"> </span>brca.tumors[,brca.tumors<span class="op">$</span>subtype <span class="op">==</span><span class="st"> </span>i]</span>
<span id="cb9-4"><a href="#cb9-4"></a>    sub.exprs.mat &lt;-<span class="st"> </span>Biobase<span class="op">::</span><span class="kw"><a href="https://www.rdocumentation.org/packages/Biobase/topics/exprs">exprs</a></span>(brca.tumors.sub)</span>
<span id="cb9-5"><a href="#cb9-5"></a>    n.samples &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/nrow">ncol</a></span>(sub.exprs.mat)</span>
<span id="cb9-6"><a href="#cb9-6"></a>    sub.genes.keep &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/colSums">rowSums</a></span>(sub.exprs.mat <span class="op">!=</span><span class="st"> </span><span class="dv">0</span>)<span class="op">/</span>n.samples <span class="op">&gt;=</span><span class="st"> </span><span class="fl">0.2</span></span>
<span id="cb9-7"><a href="#cb9-7"></a>    variable.genes[[i]] &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/colnames">rownames</a></span>(sub.exprs.mat)[sub.genes.keep]</span>
<span id="cb9-8"><a href="#cb9-8"></a>}</span>
<span id="cb9-9"><a href="#cb9-9"></a>genes<span class="fl">.1</span> &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/funprog">Reduce</a></span>(intersect,  variable.genes)</span></code></pre></div>
</div>
<div id="non-zero-variance-in-each-subtype" class="section level4">
<h4 class="hasAnchor">
<a href="#non-zero-variance-in-each-subtype" class="anchor"></a>Non-zero variance in each subtype</h4>
<div class="sourceCode" id="cb10"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb10-1"><a href="#cb10-1"></a>variable.genes &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/list">list</a></span>()</span>
<span id="cb10-2"><a href="#cb10-2"></a><span class="cf">for</span> (i <span class="cf">in</span> subtypes) {</span>
<span id="cb10-3"><a href="#cb10-3"></a>    brca.tumors.sub &lt;-<span class="st"> </span>brca.tumors[,brca.tumors<span class="op">$</span>subtype <span class="op">==</span><span class="st"> </span>i]</span>
<span id="cb10-4"><a href="#cb10-4"></a>    sub.exprs.mat &lt;-<span class="st"> </span>Biobase<span class="op">::</span><span class="kw"><a href="https://www.rdocumentation.org/packages/Biobase/topics/exprs">exprs</a></span>(brca.tumors.sub)</span>
<span id="cb10-5"><a href="#cb10-5"></a>    n.samples &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/nrow">ncol</a></span>(sub.exprs.mat)</span>
<span id="cb10-6"><a href="#cb10-6"></a>    sub.genes.keep &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/apply">apply</a></span>(sub.exprs.mat, <span class="dv">1</span>, var) <span class="op">!=</span><span class="st"> </span><span class="dv">0</span></span>
<span id="cb10-7"><a href="#cb10-7"></a>    variable.genes[[i]] &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/colnames">rownames</a></span>(sub.exprs.mat)[sub.genes.keep]</span>
<span id="cb10-8"><a href="#cb10-8"></a>}</span>
<span id="cb10-9"><a href="#cb10-9"></a>genes<span class="fl">.2</span> &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/funprog">Reduce</a></span>(intersect,  variable.genes)</span></code></pre></div>
<div class="sourceCode" id="cb11"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb11-1"><a href="#cb11-1"></a>genes.keep &lt;-<span class="st"> </span><span class="kw"><a href="https://dplyr.tidyverse.org/reference/setops.html">intersect</a></span>(genes<span class="fl">.0</span>, <span class="kw"><a href="https://dplyr.tidyverse.org/reference/setops.html">intersect</a></span>(genes<span class="fl">.1</span>, genes<span class="fl">.2</span>))</span>
<span id="cb11-2"><a href="#cb11-2"></a>brca.tumors.filtered &lt;-<span class="st"> </span>brca.tumors[genes.keep,]</span>
<span id="cb11-3"><a href="#cb11-3"></a><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/dim">dim</a></span>(brca.tumors.filtered)</span></code></pre></div>
<pre><code>Features  Samples 
   11104     1086 </code></pre>
</div>
</div>
<div id="co-expression" class="section level2">
<h2 class="hasAnchor">
<a href="#co-expression" class="anchor"></a>Co-Expression</h2>
<p>We’ll write a wrapper function for the <a href="https://horvath.genetics.ucla.edu/html/CoexpressionNetwork/Rpackages/WGCNA/">Weighted Gene Co-Expression Analysis</a> R package to define gene co-expression modules for each subtype.</p>
<div class="sourceCode" id="cb13"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb13-1"><a href="#cb13-1"></a>wgcna &lt;-<span class="st"> </span><span class="cf">function</span>(eset, subtype) {</span>
<span id="cb13-2"><a href="#cb13-2"></a></span>
<span id="cb13-3"><a href="#cb13-3"></a>    <span class="co"># Format data input</span></span>
<span id="cb13-4"><a href="#cb13-4"></a>    counts &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/t">t</a></span>(<span class="kw"><a href="https://www.rdocumentation.org/packages/Biobase/topics/exprs">exprs</a></span>(eset[,eset<span class="op">$</span>subtype <span class="op">==</span><span class="st"> </span>subtype]))</span>
<span id="cb13-5"><a href="#cb13-5"></a>    </span>
<span id="cb13-6"><a href="#cb13-6"></a>    <span class="co"># Choose a threshold which leads to best fit for scale-free topology</span></span>
<span id="cb13-7"><a href="#cb13-7"></a>    threshtest &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/WGCNA/topics/pickSoftThreshold">pickSoftThreshold</a></span>(counts, <span class="dt">powerVector=</span><span class="dv">1</span><span class="op">:</span><span class="dv">10</span>)</span>
<span id="cb13-8"><a href="#cb13-8"></a>    tmax &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/Extremes">max</a></span>(threshtest<span class="op">$</span>fitIndices<span class="op">$</span>SFT.R.sq)</span>
<span id="cb13-9"><a href="#cb13-9"></a>    threshold &lt;-<span class="st"> </span>threshtest<span class="op">$</span>fitIndices[threshtest<span class="op">$</span>fitIndices<span class="op">$</span>SFT.R.sq <span class="op">==</span><span class="st"> </span>tmax, <span class="st">"Power"</span>]    </span>
<span id="cb13-10"><a href="#cb13-10"></a>    </span>
<span id="cb13-11"><a href="#cb13-11"></a>    <span class="co"># Create adjacency matrix based on gene co-expression </span></span>
<span id="cb13-12"><a href="#cb13-12"></a>    adjacencyMatrix &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/WGCNA/topics/adjacency">adjacency</a></span>(counts, <span class="dt">power=</span>threshold) <span class="op">%&gt;%</span><span class="st"> </span></span>
<span id="cb13-13"><a href="#cb13-13"></a><span class="st">                       </span><span class="kw"><a href="https://www.rdocumentation.org/packages/WGCNA/topics/TOMsimilarity">TOMsimilarity</a></span>()</span>
<span id="cb13-14"><a href="#cb13-14"></a>    </span>
<span id="cb13-15"><a href="#cb13-15"></a>    <span class="co"># Identify modules using dynamic tree cut</span></span>
<span id="cb13-16"><a href="#cb13-16"></a>    modules &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/flashClust/topics/flashClust">hclust</a></span>(<span class="kw"><a href="https://www.rdocumentation.org/packages/stats/topics/dist">as.dist</a></span>(<span class="dv">1</span><span class="op">-</span>adjacencyMatrix), <span class="dt">method=</span><span class="st">"complete"</span>) <span class="op">%&gt;%</span></span>
<span id="cb13-17"><a href="#cb13-17"></a><span class="st">               </span><span class="kw">cutreeDynamic</span>(<span class="dt">distM=</span><span class="dv">1</span><span class="op">-</span>adjacencyMatrix, </span>
<span id="cb13-18"><a href="#cb13-18"></a>                             <span class="dt">deepSplit=</span><span class="dv">2</span>, </span>
<span id="cb13-19"><a href="#cb13-19"></a>                             <span class="dt">pamRespectsDendro=</span><span class="ot">FALSE</span>, </span>
<span id="cb13-20"><a href="#cb13-20"></a>                             <span class="dt">minClusterSize=</span><span class="dv">20</span>)</span>
<span id="cb13-21"><a href="#cb13-21"></a>    </span>
<span id="cb13-22"><a href="#cb13-22"></a>    <span class="co"># Calculate eigengenes</span></span>
<span id="cb13-23"><a href="#cb13-23"></a>    eigengenes &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/WGCNA/topics/moduleEigengenes">moduleEigengenes</a></span>(<span class="dt">expr=</span>counts, <span class="dt">colors=</span>modules)<span class="op">$</span>eigengenes</span>
<span id="cb13-24"><a href="#cb13-24"></a></span>
<span id="cb13-25"><a href="#cb13-25"></a>    <span class="co"># Merge modules with similar eigengenes</span></span>
<span id="cb13-26"><a href="#cb13-26"></a>    modules &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/WGCNA/topics/mergeCloseModules">mergeCloseModules</a></span>(<span class="dt">exprData=</span>counts, </span>
<span id="cb13-27"><a href="#cb13-27"></a>                                 <span class="dt">colors=</span>modules, </span>
<span id="cb13-28"><a href="#cb13-28"></a>                                 <span class="dt">MEs=</span>eigengenes, </span>
<span id="cb13-29"><a href="#cb13-29"></a>                                 <span class="dt">cutHeight=</span><span class="fl">0.25</span>)</span>
<span id="cb13-30"><a href="#cb13-30"></a>    </span>
<span id="cb13-31"><a href="#cb13-31"></a>    <span class="co"># Create a list of modules    </span></span>
<span id="cb13-32"><a href="#cb13-32"></a>    modulesList &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/vector">vector</a></span>(<span class="st">"list"</span>, <span class="dt">length=</span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/length">length</a></span>(<span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/unique">unique</a></span>(modules<span class="op">$</span>colors)))</span>
<span id="cb13-33"><a href="#cb13-33"></a>    </span>
<span id="cb13-34"><a href="#cb13-34"></a>    <span class="co"># Assign genes to correct module</span></span>
<span id="cb13-35"><a href="#cb13-35"></a>    uniqueModules &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/sort">sort</a></span>(<span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/unique">unique</a></span>(modules<span class="op">$</span>colors))</span>
<span id="cb13-36"><a href="#cb13-36"></a>    <span class="cf">for</span>(i <span class="cf">in</span> <span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/seq">seq</a></span>(<span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/length">length</a></span>(uniqueModules))){</span>
<span id="cb13-37"><a href="#cb13-37"></a>        pindex &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/which">which</a></span>(modules<span class="op">$</span>colors<span class="op">==</span>uniqueModules[i])</span>
<span id="cb13-38"><a href="#cb13-38"></a>        modulesList[[i]] &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/colnames">rownames</a></span>(eset)[pindex]</span>
<span id="cb13-39"><a href="#cb13-39"></a>    }</span>
<span id="cb13-40"><a href="#cb13-40"></a>    <span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/names">names</a></span>(modulesList) &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/paste">paste0</a></span>(<span class="st">'M'</span>, <span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/seq">seq</a></span>(<span class="dv">0</span>, <span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/length">length</a></span>(modulesList)<span class="op">-</span><span class="dv">1</span>))</span>
<span id="cb13-41"><a href="#cb13-41"></a>    </span>
<span id="cb13-42"><a href="#cb13-42"></a>    <span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/function">return</a></span>(modulesList)</span>
<span id="cb13-43"><a href="#cb13-43"></a>}</span></code></pre></div>
<p>The dataset we are saving is a list of gene co-expression modules defined for each subtype.</p>
<div class="sourceCode" id="cb14"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb14-1"><a href="#cb14-1"></a><span class="co"># Parallel computation</span></span>
<span id="cb14-2"><a href="#cb14-2"></a><span class="kw"><a href="https://www.rdocumentation.org/packages/doParallel/topics/registerDoParallel">registerDoParallel</a></span>(<span class="dt">cores=</span><span class="dv">3</span>)</span>
<span id="cb14-3"><a href="#cb14-3"></a><span class="kw">getDoParWorkers</span>()</span>
<span id="cb14-4"><a href="#cb14-4"></a></span>
<span id="cb14-5"><a href="#cb14-5"></a><span class="co"># Co-expression modules</span></span>
<span id="cb14-6"><a href="#cb14-6"></a>hypdat &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/list">list</a></span>(<span class="st">"LumA"</span>  =<span class="st"> </span><span class="kw">wgcna</span>(brca.tumors.filtered, <span class="dt">subtype=</span><span class="st">"LA"</span>),</span>
<span id="cb14-7"><a href="#cb14-7"></a>               <span class="st">"LumB"</span>  =<span class="st"> </span><span class="kw">wgcna</span>(brca.tumors.filtered, <span class="dt">subtype=</span><span class="st">"LB"</span>),</span>
<span id="cb14-8"><a href="#cb14-8"></a>               <span class="st">"Her2"</span>  =<span class="st"> </span><span class="kw">wgcna</span>(brca.tumors.filtered, <span class="dt">subtype=</span><span class="st">"H2"</span>),</span>
<span id="cb14-9"><a href="#cb14-9"></a>               <span class="st">"Basal"</span> =<span class="st"> </span><span class="kw">wgcna</span>(brca.tumors.filtered, <span class="dt">subtype=</span><span class="st">"BL"</span>))</span>
<span id="cb14-10"><a href="#cb14-10"></a></span>
<span id="cb14-11"><a href="#cb14-11"></a><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/readRDS">saveRDS</a></span>(hypdat, <span class="st">"data/TCGA-BRCA-Co-Expression-Modules.rds"</span>)</span></code></pre></div>
</div>
<div id="using-hyper" class="section level2">
<h2 class="hasAnchor">
<a href="#using-hyper" class="anchor"></a>Using hypeR</h2>
<p>We’d like to see if any modules are enriched with genes associated with known biological processes. We can use <strong>hypeR</strong> to perform geneset enrichment and efficiently visualize, analyze, and share the results. When dealing with many genesets, it’s often useful to understand the relationships between them. This allows researchers to summarize many enriched pathways as more general biological processes. To do this, we rely on curated relationships defined between them. For example, <a href="https://reactome.org/">REACTOME</a> conveniently defines their genesets in a <a href="https://reactome.org/PathwayBrowser/">hiearchy of pathways</a>. This data can be formatted into a relational genesets object.</p>
<div class="sourceCode" id="cb15"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb15-1"><a href="#cb15-1"></a>rgsets &lt;-<span class="st"> </span><span class="kw"><a href="../../reference/hyperdb_fetch.html">hyperdb_fetch</a></span>(<span class="dt">type=</span><span class="st">"rgsets"</span>, <span class="st">"REACTOME"</span>)</span></code></pre></div>
<div class="sourceCode" id="cb16"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb16-1"><a href="#cb16-1"></a>hypdat &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/readRDS">readRDS</a></span>(<span class="st">"data/TCGA-BRCA-Co-Expression-Modules.rds"</span>)</span>
<span id="cb16-2"><a href="#cb16-2"></a><span class="kw"><a href="https://www.rdocumentation.org/packages/utils/topics/str">str</a></span>(hypdat)</span></code></pre></div>
<pre><code>List of 4
 $ LumA :List of 25
  ..$ M0 : chr [1:2429] "MT-RNR1" "CIB4" "FEZF1" "FAM170A" ...
  ..$ M1 : chr [1:1875] "ARHGEF18" "TMEM129" "CCDC78" "TYW5" ...
  ..$ M2 : chr [1:782] "GXYLT2" "SH3PXD2B" "CPED1" "TCAF2" ...
  ..$ M3 : chr [1:687] "TMEM189" "TOMM40" "MASP2" "SULT2B1" ...
  ..$ M4 : chr [1:275] "CYP2D7" "ANO9" "SH3D21" "CARNS1" ...
  ..$ M5 : chr [1:238] "SPAAR" "PDE2A" "CLDN5" "PIK3C2B" ...
  ..$ M6 : chr [1:274] "ESYT3" "ODAM" "TRABD2B" "PCARE" ...
  ..$ M7 : chr [1:157] "CSKMT" "EIF3F" "SDHD" "GOSR2" ...
  ..$ M8 : chr [1:285] "GSAP" "WTIP" "OLFM2" "FZD9" ...
  ..$ M9 : chr [1:313] "POU6F1" "E2F8" "FAM72C" "CDC7" ...
  ..$ M10: chr [1:116] "WDR27" "CCDC66" "CFAP69" "NPIPB12" ...
  ..$ M11: chr [1:138] "LGALS8" "BCL9" "IRF6" "ARHGEF11" ...
  ..$ M12: chr [1:89] "PES1" "PLXNB2" "SLC25A17" "SCO2" ...
  ..$ M13: chr [1:120] "GPR89A" "HAX1" "CACNB4" "SCAMP3" ...
  ..$ M14: chr [1:88] "AZIN1" "EIF3H" "RIPK2" "RAD21" ...
  ..$ M15: chr [1:86] "ACACB" "AQP7" "HSPB6" "GYG2" ...
  ..$ M16: chr [1:1690] "RNF213" "NBDY" "UBA6" "ESYT2" ...
  ..$ M17: chr [1:956] "IRGM" "ZC3H12D" "TESPA1" "DENND3" ...
  ..$ M18: chr [1:107] "UBE2QL1" "MTMR11" "ENC1" "RGS16" ...
  ..$ M19: chr [1:71] "RFXAP" "ELL2" "GPAA1" "RAMP1" ...
  ..$ M20: chr [1:105] "NDUFAF8" "CENPX" "MIF4GD" "MCRIP1" ...
  ..$ M21: chr [1:62] "FAM83G" "MYO1C" "DVL2" "PFAS" ...
  ..$ M22: chr [1:79] "LGALS9" "CXCL11" "APOL1" "IFIT3" ...
  ..$ M23: chr [1:42] "C2CD4B" "GADD45B" "PPP1R15A" "FOS" ...
  ..$ M24: chr [1:40] "PSMD11" "PSMD12" "PPM1D" "SPOP" ...
 $ LumB :List of 13
  ..$ M0 : chr [1:6508] "MT-RNR1" "NBDY" "POU6F1" "ESYT3" ...
  ..$ M1 : chr [1:940] "UHRF1BP1L" "NBAS" "TYW5" "VWA8" ...
  ..$ M2 : chr [1:593] "GXYLT2" "SH3PXD2B" "CPED1" "TRABD2B" ...
  ..$ M3 : chr [1:743] "ZC3H12D" "TESPA1" "DENND3" "TMEM150B" ...
  ..$ M4 : chr [1:1287] "ARHGEF18" "CYP2D7" "TMEM129" "UBA6" ...
  ..$ M5 : chr [1:242] "E2F8" "CDC7" "PLK4" "UBE2C" ...
  ..$ M6 : chr [1:227] "WTIP" "OLFM2" "STXBP3" "TSPAN4" ...
  ..$ M7 : chr [1:89] "AZIN1" "EIF3H" "XPOT" "RAD21" ...
  ..$ M8 : chr [1:83] "NOP56" "CLDN3" "GPAA1" "PTP4A3" ...
  ..$ M9 : chr [1:129] "SPAAR" "GRM8" "PDE2A" "NR5A2" ...
  ..$ M10: chr [1:127] "CSKMT" "EIF3F" "CDC42EP2" "PDCD5" ...
  ..$ M11: chr [1:86] "RNF213" "SMTNL1" "PATL2" "BTN3A3" ...
  ..$ M12: chr [1:50] "AQP7" "HSPB6" "CIDEA" "ALDH1L1" ...
 $ Her2 :List of 16
  ..$ M0 : chr [1:6176] "ARHGEF18" "CYP2D7" "MT-RNR1" "NBDY" ...
  ..$ M1 : chr [1:669] "ESYT2" "UHRF1BP1L" "CNOT1" "CASTOR2" ...
  ..$ M2 : chr [1:430] "GXYLT2" "UBE2QL1" "SH3PXD2B" "FOXN3" ...
  ..$ M3 : chr [1:1842] "TMEM129" "UBA6" "PTPDC1" "CCDC78" ...
  ..$ M4 : chr [1:349] "E2F8" "TRABD2B" "APBB1" "PSMD12" ...
  ..$ M5 : chr [1:808] "ZC3H12D" "TESPA1" "DENND3" "TMEM150B" ...
  ..$ M6 : chr [1:136] "WTIP" "TSPAN4" "SEPT4" "TGFB1I1" ...
  ..$ M7 : chr [1:131] "CFAP69" "TMEM189" "PSMD14" "UBE2C" ...
  ..$ M8 : chr [1:126] "POU6F1" "PDE2A" "CLDN5" "PLPP1" ...
  ..$ M9 : chr [1:122] "HIP1" "EIF3F" "COX7A2L" "PDCD5" ...
  ..$ M10: chr [1:93] "CPED1" "RCCD1" "F2RL2" "NR5A2" ...
  ..$ M11: chr [1:60] "NDUFAF8" "CENPX" "MCRIP1" "HGS" ...
  ..$ M12: chr [1:57] "RNF213" "SMTNL1" "IFIT3" "UBE2L6" ...
  ..$ M13: chr [1:40] "ACACB" "AQP7" "VEGFD" "CIDEA" ...
  ..$ M14: chr [1:40] "PSMA7" "GPAA1" "DGAT1" "RECQL4" ...
  ..$ M15: chr [1:25] "LGALS9" "BTN3A3" "BTN3A1" "APOL1" ...
 $ Basal:List of 10
  ..$ M0: chr [1:6836] "ARHGEF18" "CYP2D7" "MT-RNR1" "NBDY" ...
  ..$ M1: chr [1:1523] "TMEM129" "UBA6" "UHRF1BP1L" "TYW5" ...
  ..$ M2: chr [1:960] "ZC3H12D" "TESPA1" "DENND3" "CPED1" ...
  ..$ M3: chr [1:919] "MED19" "VPS37C" "PGP" "SDHAF1" ...
  ..$ M4: chr [1:371] "GXYLT2" "NRP1" "TNFSF11" "RASGRF2" ...
  ..$ M5: chr [1:155] "C2CD4B" "F2RL2" "PDE2A" "NR5A2" ...
  ..$ M6: chr [1:124] "PSD" "TOMM40" "XRCC3" "CHMP2A" ...
  ..$ M7: chr [1:70] "NDUFAF8" "CENPX" "MCRIP1" "HGS" ...
  ..$ M8: chr [1:103] "RNF213" "ANO9" "SMTNL1" "TRIM38" ...
  ..$ M9: chr [1:43] "TULP3" "GAPDH" "LDHB" "CCNB1" ...</code></pre>
<p>By default, <code><a href="../../reference/hypeR.html">hypeR()</a></code> recognizes a single signature or a named list of signatures. Because we are using a nested list of signatures, we can use the <code>lapply</code> function to return a list of <code>multihyp</code> objects, which can be directly passed to <code><a href="../../reference/hyp_to_rmd.html">hyp_to_rmd()</a></code> for generating reports.</p>
<div class="sourceCode" id="cb18"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb18-1"><a href="#cb18-1"></a>lmultihyp_obj &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/lapply">lapply</a></span>(hypdat, hypeR, rgsets, <span class="dt">test=</span><span class="st">"hypergeometric"</span>, <span class="dt">bg=</span><span class="dv">36812</span>, <span class="dt">fdr_cutoff=</span><span class="fl">0.05</span>)</span></code></pre></div>
<p>We can pass a list of <code>multihyp</code> objects directly to <code><a href="../../reference/hyp_to_rmd.html">hyp_to_rmd()</a></code>, which will accept multiple formats, including a single <code>hyp</code> or <code>multihyp</code> object as well as a list of either, including a list of <code>hyp</code> or <code>multihyp</code> objects together. When a list of <code>multihyp</code> objects are passed for example, each experiment will become its own section, while each signature becomes its own tab within that section. In this example, each section is linked to a subtype, and each tab within that section holds the enrichment results for a specific co-expression module.</p>
<div class="sourceCode" id="cb19"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb19-1"><a href="#cb19-1"></a><span class="kw"><a href="../../reference/hyp_to_rmd.html">hyp_to_rmd</a></span>(lmultihyp_obj,</span>
<span id="cb19-2"><a href="#cb19-2"></a>           <span class="dt">file_path=</span><span class="st">"TCGA-BRCA-Workflow-Report.rmd"</span>,</span>
<span id="cb19-3"><a href="#cb19-3"></a>           <span class="dt">title=</span><span class="st">"Hyper Enrichment (hypeR)"</span>,</span>
<span id="cb19-4"><a href="#cb19-4"></a>           <span class="dt">subtitle=</span><span class="st">"Co-Expression Modules for Breast Cancer Subtypes"</span>,</span>
<span id="cb19-5"><a href="#cb19-5"></a>           <span class="dt">author=</span><span class="st">"Anthony Federico, Stefano Monti"</span>,</span>
<span id="cb19-6"><a href="#cb19-6"></a>           <span class="dt">show_dots=</span>F,</span>
<span id="cb19-7"><a href="#cb19-7"></a>           <span class="dt">show_emaps=</span>F,</span>
<span id="cb19-8"><a href="#cb19-8"></a>           <span class="dt">show_hmaps=</span>T,</span>
<span id="cb19-9"><a href="#cb19-9"></a>           <span class="dt">show_tables=</span>T)</span></code></pre></div>
<p>This report can be viewed at <a href="https://montilab.github.io/hypeR-reporting" class="uri">https://montilab.github.io/hypeR-reporting</a>.</p>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="sidebar">
        <div id="tocnav">
      <h2 class="hasAnchor">
<a href="#tocnav" class="anchor"></a>Contents</h2>
      <ul class="nav nav-pills nav-stacked">
<li><a href="#introduction">Introduction</a></li>
      <li><a href="#requirements">Requirements</a></li>
      <li><a href="#data">Data</a></li>
      <li><a href="#filtering">Filtering</a></li>
      <li><a href="#co-expression">Co-Expression</a></li>
      <li><a href="#using-hyper">Using hypeR</a></li>
      </ul>
</div>
      </div>

</div>


      <footer><div class="copyright">
  <p>Developed by <a href="https://github.com/anfederico">Anthony Federico</a>, <a href="https://github.com/montilab">Stefano Monti</a>.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.3.0.</p>
</div>
      </footer>
</div>

  

  </body>
</html>
