<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Downloading Data â€¢ hypeR</title>
<!-- favicons --><link rel="icon" type="image/png" sizes="16x16" href="../favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="../favicon-32x32.png">
<link rel="apple-touch-icon" type="image/png" sizes="180x180" href="../apple-touch-icon.png">
<link rel="apple-touch-icon" type="image/png" sizes="120x120" href="../apple-touch-icon-120x120.png">
<link rel="apple-touch-icon" type="image/png" sizes="76x76" href="../apple-touch-icon-76x76.png">
<link rel="apple-touch-icon" type="image/png" sizes="60x60" href="../apple-touch-icon-60x60.png">
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script><!-- Bootstrap --><link href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.3.7/cosmo/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha256-U5ZEeKfGNOja007MMD3YBI0A3OSZOQbeG6z2f2Y0hu8=" crossorigin="anonymous"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" integrity="sha256-eZrrJcwDc/3uDhsdt61sL2oOBY362qM3lon1gyExkL0=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.4/clipboard.min.js" integrity="sha256-FiZwavyI2V6+EXO1U+xzLG3IKldpiTFf3153ea9zikQ=" crossorigin="anonymous"></script><!-- sticky kit --><script src="https://cdnjs.cloudflare.com/ajax/libs/sticky-kit/1.1.3/sticky-kit.min.js" integrity="sha256-c4Rlo1ZozqTPE2RLuvbusY3+SU1pQaJC0TjuhygMipw=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="Downloading Data">
<meta property="og:description" content="">
<meta name="twitter:card" content="summary">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body>
    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">hypeR</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">1.10.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../index.html">Home</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Tutorials
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/installation.html">Installation</a>
    </li>
    <li>
      <a href="../articles/basics.html">Basic Usage</a>
    </li>
    <li>
      <a href="../articles/data.html">Downloading Data</a>
    </li>
    <li>
      <a href="../articles/visualize.html">Visualize Results</a>
    </li>
    <li>
      <a href="../articles/relational.html">Relational Genesets</a>
    </li>
    <li>
      <a href="../articles/multihyp.html">Multiple Signatures</a>
    </li>
    <li>
      <a href="../articles/reporting.html">Generating Reports</a>
    </li>
  </ul>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/montilab/hypeR">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      
      </header><script src="data_files/htmlwidgets-1.3/htmlwidgets.js"></script><script src="data_files/jquery-1.12.4/jquery.min.js"></script><link href="data_files/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet">
<script src="data_files/datatables-binding-0.5/datatables.js"></script><link href="data_files/dt-core-bootstrap-1.10.16/css/dataTables.bootstrap.min.css" rel="stylesheet">
<link href="data_files/dt-core-bootstrap-1.10.16/css/dataTables.bootstrap.extra.css" rel="stylesheet">
<script src="data_files/dt-core-bootstrap-1.10.16/js/jquery.dataTables.min.js"></script><script src="data_files/dt-core-bootstrap-1.10.16/js/dataTables.bootstrap.min.js"></script><link href="data_files/crosstalk-1.0.0/css/crosstalk.css" rel="stylesheet">
<script src="data_files/crosstalk-1.0.0/js/crosstalk.min.js"></script><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1>Downloading Data</h1>
            
      
      <small class="dont-index">Source: <a href="https://github.com/montilab/hypeR/blob/master/vignettes/data.Rmd"><code>vignettes/data.Rmd</code></a></small>
      <div class="hidden name"><code>data.Rmd</code></div>

    </div>

    
    
<div id="downloading-genesets-from-msigdb" class="section level3">
<h3 class="hasAnchor">
<a href="#downloading-genesets-from-msigdb" class="anchor"></a>Downloading genesets from msigdb</h3>
<p>For most user purposes, the genesets hosted by <a href="https://software.broadinstitute.org/gsea/msigdb/collections.jsp">msigdb</a> are more than adequate to perform geneset enrichment analysis. There are various types of genesets available across many species. Therefore, we have added convenient functions for retrieving msigdb data compatible with hypeR.</p>
<div id="available-genesets" class="section level4">
<h4 class="hasAnchor">
<a href="#available-genesets" class="anchor"></a>Available genesets</h4>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb1-1" title="1"><span class="kw">msigdb_info</span>()</a></code></pre></div>
<pre><code>#&gt; |------------------------------------------------------------|
#&gt; | Available Gene Sets                                 v6.2.1 |
#&gt; |------------------------------------------------------------|
#&gt; | C1             | Positional (326)                          |
#&gt; | C2.CGP         | Chemical and Genetic Perturbations (3433) |
#&gt; | C2.CP          | Canonical Pathways (252)                  |
#&gt; | C2.CP.BIOCARTA | Canonical BIOCARTA (217)                  |
#&gt; | C2.CP.KEGG     | Canonical KEGG (186)                      |
#&gt; | C2.CP.REACTOME | Canonical REACTOME (674)                  |
#&gt; | C3.MIR         | Motif miRNA Targets (221)                 |
#&gt; | C3.TFT         | Motif Transcription Factor Targets (615)  |
#&gt; | C4.CGN         | Cancer Gene Neighborhoods (427)           |
#&gt; | C4.CM          | Cancer Modules (431)                      |
#&gt; | C5.BP          | GO Biological Process (4436)              |
#&gt; | C5.CC          | GO Cellular Component (580)               |
#&gt; | C5.MF          | GO Molecular Function (901)               |
#&gt; | C6             | Oncogenic Signatures (189)                |
#&gt; | C7             | Immunologic Signatures (4872)             |
#&gt; | H              | Hallmark (50)                             |
#&gt; |------------------------------------------------------------|
#&gt; | Source: http://software.broadinstitute.org/gsea/msigdb     |
#&gt; |------------------------------------------------------------|</code></pre>
</div>
<div id="downloading-and-loading-genesets" class="section level4">
<h4 class="hasAnchor">
<a href="#downloading-and-loading-genesets" class="anchor"></a>Downloading and loading genesets</h4>
<p>Use <strong><em>msigdb_download_one</em></strong> to download a single geneset into memory.</p>
<div class="sourceCode" id="cb3"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb3-1" title="1">HALLMARK &lt;-<span class="st"> </span><span class="kw">msigdb_download_one</span>(<span class="dt">species=</span><span class="st">"Homo sapiens"</span>, <span class="dt">category=</span><span class="st">"H"</span>)</a></code></pre></div>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb4-1" title="1"><span class="kw"><a href="https://www.rdocumentation.org/packages/utils/topics/head">head</a></span>(<span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/names">names</a></span>(HALLMARK))</a></code></pre></div>
<pre><code>#&gt; [1] "HALLMARK_ADIPOGENESIS"        "HALLMARK_ALLOGRAFT_REJECTION"
#&gt; [3] "HALLMARK_ANDROGEN_RESPONSE"   "HALLMARK_ANGIOGENESIS"       
#&gt; [5] "HALLMARK_APICAL_JUNCTION"     "HALLMARK_APICAL_SURFACE"</code></pre>
<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb6-1" title="1"><span class="kw"><a href="https://www.rdocumentation.org/packages/utils/topics/head">head</a></span>(HALLMARK[[<span class="dv">1</span>]])</a></code></pre></div>
<pre><code>#&gt; [1] "ABCA1" "ABCB8" "ACAA2" "ACADL" "ACADM" "ACADS"</code></pre>
<p>Use <strong><em>msigdb_download_all</em></strong> to retrieve all genesets for a given species. By default, genesets are cached to a temporary directory, therefore a path object is returned. Use the path object with <strong><em>msigdb_fetch</em></strong> to load genesets into memory. Users can also specify a directory to download to for repeated use.</p>
<div class="sourceCode" id="cb8"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb8-1" title="1">msigdb_path &lt;-<span class="st"> </span><span class="kw">msigdb_download_all</span>(<span class="dt">species=</span><span class="st">"Homo sapiens"</span>)</a></code></pre></div>
<pre><code>#&gt; v6.2.1
#&gt; Downloading Gene Sets to...
#&gt; /var/folders/7j/bg701dss38lc2xq26vt_by240000gn/T//RtmpD5s5yA
#&gt; - C1 -&gt; 326 Gene Sets 
#&gt; - C2.CGP -&gt; 3433 Gene Sets 
#&gt; - C2.CP -&gt; 252 Gene Sets 
#&gt; - C2.CP.BIOCARTA -&gt; 217 Gene Sets 
#&gt; - C2.CP.KEGG -&gt; 186 Gene Sets 
#&gt; - C2.CP.REACTOME -&gt; 674 Gene Sets 
#&gt; - C3.MIR -&gt; 221 Gene Sets 
#&gt; - C3.TFT -&gt; 615 Gene Sets 
#&gt; - C4.CGN -&gt; 427 Gene Sets 
#&gt; - C4.CM -&gt; 431 Gene Sets 
#&gt; - C5.BP -&gt; 4436 Gene Sets 
#&gt; - C5.CC -&gt; 580 Gene Sets 
#&gt; - C5.MF -&gt; 901 Gene Sets 
#&gt; - C6 -&gt; 189 Gene Sets 
#&gt; - C7 -&gt; 4872 Gene Sets 
#&gt; - H -&gt; 50 Gene Sets</code></pre>
<div class="sourceCode" id="cb10"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb10-1" title="1">BIOCARTA &lt;-<span class="st"> </span><span class="kw">msigdb_fetch</span>(msigdb_path, <span class="st">"C2.CP.BIOCARTA"</span>)</a>
<a class="sourceLine" id="cb10-2" title="2">KEGG     &lt;-<span class="st"> </span><span class="kw">msigdb_fetch</span>(msigdb_path, <span class="st">"C2.CP.KEGG"</span>)</a>
<a class="sourceLine" id="cb10-3" title="3">REACTOME &lt;-<span class="st"> </span><span class="kw">msigdb_fetch</span>(msigdb_path, <span class="st">"C2.CP.REACTOME"</span>)</a></code></pre></div>
<p>In this example, we are interested in all three of the following genesets, therefore we concatenate them. A geneset is simply a named list of vectors, therefore, one can use any custom geneset in their analysis, as long as it is appropriately defined.</p>
<div class="sourceCode" id="cb11"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb11-1" title="1">gsets &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/c">c</a></span>(BIOCARTA, KEGG, REACTOME)</a></code></pre></div>
</div>
</div>
<div id="defining-custom-genesets" class="section level3">
<h3 class="hasAnchor">
<a href="#defining-custom-genesets" class="anchor"></a>Defining custom genesets</h3>
<p>As mentioned previously, one can use custom genesets with hypeR. In this example, we download one of the many publicly available genesets hosted by <a href="https://amp.pharm.mssm.edu/Enrichr/">Enrichr</a>. Once downloaded, one performs hyper enrichment as normal.</p>
<div class="sourceCode" id="cb12"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb12-1" title="1">url =<span class="st"> "http://amp.pharm.mssm.edu/Enrichr/geneSetLibrary?mode=text&amp;libraryName=Cancer_Cell_Line_Encyclopedia"</span></a>
<a class="sourceLine" id="cb12-2" title="2">r &lt;-<span class="st"> </span>httr<span class="op">::</span><span class="kw"><a href="https://www.rdocumentation.org/packages/httr/topics/GET">GET</a></span>(url)</a>
<a class="sourceLine" id="cb12-3" title="3">text &lt;-<span class="st"> </span>httr<span class="op">::</span><span class="kw"><a href="https://www.rdocumentation.org/packages/httr/topics/content">content</a></span>(r, <span class="st">"text"</span>, <span class="dt">encoding=</span><span class="st">"ISO-8859-1"</span>)</a>
<a class="sourceLine" id="cb12-4" title="4">text.split &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/strsplit">strsplit</a></span>(text, <span class="st">"</span><span class="ch">\n</span><span class="st">"</span>)[[<span class="dv">1</span>]]</a>
<a class="sourceLine" id="cb12-5" title="5">gsets &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/lapply">sapply</a></span>(text.split, <span class="cf">function</span>(x) {</a>
<a class="sourceLine" id="cb12-6" title="6">    genes &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/strsplit">strsplit</a></span>(x, <span class="st">"</span><span class="ch">\t</span><span class="st">"</span>)[[<span class="dv">1</span>]]</a>
<a class="sourceLine" id="cb12-7" title="7">    <span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/function">return</a></span>(genes[<span class="dv">3</span><span class="op">:</span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/length">length</a></span>(genes)])</a>
<a class="sourceLine" id="cb12-8" title="8">})</a>
<a class="sourceLine" id="cb12-9" title="9"><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/names">names</a></span>(gsets) &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/unlist">unlist</a></span>(<span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/lapply">lapply</a></span>(text.split, <span class="cf">function</span>(x) <span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/strsplit">strsplit</a></span>(x, <span class="st">"</span><span class="ch">\t</span><span class="st">"</span>)[[<span class="dv">1</span>]][<span class="dv">1</span>]))</a></code></pre></div>
</div>
<div id="downloading-genesets-from-hyperdb" class="section level3">
<h3 class="hasAnchor">
<a href="#downloading-genesets-from-hyperdb" class="anchor"></a>Downloading genesets from hyperdb</h3>
<p>If msigdb genesets are not sufficient, we have also provided another set of functions for downloading and loading other open source genesets. Essentially, we wrap the method used in the example defining custom genesets. This is facilitated by interfacing with the publicly available <a href="https://amp.pharm.mssm.edu/Enrichr/#stats">libraries</a> hosted by <a href="https://amp.pharm.mssm.edu/Enrichr/">Enrichr</a>.</p>
<div id="downloading-and-loading-genesets-1" class="section level4">
<h4 class="hasAnchor">
<a href="#downloading-and-loading-genesets-1" class="anchor"></a>Downloading and loading genesets</h4>
<div class="sourceCode" id="cb13"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb13-1" title="1">gsets &lt;-<span class="st"> </span><span class="kw">hyperdb_fetch</span>(<span class="st">"Cancer_Cell_Line_Encyclopedia"</span>)</a></code></pre></div>
</div>
<div id="available-genesets-1" class="section level4">
<h4 class="hasAnchor">
<a href="#available-genesets-1" class="anchor"></a>Available genesets</h4>
<div class="sourceCode" id="cb14"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb14-1" title="1"><span class="kw">hyperdb_info</span>()</a></code></pre></div>
<div id="htmlwidget-43ef7b18bde305d6ba31" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-43ef7b18bde305d6ba31">{"x":{"style":"bootstrap","filter":"none","fillContainer":true,"data":[["Genes_Associated_with_NIH_Grants","Cancer_Cell_Line_Encyclopedia","Achilles_fitness_decrease","Achilles_fitness_increase","Aging_Perturbations_from_GEO_down","Aging_Perturbations_from_GEO_up","Allen_Brain_Atlas_down","Allen_Brain_Atlas_up","ARCHS4_Cell-lines","ARCHS4_IDG_Coexp","ARCHS4_Kinases_Coexp","ARCHS4_TFs_Coexp","ARCHS4_Tissues","BioCarta_2013","BioCarta_2015","BioCarta_2016","BioPlex_2017","ChEA_2013","ChEA_2015","ChEA_2016","Chromosome_Location","Chromosome_Location_hg19","CORUM","Data_Acquisition_Method_Most_Popular_Genes","dbGaP","Disease_Perturbations_from_GEO_down","Disease_Perturbations_from_GEO_up","Disease_Signatures_from_GEO_down_2014","Disease_Signatures_from_GEO_up_2014","Drug_Perturbations_from_GEO_2014","Drug_Perturbations_from_GEO_down","Drug_Perturbations_from_GEO_up","DrugMatrix","DSigDB","ENCODE_and_ChEA_Consensus_TFs_from_ChIP-X","ENCODE_Histone_Modifications_2013","ENCODE_Histone_Modifications_2015","ENCODE_TF_ChIP-seq_2014","ENCODE_TF_ChIP-seq_2015","Enrichr_Libraries_Most_Popular_Genes","Enrichr_Submissions_TF-Gene_Coocurrence","Epigenomics_Roadmap_HM_ChIP-seq","ESCAPE","GeneSigDB","Genome_Browser_PWMs","GO_Biological_Process_2013","GO_Biological_Process_2015","GO_Biological_Process_2017","GO_Biological_Process_2017b","GO_Biological_Process_2018","GO_Cellular_Component_2013","GO_Cellular_Component_2015","GO_Cellular_Component_2017","GO_Cellular_Component_2017b","GO_Cellular_Component_2018","GO_Molecular_Function_2013","GO_Molecular_Function_2015","GO_Molecular_Function_2017","GO_Molecular_Function_2017b","GO_Molecular_Function_2018","GTEx_Tissue_Sample_Gene_Expression_Profiles_down","GTEx_Tissue_Sample_Gene_Expression_Profiles_up","GWAS_Catalog_2019","HMDB_Metabolites","HomoloGene","Human_Gene_Atlas","Human_Phenotype_Ontology","HumanCyc_2015","HumanCyc_2016","huMAP","Jensen_COMPARTMENTS","Jensen_DISEASES","Jensen_TISSUES","KEA_2013","KEA_2015","KEGG_2013","KEGG_2015","KEGG_2016","Kinase_Perturbations_from_GEO_down","Kinase_Perturbations_from_GEO_up","Ligand_Perturbations_from_GEO_down","Ligand_Perturbations_from_GEO_up","LINCS_L1000_Chem_Pert_down","LINCS_L1000_Chem_Pert_up","LINCS_L1000_Kinase_Perturbations_down","LINCS_L1000_Kinase_Perturbations_up","LINCS_L1000_Ligand_Perturbations_down","LINCS_L1000_Ligand_Perturbations_up","MCF7_Perturbations_from_GEO_down","MCF7_Perturbations_from_GEO_up","MGI_Mammalian_Phenotype_2013","MGI_Mammalian_Phenotype_2017","MGI_Mammalian_Phenotype_Level_3","MGI_Mammalian_Phenotype_Level_4","Microbe_Perturbations_from_GEO_down","Microbe_Perturbations_from_GEO_up","miRTarBase_2017","Mouse_Gene_Atlas","MSigDB_Computational","MSigDB_Oncogenic_Signatures","NCI-60_Cancer_Cell_Lines","NCI-Nature_2015","NCI-Nature_2016","NIH_Funded_PIs_2017_AutoRIF_ARCHS4_Predictions","NIH_Funded_PIs_2017_GeneRIF_ARCHS4_Predictions","NIH_Funded_PIs_2017_Human_AutoRIF","NIH_Funded_PIs_2017_Human_GeneRIF","NURSA_Human_Endogenous_Complexome","Old_CMAP_down","Old_CMAP_up","OMIM_Disease","OMIM_Expanded","Panther_2015","Panther_2016","Pfam_InterPro_Domains","Phosphatase_Substrates_from_DEPOD","PPI_Hub_Proteins","Rare_Diseases_AutoRIF_ARCHS4_Predictions","Rare_Diseases_AutoRIF_Gene_Lists","Rare_Diseases_GeneRIF_ARCHS4_Predictions","Rare_Diseases_GeneRIF_Gene_Lists","Reactome_2013","Reactome_2015","Reactome_2016","RNA-Seq_Disease_Gene_and_Drug_Signatures_from_GEO","SILAC_Phosphoproteomics","Single_Gene_Perturbations_from_GEO_down","Single_Gene_Perturbations_from_GEO_up","SubCell_BarCode","SysMyo_Muscle_Gene_Sets","TargetScan_microRNA","TargetScan_microRNA_2017","TF-LOF_Expression_from_GEO","TF_Perturbations_Followed_by_Expression","Tissue_Protein_Expression_from_Human_Proteome_Map","Tissue_Protein_Expression_from_ProteomicsDB","Transcription_Factor_PPIs","TRANSFAC_and_JASPAR_PWMs","TRRUST_Transcription_Factors_2019","Virus_Perturbations_from_GEO_down","Virus_Perturbations_from_GEO_up","VirusMINT","WikiPathways_2013","WikiPathways_2015","WikiPathways_2016","WikiPathways_2019_Human","WikiPathways_2019_Mouse"],[32876,967,216,216,286,286,2192,2192,125,352,498,1724,108,249,239,237,3915,353,395,645,386,36,1658,12,345,839,839,142,142,701,906,906,7876,4026,104,109,412,498,816,121,1722,383,315,2139,615,941,5192,3166,10125,5103,205,641,636,816,446,402,1136,972,3271,1151,2918,2918,1737,3906,12,84,1779,125,152,995,2283,1811,1842,474,428,200,179,293,285,285,261,261,33132,33132,3644,3644,96,96,401,401,476,5231,71,476,312,312,3240,96,858,189,93,209,209,12558,5684,12558,5687,1796,6100,6100,90,187,104,112,311,59,385,3725,3725,2244,2244,78,1389,1530,1302,84,2460,2460,104,1135,222,683,269,1958,30,207,290,326,571,323,323,85,199,404,437,472,176],[15886,15797,4271,4320,16129,15309,13877,13121,23601,20883,19612,25983,21809,1295,1678,1348,10271,47172,48230,49238,32740,27360,2741,1073,5613,23939,23561,15406,15057,47107,23877,24350,5209,19513,15562,15852,29065,21493,26382,5902,12486,22288,25651,23726,13362,7682,14264,13822,13247,14433,7324,13236,10427,8002,8655,8469,12753,10601,10089,11459,16725,19249,19378,3723,19129,13373,3096,756,934,2243,18329,15755,19586,4533,3102,4128,3800,7010,17850,17660,15090,15103,9448,9559,12668,12638,3788,3357,15022,15676,10496,8184,10406,10493,15854,15015,14893,19270,10061,11250,12232,2541,2541,16964,17258,13464,13072,10231,8695,11251,1759,2178,1918,2041,7588,280,16399,13787,10471,13929,10352,3185,6768,8973,22440,5655,30832,31132,12419,19500,7504,17598,34061,19741,6454,13572,6002,27884,3264,17576,17711,851,2854,5863,5966,6201,4558],[9,176,128,129,292,308,304,305,2395,299,299,299,2316,18,21,19,22,1370,1429,1550,85,802,5,100,36,293,307,300,300,509,302,299,300,117,887,912,2123,3713,1811,109,299,4368,807,127,275,78,58,21,49,36,172,82,38,143,61,122,57,25,45,39,1443,1443,37,47,1594,258,31,12,13,19,161,28,545,37,25,48,48,87,300,300,282,318,63,73,300,300,159,153,290,310,201,24,715,200,279,321,128,388,106,166,343,39,39,200,200,45,26,158,100,100,25,89,39,42,35,9,247,200,76,200,58,73,47,64,505,342,302,298,491,137,155,1208,641,270,301,301,77,1284,22,300,300,15,34,51,51,45,54]],"container":"<table class=\"table table-bordered table-condensed fill-container\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Terms<\/th>\n      <th>Gene_Coverage<\/th>\n      <th>Genes_Per_Term<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"columnDefs":[{"className":"dt-right","targets":[1,2,3]},{"orderable":false,"targets":0}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="sidebar">
      </div>

</div>


      <footer><div class="copyright">
  <p>Developed by <a href="https://github.com/anfederico">Anthony Federico</a>.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.3.0.</p>
</div>
      </footer>
</div>

  

  </body>
</html>
