<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>hypeR â€¢ hypeR</title>
<!-- favicons --><link rel="icon" type="image/png" sizes="16x16" href="../favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="../favicon-32x32.png">
<link rel="apple-touch-icon" type="image/png" sizes="180x180" href="../apple-touch-icon.png">
<link rel="apple-touch-icon" type="image/png" sizes="120x120" href="../apple-touch-icon-120x120.png">
<link rel="apple-touch-icon" type="image/png" sizes="76x76" href="../apple-touch-icon-76x76.png">
<link rel="apple-touch-icon" type="image/png" sizes="60x60" href="../apple-touch-icon-60x60.png">
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script><!-- Bootstrap --><link href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.3.7/cosmo/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha256-U5ZEeKfGNOja007MMD3YBI0A3OSZOQbeG6z2f2Y0hu8=" crossorigin="anonymous"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" integrity="sha256-eZrrJcwDc/3uDhsdt61sL2oOBY362qM3lon1gyExkL0=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.4/clipboard.min.js" integrity="sha256-FiZwavyI2V6+EXO1U+xzLG3IKldpiTFf3153ea9zikQ=" crossorigin="anonymous"></script><!-- sticky kit --><script src="https://cdnjs.cloudflare.com/ajax/libs/sticky-kit/1.1.3/sticky-kit.min.js" integrity="sha256-c4Rlo1ZozqTPE2RLuvbusY3+SU1pQaJC0TjuhygMipw=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="hypeR">
<meta property="og:description" content="">
<meta name="twitter:card" content="summary">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body>
    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">hypeR</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">1.10.1</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../index.html">Home</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Tutorials
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/installation.html">Installation</a>
    </li>
    <li>
      <a href="../articles/basics.html">Basic Usage</a>
    </li>
    <li>
      <a href="../articles/data.html">Downloading Data</a>
    </li>
    <li>
      <a href="../articles/visualize.html">Visualize Results</a>
    </li>
    <li>
      <a href="../articles/relational.html">Relational Genesets</a>
    </li>
    <li>
      <a href="../articles/multihyp.html">Multiple Signatures</a>
    </li>
    <li>
      <a href="../articles/reporting.html">Generating Reports</a>
    </li>
  </ul>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/montilab/hypeR">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      
      </header><script src="hypeR_files/htmlwidgets-1.3/htmlwidgets.js"></script><script src="hypeR_files/plotly-binding-4.9.0/plotly.js"></script><script src="hypeR_files/typedarray-0.1/typedarray.min.js"></script><script src="hypeR_files/jquery-1.11.3/jquery.min.js"></script><link href="hypeR_files/crosstalk-1.0.0/css/crosstalk.css" rel="stylesheet">
<script src="hypeR_files/crosstalk-1.0.0/js/crosstalk.min.js"></script><link href="hypeR_files/plotly-htmlwidgets-css-1.46.1/plotly-htmlwidgets.css" rel="stylesheet">
<script src="hypeR_files/plotly-main-1.46.1/plotly-latest.min.js"></script><link href="hypeR_files/vis-4.20.1/vis.css" rel="stylesheet">
<script src="hypeR_files/vis-4.20.1/vis.min.js"></script><script src="hypeR_files/visNetwork-binding-2.0.6/visNetwork.js"></script><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1>hypeR</h1>
                        <h4 class="author">Anthony Federico</h4>
            <address class="author_afil">
      Boston University School of Medicine, Boston, MA<br><h4 class="author">Stefano Monti</h4>
            <address class="author_afil">
      Boston University School of Medicine, Boston, MA<br><h4 class="date">May 2, 2019</h4>
      
      <small class="dont-index">Source: <a href="https://github.com/montilab/hypeR/blob/master/vignettes/hypeR.Rmd"><code>vignettes/hypeR.Rmd</code></a></small>
      <div class="hidden name"><code>hypeR.Rmd</code></div>

    </address>
</address>
</div>

    
    
<div id="introduction" class="section level1">
<h1 class="hasAnchor">
<a href="#introduction" class="anchor"></a>Introduction</h1>
<p>Geneset enrichment is an important step in many biological data analysis workflows, particularly in bioinformatics and computational biology. At a basic level, one is testing if a group of genes is overrepresented in a series of pre-defined sets of genes, which typically signify some biological relevance. The R package hypeR enables users to easily perform this type of analysis via a hypergeometric test with a focus on easing downstream visualization, analysis, and reporting. Additionally, users can use custom genesets that are easily defined, extending the analysis of genes to other areas of interest such as proteins, microbes, metabolites etc. The hypeR package is designed to make routine geneset enrichment seamless for scientist working in R.</p>
</div>
<div id="installation" class="section level1">
<h1 class="hasAnchor">
<a href="#installation" class="anchor"></a>Installation</h1>
<p>Download the package from Bioconductor.</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb1-1" title="1"><span class="cf">if</span> (<span class="op">!</span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/ns-load">requireNamespace</a></span>(<span class="st">"BiocManager"</span>, <span class="dt">quietly =</span> <span class="ot">TRUE</span>))</a>
<a class="sourceLine" id="cb1-2" title="2"><span class="kw"><a href="https://www.rdocumentation.org/packages/utils/topics/install.packages">install.packages</a></span>(<span class="st">"BiocManager"</span>)</a>
<a class="sourceLine" id="cb1-3" title="3">BiocManager<span class="op">::</span><span class="kw"><a href="https://www.rdocumentation.org/packages/BiocManager/topics/install">install</a></span>(<span class="st">"hypeR"</span>)</a></code></pre></div>
<p>Or install the development version of the package from Bioconductor.</p>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb2-1" title="1">BiocManager<span class="op">::</span><span class="kw"><a href="https://www.rdocumentation.org/packages/BiocManager/topics/install">install</a></span>(<span class="st">"montilab/hypeR"</span>)</a></code></pre></div>
<p>If not using the current version of R, try installing from our dev branch.</p>
<div class="sourceCode" id="cb3"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb3-1" title="1">devtools<span class="op">::</span><span class="kw"><a href="https://www.rdocumentation.org/packages/devtools/topics/reexports">install_github</a></span>(<span class="st">"montilab/hypeR"</span>, <span class="dt">ref=</span><span class="st">"dev"</span>)</a></code></pre></div>
<p>Load the package into an R session.</p>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb4-1" title="1"><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/library">library</a></span>(hypeR)</a></code></pre></div>
</div>
<div id="basics" class="section level1">
<h1 class="hasAnchor">
<a href="#basics" class="anchor"></a>Basics</h1>
<div id="example-data" class="section level2">
<h2 class="hasAnchor">
<a href="#example-data" class="anchor"></a>Example Data</h2>
<div id="example-of-a-signature" class="section level3">
<h3 class="hasAnchor">
<a href="#example-of-a-signature" class="anchor"></a>Example of a signature</h3>
<p>Here we define our signature of interest as a group of genes randomly sampled from the <em>estrogen-dependent gene expression</em> gset. A signature is simply a vector of symbols.</p>
<div class="sourceCode" id="cb5"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb5-1" title="1">signatures &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/readRDS">readRDS</a></span>(<span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/system.file">system.file</a></span>(<span class="st">"extdata/signatures.rds"</span>, <span class="dt">package=</span><span class="st">"hypeR"</span>))</a>
<a class="sourceLine" id="cb5-2" title="2">signature &lt;-<span class="st"> </span>signatures<span class="op">$</span>signature</a></code></pre></div>
<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb6-1" title="1"><span class="kw"><a href="https://www.rdocumentation.org/packages/utils/topics/head">head</a></span>(signature)</a></code></pre></div>
<pre><code>## [1] "GTF2F1"    "HIST1H2AJ" "KANK1"     "STAG1"     "FKBP4"     "SMC1A"</code></pre>
</div>
<div id="example-of-a-geneset" class="section level3">
<h3 class="hasAnchor">
<a href="#example-of-a-geneset" class="anchor"></a>Example of a geneset</h3>
<p>Here we use a genesets example from <a href="https://reactome.org/">REACTOME</a>. A geneset is simply a list of vectors, therefore, one can use any custom geneset in their analysis, as long as it is appropriately defined.</p>
<div class="sourceCode" id="cb8"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb8-1" title="1">gsets &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/readRDS">readRDS</a></span>(<span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/system.file">system.file</a></span>(<span class="st">"extdata/gsets.rds"</span>, <span class="dt">package=</span><span class="st">"hypeR"</span>))</a>
<a class="sourceLine" id="cb8-2" title="2">gsets &lt;-<span class="st"> </span>gsets<span class="op">$</span>REACTOME</a></code></pre></div>
<div class="sourceCode" id="cb9"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb9-1" title="1"><span class="kw"><a href="https://www.rdocumentation.org/packages/utils/topics/head">head</a></span>(<span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/names">names</a></span>(gsets))</a></code></pre></div>
<pre><code>## [1] "REACTOME_3_UTR_MEDIATED_TRANSLATIONAL_REGULATION"                        
## [2] "REACTOME_A_TETRASACCHARIDE_LINKER_SEQUENCE_IS_REQUIRED_FOR_GAG_SYNTHESIS"
## [3] "REACTOME_ABACAVIR_TRANSPORT_AND_METABOLISM"                              
## [4] "REACTOME_ABC_FAMILY_PROTEINS_MEDIATED_TRANSPORT"                         
## [5] "REACTOME_ABCA_TRANSPORTERS_IN_LIPID_HOMEOSTASIS"                         
## [6] "REACTOME_ABORTIVE_ELONGATION_OF_HIV1_TRANSCRIPT_IN_THE_ABSENCE_OF_TAT"</code></pre>
<div class="sourceCode" id="cb11"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb11-1" title="1"><span class="kw"><a href="https://www.rdocumentation.org/packages/utils/topics/head">head</a></span>(gsets[[<span class="dv">1</span>]])</a></code></pre></div>
<pre><code>## [1] "EIF1AX" "EIF2S1" "EIF2S2" "EIF2S3" "EIF3A"  "EIF3B"</code></pre>
</div>
</div>
<div id="hyper-enrichment" class="section level2">
<h2 class="hasAnchor">
<a href="#hyper-enrichment" class="anchor"></a>Hyper enrichment</h2>
<p>All workflows begin with performing hyper enrichment with <strong><em>hypeR</em></strong>. Often we are just interested in a single signature, as described above. In this case, <strong><em>hypeR</em></strong> will return a <strong><em>hyp</em></strong> object. This object contains relevant information to the enrichment results and is recognized by downstream methods.</p>
<div class="sourceCode" id="cb13"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb13-1" title="1">hyp_obj &lt;-<span class="st"> </span><span class="kw"><a href="../reference/hypeR.html">hypeR</a></span>(signature, gsets, <span class="dt">fdr=</span><span class="fl">0.05</span>)</a></code></pre></div>
<div class="sourceCode" id="cb14"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb14-1" title="1"><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/print">print</a></span>(hyp_obj)</a></code></pre></div>
<pre><code>## hyp object: 
##   data: 73 x 8
##   args: symbols
##         gsets
##         gsets_relational
##         bg
##         min_drawsize
##         pval_cutoff
##         fdr_cutoff
##         verbose</code></pre>
<div class="sourceCode" id="cb16"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb16-1" title="1">hyp_df &lt;-<span class="st"> </span>hyp_obj<span class="op">$</span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/as.data.frame">as.data.frame</a></span>()</a>
<a class="sourceLine" id="cb16-2" title="2"><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/print">print</a></span>(hyp_df[<span class="dv">5</span><span class="op">:</span><span class="dv">10</span>,<span class="dv">1</span><span class="op">:</span><span class="dv">4</span>])</a></code></pre></div>
<pre><code>##                                        pval     fdr set.annotated set.size
## REACTOME_RNA_POL_I_TRANSCRIPTION    1.9e-33 2.6e-31            22      100
## REACTOME_RNA_POL_I_PROMOTER_OPENING 4.1e-33 4.6e-31            20      100
## REACTOME_CHROMOSOME_MAINTENANCE     8.8e-32 8.5e-30            23      100
## REACTOME_PACKAGING_OF_TELOMERE_ENDS 2.8e-31 2.4e-29            18      100
## REACTOME_AMYLOIDS                   3.4e-30 2.6e-28            20      100
## REACTOME_MEIOTIC_RECOMBINATION      7.6e-30 5.1e-28            20      100</code></pre>
<div id="show-an-interactive-table" class="section level3">
<h3 class="hasAnchor">
<a href="#show-an-interactive-table" class="anchor"></a>Show an interactive table</h3>
<div class="sourceCode" id="cb18"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb18-1" title="1"><span class="kw"><a href="../reference/hyp_show.html">hyp_show</a></span>(hyp_obj, <span class="dt">simple=</span><span class="ot">FALSE</span>, <span class="dt">stylish=</span><span class="ot">TRUE</span>)</a></code></pre></div>
</div>
<div id="set-background-population-size" class="section level3">
<h3 class="hasAnchor">
<a href="#set-background-population-size" class="anchor"></a>Set background population size</h3>
<p>Because genomic experiments often vary in the number of genes probed, it is important to be able to adjust the background population size accordingly to ensure accurate enrichment statistics.</p>
<div class="sourceCode" id="cb19"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb19-1" title="1">hyp_obj &lt;-<span class="st"> </span><span class="kw"><a href="../reference/hypeR.html">hypeR</a></span>(signature, gsets, <span class="dt">bg=</span><span class="dv">10000</span>, <span class="dt">fdr=</span><span class="fl">0.05</span>)</a></code></pre></div>
</div>
<div id="save-results-to-excel" class="section level3">
<h3 class="hasAnchor">
<a href="#save-results-to-excel" class="anchor"></a>Save results to excel</h3>
<div class="sourceCode" id="cb20"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb20-1" title="1"><span class="kw"><a href="../reference/hyp_to_excel.html">hyp_to_excel</a></span>(hyp_obj, <span class="dt">file_path=</span><span class="st">"pathways.xlsx"</span>)</a></code></pre></div>
</div>
<div id="save-results-to-table" class="section level3">
<h3 class="hasAnchor">
<a href="#save-results-to-table" class="anchor"></a>Save results to table</h3>
<div class="sourceCode" id="cb21"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb21-1" title="1"><span class="kw"><a href="../reference/hyp_to_table.html">hyp_to_table</a></span>(hyp_obj, <span class="dt">file_path=</span><span class="st">"pathways.txt"</span>)</a></code></pre></div>
</div>
</div>
<div id="visualizations" class="section level2">
<h2 class="hasAnchor">
<a href="#visualizations" class="anchor"></a>Visualizations</h2>
<div id="plot-results" class="section level3">
<h3 class="hasAnchor">
<a href="#plot-results" class="anchor"></a>Plot results</h3>
<p>One can visualize the top enriched pathways using <strong><em>hyp_plot</em></strong> which will return a horizontal bar plot. Each bar is a geneset, where dark blue represents the proportion of signature genes found in the entire geneset.</p>
<div class="sourceCode" id="cb22"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb22-1" title="1"><span class="kw"><a href="../reference/hyp_plot.html">hyp_plot</a></span>(hyp_obj, <span class="dt">return_plots=</span><span class="ot">TRUE</span>)</a></code></pre></div>
<div id="htmlwidget-8f72e62f72a442908960" style="width:648px;height:432.632880098888px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-8f72e62f72a442908960">{"x":{"visdat":{"8bd370c0620c":["function () ","plotlyVisDat"]},"cur_data":"8bd370c0620c","attrs":{"8bd370c0620c":{"x":{},"y":{},"orientation":"h","hoverinfo":"y","marker":{"color":"#4CA1AF","line":{"color":"white","width":1}},"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"type":"bar"},"8bd370c0620c.1":{"x":{},"y":{},"orientation":"h","hoverinfo":"y","marker":{"color":"#C4E0E5","line":{"color":"white","width":1}},"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"type":"bar","inherit":true}},"layout":{"margin":{"b":40,"l":300,"t":30,"r":0},"title":"","xaxis":{"domain":[0.16,1],"automargin":true,"title":"-log<sub>10<\/sub>(FDR)","tickvals":[1.30102999566398,2,3,5,10,15,20,25,30,35,40,45,50,55,60,65,70,75,80,85,90,95,100,105,110,115,120,125,130,135,140,145,150,155,160,165,170,175,180,185,190,195,200,205,210,215,220,225,230,235,240,245,250,255,260,265,270,275,280,285,290,295,300,305,310,315,320,325,330,335,340,345,350,355,360,365,370,375,380,385,390,395,400,405,410,415,420,425,430,435,440,445,450,455,460,465,470,475,480,485,490,495,500,505,510,515,520,525,530,535,540,545,550,555,560,565,570,575,580,585,590,595,600,605,610,615,620,625,630,635,640,645,650,655,660,665,670,675,680,685,690,695,700,705,710,715,720,725,730,735,740,745,750,755,760,765,770,775,780,785,790,795,800,805,810,815,820,825,830,835,840,845,850,855,860,865,870,875,880,885,890,895,900,905,910,915,920,925,930,935,940,945,950,955,960,965,970,975,980,985,990,995,1000],"ticktext":["-log(0.05)","-log(0.01)","-log(0.001)"],"tickfont":{"size":9},"showgrid":true,"showline":true,"showticklabels":true,"zeroline":true},"yaxis":{"domain":[0,1],"automargin":true,"title":"","categoryarray":["REACTOME_MEIOTIC_RECOMBINATION","REACTOME_AMYLOIDS","REACTOME_CHROMOSOME_MAINTENANCE","REACTOME_PACKAGING_OF_TELOMERE_ENDS","REACTOME_RNA_POL_I_TRANSCRIPTION","REACTOME_RNA_POL_I_PROMOTER_OPENING","REACTOME_RNA_POL_I_RNA_POL_III_AND_MITOCHONDRIAL_TRANSCRIPTION","REACTOME_MEIOSIS","REACTOME_MEIOTIC_SYNAPSIS","REACTOME_TRANSCRIPTION"],"categoryorder":"array","showgrid":true,"showline":true,"showticklabels":false,"zeroline":true,"type":"category"},"barmode":"stack","paper_bgcolor":"white","plot_bgcolor":"white","showlegend":false,"annotations":[{"text":"REACTOME_TRANSCRIPTION","xref":"paper","yref":"y","x":0.15,"y":"REACTOME_TRANSCRIPTION","xanchor":"right","categoryorder":"array","font":{"family":"Arial","size":10,"color":"black"},"showarrow":false,"align":"right"},{"text":"REACTOME_MEIOTIC_SYNAPSIS","xref":"paper","yref":"y","x":0.15,"y":"REACTOME_MEIOTIC_SYNAPSIS","xanchor":"right","categoryorder":"array","font":{"family":"Arial","size":10,"color":"black"},"showarrow":false,"align":"right"},{"text":"REACTOME_MEIOSIS","xref":"paper","yref":"y","x":0.15,"y":"REACTOME_MEIOSIS","xanchor":"right","categoryorder":"array","font":{"family":"Arial","size":10,"color":"black"},"showarrow":false,"align":"right"},{"text":"REACTOME_RNA_POL_I_RNA_POL_III_AND_MITOCHONDRIAL_TRANSCRIPTION","xref":"paper","yref":"y","x":0.15,"y":"REACTOME_RNA_POL_I_RNA_POL_III_AND_MITOCHONDRIAL_TRANSCRIPTION","xanchor":"right","categoryorder":"array","font":{"family":"Arial","size":10,"color":"black"},"showarrow":false,"align":"right"},{"text":"REACTOME_RNA_POL_I_PROMOTER_OPENING","xref":"paper","yref":"y","x":0.15,"y":"REACTOME_RNA_POL_I_PROMOTER_OPENING","xanchor":"right","categoryorder":"array","font":{"family":"Arial","size":10,"color":"black"},"showarrow":false,"align":"right"},{"text":"REACTOME_RNA_POL_I_TRANSCRIPTION","xref":"paper","yref":"y","x":0.15,"y":"REACTOME_RNA_POL_I_TRANSCRIPTION","xanchor":"right","categoryorder":"array","font":{"family":"Arial","size":10,"color":"black"},"showarrow":false,"align":"right"},{"text":"REACTOME_PACKAGING_OF_TELOMERE_ENDS","xref":"paper","yref":"y","x":0.15,"y":"REACTOME_PACKAGING_OF_TELOMERE_ENDS","xanchor":"right","categoryorder":"array","font":{"family":"Arial","size":10,"color":"black"},"showarrow":false,"align":"right"},{"text":"REACTOME_CHROMOSOME_MAINTENANCE","xref":"paper","yref":"y","x":0.15,"y":"REACTOME_CHROMOSOME_MAINTENANCE","xanchor":"right","categoryorder":"array","font":{"family":"Arial","size":10,"color":"black"},"showarrow":false,"align":"right"},{"text":"REACTOME_AMYLOIDS","xref":"paper","yref":"y","x":0.15,"y":"REACTOME_AMYLOIDS","xanchor":"right","categoryorder":"array","font":{"family":"Arial","size":10,"color":"black"},"showarrow":false,"align":"right"},{"text":"REACTOME_MEIOTIC_RECOMBINATION","xref":"paper","yref":"y","x":0.15,"y":"REACTOME_MEIOTIC_RECOMBINATION","xanchor":"right","categoryorder":"array","font":{"family":"Arial","size":10,"color":"black"},"showarrow":false,"align":"right"}],"hovermode":"closest"},"source":"A","config":{"showSendToCloud":false},"data":[{"x":[5.92348659074605,8.21083639985989,5.2018700732499,4.85200424288239,7.39381250127496,5.59597738884958,8.23447774306567,3.85730032579769,4.8918144567865,4.6521778617215],"y":["REACTOME_TRANSCRIPTION","REACTOME_MEIOTIC_SYNAPSIS","REACTOME_MEIOSIS","REACTOME_RNA_POL_I_RNA_POL_III_AND_MITOCHONDRIAL_TRANSCRIPTION","REACTOME_RNA_POL_I_PROMOTER_OPENING","REACTOME_RNA_POL_I_TRANSCRIPTION","REACTOME_PACKAGING_OF_TELOMERE_ENDS","REACTOME_CHROMOSOME_MAINTENANCE","REACTOME_AMYLOIDS","REACTOME_MEIOTIC_RECOMBINATION"],"orientation":"h","hoverinfo":["y","y","y","y","y","y","y","y","y","y"],"marker":{"color":"#4CA1AF","line":{"color":"white","width":1}},"type":"bar","error_y":{"color":"rgba(31,119,180,1)"},"error_x":{"color":"rgba(31,119,180,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[27.6963021675423,17.8496443475215,18.9348070666296,18.8257764623837,15.5270062526774,17.0422947751328,13.7241295717761,16.9385796915464,15.4092155388775,15.3521869436809],"y":["REACTOME_TRANSCRIPTION","REACTOME_MEIOTIC_SYNAPSIS","REACTOME_MEIOSIS","REACTOME_RNA_POL_I_RNA_POL_III_AND_MITOCHONDRIAL_TRANSCRIPTION","REACTOME_RNA_POL_I_PROMOTER_OPENING","REACTOME_RNA_POL_I_TRANSCRIPTION","REACTOME_PACKAGING_OF_TELOMERE_ENDS","REACTOME_CHROMOSOME_MAINTENANCE","REACTOME_AMYLOIDS","REACTOME_MEIOTIC_RECOMBINATION"],"orientation":"h","hoverinfo":["y","y","y","y","y","y","y","y","y","y"],"marker":{"color":"#C4E0E5","line":{"color":"white","width":1}},"type":"bar","error_y":{"color":"rgba(255,127,14,1)"},"error_x":{"color":"rgba(255,127,14,1)"},"xaxis":"x","yaxis":"y","frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="enrichment-map" class="section level3">
<h3 class="hasAnchor">
<a href="#enrichment-map" class="anchor"></a>Enrichment map</h3>
<p>One can visualize the top enriched pathways using <strong><em>hyp_emap</em></strong> which will return an enrichment map. Each node represents a geneset, where the shade of red indicates the normalized significance of enrichment. Hover over the node to view the raw value. Edges represent geneset similarity, calculated by either jaccard or overlap similarity metrics.</p>
<div class="sourceCode" id="cb23"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb23-1" title="1"><span class="kw"><a href="../reference/hyp_emap.html">hyp_emap</a></span>(hyp_obj, <span class="dt">similarity_cutoff=</span><span class="fl">0.5</span>, <span class="dt">return_plots=</span><span class="ot">TRUE</span>)</a></code></pre></div>
<div id="htmlwidget-5b35acb7700d6c17fceb" style="width:648px;height:432.632880098888px;" class="visNetwork html-widget"></div>
<script type="application/json" data-for="htmlwidget-5b35acb7700d6c17fceb">{"x":{"nodes":{"id":["REACTOME_TRANSCRIPTION","REACTOME_MEIOTIC_SYNAPSIS","REACTOME_MEIOSIS","REACTOME_RNA_POL_I_RNA_POL_III_AND_MITOCHONDRIAL_TRANSCRIPTION","REACTOME_RNA_POL_I_PROMOTER_OPENING","REACTOME_RNA_POL_I_TRANSCRIPTION","REACTOME_PACKAGING_OF_TELOMERE_ENDS","REACTOME_CHROMOSOME_MAINTENANCE","REACTOME_AMYLOIDS","REACTOME_MEIOTIC_RECOMBINATION","REACTOME_DEPOSITION_OF_NEW_CENPA_CONTAINING_NUCLEOSOMES_AT_THE_CENTROMERE","REACTOME_TELOMERE_MAINTENANCE","REACTOME_VIRAL_MESSENGER_RNA_SYNTHESIS","REACTOME_RNA_POL_II_PRE_TRANSCRIPTION_EVENTS","REACTOME_RNA_POL_II_TRANSCRIPTION_PRE_INITIATION_AND_PROMOTER_OPENING","REACTOME_ELONGATION_ARREST_AND_RECOVERY","REACTOME_REGULATORY_RNA_PATHWAYS","REACTOME_ABORTIVE_ELONGATION_OF_HIV1_TRANSCRIPT_IN_THE_ABSENCE_OF_TAT","REACTOME_FORMATION_OF_RNA_POL_II_ELONGATION_COMPLEX_","REACTOME_MRNA_CAPPING","REACTOME_LATE_PHASE_OF_HIV_LIFE_CYCLE","REACTOME_RNA_POL_II_TRANSCRIPTION","REACTOME_FORMATION_OF_THE_HIV1_EARLY_ELONGATION_COMPLEX","REACTOME_CELL_CYCLE","REACTOME_HIV_LIFE_CYCLE","REACTOME_MRNA_SPLICING_MINOR_PATHWAY","REACTOME_MICRORNA_MIRNA_BIOGENESIS","REACTOME_FORMATION_OF_TRANSCRIPTION_COUPLED_NER_TC_NER_REPAIR_COMPLEX","REACTOME_HIV_INFECTION","REACTOME_TRANSCRIPTION_COUPLED_NER_TC_NER","REACTOME_MRNA_SPLICING","REACTOME_NUCLEOTIDE_EXCISION_REPAIR","REACTOME_PRE_NOTCH_TRANSCRIPTION_AND_TRANSLATION","REACTOME_PROCESSING_OF_CAPPED_INTRON_CONTAINING_PRE_MRNA","REACTOME_MRNA_PROCESSING","REACTOME_SIGNALING_BY_NOTCH","REACTOME_DNA_REPAIR","REACTOME_PRE_NOTCH_EXPRESSION_AND_PROCESSING","REACTOME_INFLUENZA_VIRAL_RNA_TRANSCRIPTION_AND_REPLICATION","REACTOME_CIRCADIAN_REPRESSION_OF_EXPRESSION_BY_REV_ERBA","REACTOME_RORA_ACTIVATES_CIRCADIAN_EXPRESSION","REACTOME_RNA_POL_III_TRANSCRIPTION_INITIATION_FROM_TYPE_3_PROMOTER","REACTOME_INFLUENZA_LIFE_CYCLE","REACTOME_SMAD2_SMAD3_SMAD4_HETEROTRIMER_REGULATES_TRANSCRIPTION","REACTOME_CIRCADIAN_CLOCK","REACTOME_RNA_POL_III_TRANSCRIPTION","REACTOME_BMAL1_CLOCK_NPAS2_ACTIVATES_CIRCADIAN_EXPRESSION","REACTOME_TRANSCRIPTIONAL_ACTIVITY_OF_SMAD2_SMAD3_SMAD4_HETEROTRIMER","REACTOME_FACTORS_INVOLVED_IN_MEGAKARYOCYTE_DEVELOPMENT_AND_PLATELET_PRODUCTION","REACTOME_RNA_POL_III_TRANSCRIPTION_INITIATION_FROM_TYPE_2_PROMOTER","REACTOME_YAP1_AND_WWTR1_TAZ_STIMULATED_GENE_EXPRESSION","REACTOME_NOTCH1_INTRACELLULAR_DOMAIN_REGULATES_TRANSCRIPTION","REACTOME_SIGNALING_BY_TGF_BETA_RECEPTOR_COMPLEX","REACTOME_RNA_POL_III_CHAIN_ELONGATION","REACTOME_SIGNALING_BY_NOTCH1","REACTOME_TRANSCRIPTIONAL_REGULATION_OF_WHITE_ADIPOCYTE_DIFFERENTIATION","REACTOME_GENERIC_TRANSCRIPTION_PATHWAY","REACTOME_RNA_POL_III_TRANSCRIPTION_TERMINATION","REACTOME_MITOTIC_PROMETAPHASE","REACTOME_PPARA_ACTIVATES_GENE_EXPRESSION"],"label":["REACTOME_TRANSCRIPTION","REACTOME_MEIOTIC_SYNAPSIS","REACTOME_MEIOSIS","REACTOME_RNA_POL_I_RNA_POL_III_AND_MITOCHONDRIAL_TRANSCRIPTION","REACTOME_RNA_POL_I_PROMOTER_OPENING","REACTOME_RNA_POL_I_TRANSCRIPTION","REACTOME_PACKAGING_OF_TELOMERE_ENDS","REACTOME_CHROMOSOME_MAINTENANCE","REACTOME_AMYLOIDS","REACTOME_MEIOTIC_RECOMBINATION","REACTOME_DEPOSITION_OF_NEW_CENPA_CONTAINING_NUCLEOSOMES_AT_THE_CENTROMERE","REACTOME_TELOMERE_MAINTENANCE","REACTOME_VIRAL_MESSENGER_RNA_SYNTHESIS","REACTOME_RNA_POL_II_PRE_TRANSCRIPTION_EVENTS","REACTOME_RNA_POL_II_TRANSCRIPTION_PRE_INITIATION_AND_PROMOTER_OPENING","REACTOME_ELONGATION_ARREST_AND_RECOVERY","REACTOME_REGULATORY_RNA_PATHWAYS","REACTOME_ABORTIVE_ELONGATION_OF_HIV1_TRANSCRIPT_IN_THE_ABSENCE_OF_TAT","REACTOME_FORMATION_OF_RNA_POL_II_ELONGATION_COMPLEX_","REACTOME_MRNA_CAPPING","REACTOME_LATE_PHASE_OF_HIV_LIFE_CYCLE","REACTOME_RNA_POL_II_TRANSCRIPTION","REACTOME_FORMATION_OF_THE_HIV1_EARLY_ELONGATION_COMPLEX","REACTOME_CELL_CYCLE","REACTOME_HIV_LIFE_CYCLE","REACTOME_MRNA_SPLICING_MINOR_PATHWAY","REACTOME_MICRORNA_MIRNA_BIOGENESIS","REACTOME_FORMATION_OF_TRANSCRIPTION_COUPLED_NER_TC_NER_REPAIR_COMPLEX","REACTOME_HIV_INFECTION","REACTOME_TRANSCRIPTION_COUPLED_NER_TC_NER","REACTOME_MRNA_SPLICING","REACTOME_NUCLEOTIDE_EXCISION_REPAIR","REACTOME_PRE_NOTCH_TRANSCRIPTION_AND_TRANSLATION","REACTOME_PROCESSING_OF_CAPPED_INTRON_CONTAINING_PRE_MRNA","REACTOME_MRNA_PROCESSING","REACTOME_SIGNALING_BY_NOTCH","REACTOME_DNA_REPAIR","REACTOME_PRE_NOTCH_EXPRESSION_AND_PROCESSING","REACTOME_INFLUENZA_VIRAL_RNA_TRANSCRIPTION_AND_REPLICATION","REACTOME_CIRCADIAN_REPRESSION_OF_EXPRESSION_BY_REV_ERBA","REACTOME_RORA_ACTIVATES_CIRCADIAN_EXPRESSION","REACTOME_RNA_POL_III_TRANSCRIPTION_INITIATION_FROM_TYPE_3_PROMOTER","REACTOME_INFLUENZA_LIFE_CYCLE","REACTOME_SMAD2_SMAD3_SMAD4_HETEROTRIMER_REGULATES_TRANSCRIPTION","REACTOME_CIRCADIAN_CLOCK","REACTOME_RNA_POL_III_TRANSCRIPTION","REACTOME_BMAL1_CLOCK_NPAS2_ACTIVATES_CIRCADIAN_EXPRESSION","REACTOME_TRANSCRIPTIONAL_ACTIVITY_OF_SMAD2_SMAD3_SMAD4_HETEROTRIMER","REACTOME_FACTORS_INVOLVED_IN_MEGAKARYOCYTE_DEVELOPMENT_AND_PLATELET_PRODUCTION","REACTOME_RNA_POL_III_TRANSCRIPTION_INITIATION_FROM_TYPE_2_PROMOTER","REACTOME_YAP1_AND_WWTR1_TAZ_STIMULATED_GENE_EXPRESSION","REACTOME_NOTCH1_INTRACELLULAR_DOMAIN_REGULATES_TRANSCRIPTION","REACTOME_SIGNALING_BY_TGF_BETA_RECEPTOR_COMPLEX","REACTOME_RNA_POL_III_CHAIN_ELONGATION","REACTOME_SIGNALING_BY_NOTCH1","REACTOME_TRANSCRIPTIONAL_REGULATION_OF_WHITE_ADIPOCYTE_DIFFERENTIATION","REACTOME_GENERIC_TRANSCRIPTION_PATHWAY","REACTOME_RNA_POL_III_TRANSCRIPTION_TERMINATION","REACTOME_MITOTIC_PROMETAPHASE","REACTOME_PPARA_ACTIVATES_GENE_EXPRESSION"],"size":[34.3065693430657,24.3065693430657,27.4452554744526,27.8832116788321,23.5036496350365,25.4744525547445,22.4817518248175,28.029197080292,25.036496350365,25.2554744525547,23.7956204379562,24.4525547445255,20,23.4306569343066,21.970802919708,21.3138686131387,20.8759124087591,20.6569343065693,22.2627737226277,21.1678832116788,26.5693430656934,26.6423357664234,21.4598540145985,50,28.1021897810219,22.2627737226277,20.6569343065693,21.1678832116788,34.0875912408759,22.2627737226277,27.0802919708029,22.7007299270073,21.0948905109489,29.1970802919708,30.7299270072993,26.4963503649635,27.1532846715328,22.1897810218978,31.3138686131387,20.6569343065693,20.7299270072993,20.8759124087591,33.7956204379562,20.9489051094891,22.8467153284672,21.3868613138686,21.6058394160584,21.7518248175182,28.6131386861314,20.6569343065693,20.7299270072993,22.3357664233577,23.5766423357664,20.2189781021898,24.0875912408759,24.2335766423358,44.6715328467153,20.3649635036496,25.6204379562044,26.5693430656934],"title":["FDR: 2.4e-34","FDR: 8.7e-27","FDR: 7.3e-25","FDR: 2.1e-24","FDR: 1.2e-23","FDR: 2.3e-23","FDR: 1.1e-22","FDR: 1.6e-21","FDR: 5e-21","FDR: 9.9e-21","FDR: 5.7e-20","FDR: 6.9e-19","FDR: 1e-18","FDR: 1.9e-17","FDR: 4.9e-17","FDR: 5.6e-17","FDR: 1.7e-16","FDR: 2.6e-15","FDR: 8.9e-15","FDR: 9e-14","FDR: 1.2e-13","FDR: 1.3e-13","FDR: 3.9e-13","FDR: 1.1e-12","FDR: 2e-12","FDR: 1.1e-11","FDR: 1.3e-11","FDR: 2e-10","FDR: 4.3e-09","FDR: 1e-08","FDR: 1.9e-08","FDR: 3.1e-08","FDR: 2.2e-07","FDR: 2.6e-07","FDR: 1.2e-06","FDR: 1.3e-06","FDR: 2.8e-06","FDR: 4.1e-06","FDR: 1.5e-05","FDR: 4.4e-05","FDR: 5.4e-05","FDR: 8.1e-05","FDR: 8.4e-05","FDR: 9.4e-05","FDR: 2e-04","FDR: 0.00025","FDR: 0.00038","FDR: 0.00049","FDR: 0.00069","FDR: 0.00097","FDR: 0.0011","FDR: 0.0012","FDR: 0.0051","FDR: 0.0074","FDR: 0.008","FDR: 0.0089","FDR: 0.0089","FDR: 0.0097","FDR: 0.024","FDR: 0.043"],"color.border":["rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)"],"color.highlight":["rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)"],"color.background":["rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,0.999)","rgba(199,0,57,0.999)","rgba(199,0,57,0.998)","rgba(199,0,57,0.998)","rgba(199,0,57,0.998)","rgba(199,0,57,0.995)","rgba(199,0,57,0.994)","rgba(199,0,57,0.991)","rgba(199,0,57,0.989)","rgba(199,0,57,0.984)","rgba(199,0,57,0.977)","rgba(199,0,57,0.974)","rgba(199,0,57,0.972)","rgba(199,0,57,0.881)","rgba(199,0,57,0.828)","rgba(199,0,57,0.814)","rgba(199,0,57,0.793)","rgba(199,0,57,0.793)","rgba(199,0,57,0.774)","rgba(199,0,57,0.442)","rgba(199,0,57,0)"],"x":[-0.513088627643889,-0.859275829955052,-1,-0.674332133877546,-0.770269696125383,-0.823613262224366,-0.725312528832804,-0.662091657510681,-0.939912804452102,-0.932526672432695,-0.562547575055278,-0.773756899828308,0.392158613397469,-0.0829886710409031,-0.0270971140827557,0.0701267761919044,0.728282002997192,0.240946325707062,0.0992710742100509,0.139655491470567,-0.146039863760769,-0.319288706225017,0.137910280812832,-0.241519654721217,-0.21650467769086,1,0.683173787611673,0.317402173343347,-0.0863564038293171,0.503255738922112,-0.830658743663132,0.43610536581048,0.278551184276108,-0.960946311839519,-0.855721753847697,0.552161357746852,0.972155551338255,0.165570162494601,0.837319556059746,0.11356130064463,-0.0177577087488198,-0.499075799475658,0.766609845050477,-0.0597439215639805,0.0244351101150273,-0.392299885486205,0.04699784645204,-0.220178250837595,0.570388671062014,-0.345611737700251,0.313094000515771,0.848531966641762,-0.380453474841305,-0.509227285032623,0.699238448085701,0.875489380956635,0.520651732661003,-0.419611325063977,0.711557174201765,0.37170365217056],"y":[-0.609601429690536,0.0198867671723442,-0.0973729822935134,-0.5144036407809,-0.324935403892292,-0.452291835106598,-0.0849434728373869,0.128091762460906,-0.417292139120801,-0.283322540119195,0.00948051107703729,0.121465839423971,-0.27166836901698,-0.45814807997505,-0.652388464497379,-0.333964234174309,-0.0798173302344864,-0.387205091742161,-0.479055552941874,-0.690797257245561,-0.204995466925163,-0.550272314555209,-0.5753676782253,1,-0.0404993696229935,0.0529386703913313,-0.202762276085291,-0.72373986657654,-0.0218618635764743,-0.731539575500938,0.445797362581168,-0.845058165657131,0.317882391915711,0.472898062273867,0.578656528159608,0.15791052068915,-0.213393281467264,0.239020065269877,-0.44326922537395,0.83255722846218,0.838430880937366,0.712204788014803,-0.555184944913945,-1,0.535498080050958,0.721147363807218,0.695115041755939,-0.960922027144593,0.467524585380633,0.624782857633081,0.95601707365943,0.243433572540203,-0.918829105101387,0.609577696873848,0.20705857481533,0.474911093725926,0.812337701041562,0.547038023393427,0.667001663459949,0.61711940397206]},"edges":{"from":["REACTOME_TRANSCRIPTION","REACTOME_TRANSCRIPTION","REACTOME_MEIOTIC_SYNAPSIS","REACTOME_MEIOTIC_SYNAPSIS","REACTOME_MEIOTIC_SYNAPSIS","REACTOME_MEIOSIS","REACTOME_RNA_POL_I_RNA_POL_III_AND_MITOCHONDRIAL_TRANSCRIPTION","REACTOME_RNA_POL_I_RNA_POL_III_AND_MITOCHONDRIAL_TRANSCRIPTION","REACTOME_RNA_POL_I_PROMOTER_OPENING","REACTOME_RNA_POL_I_PROMOTER_OPENING","REACTOME_RNA_POL_I_PROMOTER_OPENING","REACTOME_RNA_POL_I_PROMOTER_OPENING","REACTOME_RNA_POL_I_TRANSCRIPTION","REACTOME_RNA_POL_I_TRANSCRIPTION","REACTOME_PACKAGING_OF_TELOMERE_ENDS","REACTOME_PACKAGING_OF_TELOMERE_ENDS","REACTOME_CHROMOSOME_MAINTENANCE","REACTOME_CHROMOSOME_MAINTENANCE","REACTOME_AMYLOIDS","REACTOME_VIRAL_MESSENGER_RNA_SYNTHESIS","REACTOME_RNA_POL_II_PRE_TRANSCRIPTION_EVENTS","REACTOME_RNA_POL_II_PRE_TRANSCRIPTION_EVENTS","REACTOME_RNA_POL_II_PRE_TRANSCRIPTION_EVENTS","REACTOME_RNA_POL_II_PRE_TRANSCRIPTION_EVENTS","REACTOME_RNA_POL_II_PRE_TRANSCRIPTION_EVENTS","REACTOME_RNA_POL_II_PRE_TRANSCRIPTION_EVENTS","REACTOME_RNA_POL_II_TRANSCRIPTION_PRE_INITIATION_AND_PROMOTER_OPENING","REACTOME_RNA_POL_II_TRANSCRIPTION_PRE_INITIATION_AND_PROMOTER_OPENING","REACTOME_ELONGATION_ARREST_AND_RECOVERY","REACTOME_ELONGATION_ARREST_AND_RECOVERY","REACTOME_REGULATORY_RNA_PATHWAYS","REACTOME_ABORTIVE_ELONGATION_OF_HIV1_TRANSCRIPT_IN_THE_ABSENCE_OF_TAT","REACTOME_ABORTIVE_ELONGATION_OF_HIV1_TRANSCRIPT_IN_THE_ABSENCE_OF_TAT","REACTOME_FORMATION_OF_RNA_POL_II_ELONGATION_COMPLEX_","REACTOME_FORMATION_OF_RNA_POL_II_ELONGATION_COMPLEX_","REACTOME_MRNA_CAPPING","REACTOME_MRNA_CAPPING","REACTOME_LATE_PHASE_OF_HIV_LIFE_CYCLE","REACTOME_LATE_PHASE_OF_HIV_LIFE_CYCLE","REACTOME_FORMATION_OF_THE_HIV1_EARLY_ELONGATION_COMPLEX","REACTOME_HIV_LIFE_CYCLE","REACTOME_FORMATION_OF_TRANSCRIPTION_COUPLED_NER_TC_NER_REPAIR_COMPLEX","REACTOME_FORMATION_OF_TRANSCRIPTION_COUPLED_NER_TC_NER_REPAIR_COMPLEX","REACTOME_TRANSCRIPTION_COUPLED_NER_TC_NER","REACTOME_MRNA_SPLICING","REACTOME_MRNA_SPLICING","REACTOME_PRE_NOTCH_TRANSCRIPTION_AND_TRANSLATION","REACTOME_PROCESSING_OF_CAPPED_INTRON_CONTAINING_PRE_MRNA","REACTOME_SIGNALING_BY_NOTCH","REACTOME_INFLUENZA_VIRAL_RNA_TRANSCRIPTION_AND_REPLICATION","REACTOME_CIRCADIAN_REPRESSION_OF_EXPRESSION_BY_REV_ERBA","REACTOME_CIRCADIAN_REPRESSION_OF_EXPRESSION_BY_REV_ERBA","REACTOME_RORA_ACTIVATES_CIRCADIAN_EXPRESSION","REACTOME_RNA_POL_III_TRANSCRIPTION_INITIATION_FROM_TYPE_3_PROMOTER","REACTOME_RNA_POL_III_TRANSCRIPTION_INITIATION_FROM_TYPE_3_PROMOTER","REACTOME_RNA_POL_III_TRANSCRIPTION_INITIATION_FROM_TYPE_3_PROMOTER","REACTOME_RNA_POL_III_TRANSCRIPTION_INITIATION_FROM_TYPE_3_PROMOTER","REACTOME_SMAD2_SMAD3_SMAD4_HETEROTRIMER_REGULATES_TRANSCRIPTION","REACTOME_CIRCADIAN_CLOCK","REACTOME_RNA_POL_III_TRANSCRIPTION","REACTOME_RNA_POL_III_TRANSCRIPTION","REACTOME_RNA_POL_III_TRANSCRIPTION","REACTOME_TRANSCRIPTIONAL_ACTIVITY_OF_SMAD2_SMAD3_SMAD4_HETEROTRIMER","REACTOME_RNA_POL_III_TRANSCRIPTION_INITIATION_FROM_TYPE_2_PROMOTER","REACTOME_RNA_POL_III_TRANSCRIPTION_INITIATION_FROM_TYPE_2_PROMOTER","REACTOME_NOTCH1_INTRACELLULAR_DOMAIN_REGULATES_TRANSCRIPTION","REACTOME_RNA_POL_III_CHAIN_ELONGATION"],"to":["REACTOME_RNA_POL_I_RNA_POL_III_AND_MITOCHONDRIAL_TRANSCRIPTION","REACTOME_RNA_POL_II_TRANSCRIPTION","REACTOME_MEIOSIS","REACTOME_PACKAGING_OF_TELOMERE_ENDS","REACTOME_CHROMOSOME_MAINTENANCE","REACTOME_MEIOTIC_RECOMBINATION","REACTOME_RNA_POL_I_PROMOTER_OPENING","REACTOME_RNA_POL_I_TRANSCRIPTION","REACTOME_RNA_POL_I_TRANSCRIPTION","REACTOME_PACKAGING_OF_TELOMERE_ENDS","REACTOME_AMYLOIDS","REACTOME_MEIOTIC_RECOMBINATION","REACTOME_AMYLOIDS","REACTOME_MEIOTIC_RECOMBINATION","REACTOME_DEPOSITION_OF_NEW_CENPA_CONTAINING_NUCLEOSOMES_AT_THE_CENTROMERE","REACTOME_TELOMERE_MAINTENANCE","REACTOME_DEPOSITION_OF_NEW_CENPA_CONTAINING_NUCLEOSOMES_AT_THE_CENTROMERE","REACTOME_TELOMERE_MAINTENANCE","REACTOME_MEIOTIC_RECOMBINATION","REACTOME_ABORTIVE_ELONGATION_OF_HIV1_TRANSCRIPT_IN_THE_ABSENCE_OF_TAT","REACTOME_RNA_POL_II_TRANSCRIPTION_PRE_INITIATION_AND_PROMOTER_OPENING","REACTOME_ELONGATION_ARREST_AND_RECOVERY","REACTOME_FORMATION_OF_RNA_POL_II_ELONGATION_COMPLEX_","REACTOME_LATE_PHASE_OF_HIV_LIFE_CYCLE","REACTOME_RNA_POL_II_TRANSCRIPTION","REACTOME_FORMATION_OF_THE_HIV1_EARLY_ELONGATION_COMPLEX","REACTOME_MRNA_CAPPING","REACTOME_FORMATION_OF_THE_HIV1_EARLY_ELONGATION_COMPLEX","REACTOME_ABORTIVE_ELONGATION_OF_HIV1_TRANSCRIPT_IN_THE_ABSENCE_OF_TAT","REACTOME_FORMATION_OF_RNA_POL_II_ELONGATION_COMPLEX_","REACTOME_MICRORNA_MIRNA_BIOGENESIS","REACTOME_FORMATION_OF_RNA_POL_II_ELONGATION_COMPLEX_","REACTOME_FORMATION_OF_THE_HIV1_EARLY_ELONGATION_COMPLEX","REACTOME_MRNA_CAPPING","REACTOME_FORMATION_OF_THE_HIV1_EARLY_ELONGATION_COMPLEX","REACTOME_FORMATION_OF_THE_HIV1_EARLY_ELONGATION_COMPLEX","REACTOME_FORMATION_OF_TRANSCRIPTION_COUPLED_NER_TC_NER_REPAIR_COMPLEX","REACTOME_HIV_LIFE_CYCLE","REACTOME_HIV_INFECTION","REACTOME_FORMATION_OF_TRANSCRIPTION_COUPLED_NER_TC_NER_REPAIR_COMPLEX","REACTOME_HIV_INFECTION","REACTOME_TRANSCRIPTION_COUPLED_NER_TC_NER","REACTOME_NUCLEOTIDE_EXCISION_REPAIR","REACTOME_NUCLEOTIDE_EXCISION_REPAIR","REACTOME_PROCESSING_OF_CAPPED_INTRON_CONTAINING_PRE_MRNA","REACTOME_MRNA_PROCESSING","REACTOME_PRE_NOTCH_EXPRESSION_AND_PROCESSING","REACTOME_MRNA_PROCESSING","REACTOME_SIGNALING_BY_NOTCH1","REACTOME_INFLUENZA_LIFE_CYCLE","REACTOME_RORA_ACTIVATES_CIRCADIAN_EXPRESSION","REACTOME_BMAL1_CLOCK_NPAS2_ACTIVATES_CIRCADIAN_EXPRESSION","REACTOME_BMAL1_CLOCK_NPAS2_ACTIVATES_CIRCADIAN_EXPRESSION","REACTOME_RNA_POL_III_TRANSCRIPTION","REACTOME_RNA_POL_III_TRANSCRIPTION_INITIATION_FROM_TYPE_2_PROMOTER","REACTOME_RNA_POL_III_CHAIN_ELONGATION","REACTOME_RNA_POL_III_TRANSCRIPTION_TERMINATION","REACTOME_TRANSCRIPTIONAL_ACTIVITY_OF_SMAD2_SMAD3_SMAD4_HETEROTRIMER","REACTOME_BMAL1_CLOCK_NPAS2_ACTIVATES_CIRCADIAN_EXPRESSION","REACTOME_RNA_POL_III_TRANSCRIPTION_INITIATION_FROM_TYPE_2_PROMOTER","REACTOME_RNA_POL_III_CHAIN_ELONGATION","REACTOME_RNA_POL_III_TRANSCRIPTION_TERMINATION","REACTOME_SIGNALING_BY_TGF_BETA_RECEPTOR_COMPLEX","REACTOME_RNA_POL_III_CHAIN_ELONGATION","REACTOME_RNA_POL_III_TRANSCRIPTION_TERMINATION","REACTOME_SIGNALING_BY_NOTCH1","REACTOME_RNA_POL_III_TRANSCRIPTION_TERMINATION"],"weight":[0.580952380952381,0.5,0.629310344827586,0.657534246575342,0.588709677419355,0.741379310344828,0.508196721311475,0.729508196721312,0.696629213483146,0.594202898550725,0.686046511627907,0.662921348314607,0.52212389380531,0.508620689655172,0.561643835616438,0.64,0.532258064516129,0.604838709677419,0.536363636363636,0.608695652173913,0.672131147540984,0.524590163934426,0.737704918032787,0.586538461538462,0.580952380952381,0.557377049180328,0.543478260869565,0.5,0.617647058823529,0.711111111111111,0.884615384615385,0.511111111111111,0.676470588235294,0.595744680851064,0.755555555555556,0.777777777777778,0.621621621621622,0.832,0.502415458937198,0.560975609756098,0.603864734299517,0.666666666666667,0.588235294117647,0.882352941176471,0.792857142857143,0.68944099378882,0.659090909090909,0.869565217391304,0.679611650485437,0.832512315270936,0.88,0.594594594594595,0.578947368421053,0.787878787878788,0.580645161290323,0.653846153846154,0.607142857142857,0.710526315789474,0.679245283018868,0.696969696969697,0.515151515151515,0.575757575757576,0.603174603174603,0.739130434782609,0.68,0.657142857142857,0.894736842105263],"value":[0.580952380952381,0.5,0.629310344827586,0.657534246575342,0.588709677419355,0.741379310344828,0.508196721311475,0.729508196721312,0.696629213483146,0.594202898550725,0.686046511627907,0.662921348314607,0.52212389380531,0.508620689655172,0.561643835616438,0.64,0.532258064516129,0.604838709677419,0.536363636363636,0.608695652173913,0.672131147540984,0.524590163934426,0.737704918032787,0.586538461538462,0.580952380952381,0.557377049180328,0.543478260869565,0.5,0.617647058823529,0.711111111111111,0.884615384615385,0.511111111111111,0.676470588235294,0.595744680851064,0.755555555555556,0.777777777777778,0.621621621621622,0.832,0.502415458937198,0.560975609756098,0.603864734299517,0.666666666666667,0.588235294117647,0.882352941176471,0.792857142857143,0.68944099378882,0.659090909090909,0.869565217391304,0.679611650485437,0.832512315270936,0.88,0.594594594594595,0.578947368421053,0.787878787878788,0.580645161290323,0.653846153846154,0.607142857142857,0.710526315789474,0.679245283018868,0.696969696969697,0.515151515151515,0.575757575757576,0.603174603174603,0.739130434782609,0.68,0.657142857142857,0.894736842105263]},"nodesToDataframe":true,"edgesToDataframe":true,"options":{"width":"100%","height":"100%","nodes":{"shape":"dot","borderWidth":1,"borderWidthSelected":0,"physics":false},"manipulation":{"enabled":false},"edges":{"color":"rgb(88,24,69)","smooth":false},"interaction":{"multiselect":true,"tooltipDelay":300},"physics":{"stabilization":false}},"groups":null,"width":null,"height":null,"idselection":{"enabled":false,"style":"width: 150px; height: 26px","useLabels":true,"main":"Select by id"},"byselection":{"enabled":false,"style":"width: 150px; height: 26px","multiple":false,"hideColor":"rgba(200,200,200,0.5)"},"main":{"text":"","style":"font-family:Helvetica"},"submain":null,"footer":null,"background":"rgba(0, 0, 0, 0)","highlight":{"enabled":true,"hoverNearest":false,"degree":1,"algorithm":"all","hideColor":"rgba(200,200,200,0.5)","labelOnly":true},"collapse":{"enabled":false,"fit":false,"resetHighlight":true,"clusterOptions":null,"keepCoord":true,"labelSuffix":"(cluster)"},"tooltipStay":300,"tooltipStyle":"position: fixed;visibility:hidden;padding: 5px;white-space: nowrap;font-family: verdana;font-size:14px;font-color:#000000;background-color: #f5f4ed;-moz-border-radius: 3px;-webkit-border-radius: 3px;border-radius: 3px;border: 1px solid #808074;box-shadow: 3px 3px 10px rgba(0, 0, 0, 0.2);","igraphlayout":{"type":"square"}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</div>
<div id="advanced" class="section level1">
<h1 class="hasAnchor">
<a href="#advanced" class="anchor"></a>Advanced</h1>
<div id="relational-genesets" class="section level2">
<h2 class="hasAnchor">
<a href="#relational-genesets" class="anchor"></a>Relational genesets</h2>
<p>When dealing with hundreds of genesets, itâ€™s often useful to understand the relationships between them. This allows researchers to summarize many enriched pathways as more general biological processes. To do this, we rely on curated relationships defined between them. For example, <a href="https://reactome.org/">REACTOME</a> conveniently defines their genesets in a <a href="https://reactome.org/PathwayBrowser/">hiearchy of pathways</a>. This data can be formatted into a relational genesets object called <strong><em>rgsets</em></strong>.</p>
<div class="sourceCode" id="cb24"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb24-1" title="1">rgsets_lst &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/readRDS">readRDS</a></span>(<span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/system.file">system.file</a></span>(<span class="st">"extdata/rgsets.rds"</span>, <span class="dt">package=</span><span class="st">"hypeR"</span>))</a>
<a class="sourceLine" id="cb24-2" title="2">rgsets_obj &lt;-<span class="st"> </span>rgsets_lst<span class="op">$</span>REACTOME</a></code></pre></div>
<p>Relational genesets have three data atrributes including gsets, nodes, and edges. Using hierarchical terms, the <em>gsets</em> attribute includes the geneset information for the <em>leaf nodes</em> of the hiearchy, the <em>nodes</em> attribute describes all nodes in the hiearchical, including <em>internal nodes</em>, and the <em>edges</em> attribute describes the edges in the hiearchy.</p>
<div id="gsets" class="section level4">
<h4 class="hasAnchor">
<a href="#gsets" class="anchor"></a>$gsets</h4>
<div class="sourceCode" id="cb25"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb25-1" title="1">gsets &lt;-<span class="st"> </span>rgsets_obj<span class="op">$</span>gsets</a>
<a class="sourceLine" id="cb25-2" title="2"><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/names">names</a></span>(gsets)[<span class="dv">800</span><span class="op">:</span><span class="dv">805</span>]</a></code></pre></div>
<pre><code>## [1] "MET activates PTK2 signaling"    "MET activates PTPN11"           
## [3] "MET activates RAP1 and RAC1"     "MET activates RAS signaling"    
## [5] "MET activates STAT3"             "MET interacts with TNS proteins"</code></pre>
</div>
<div id="nodes" class="section level4">
<h4 class="hasAnchor">
<a href="#nodes" class="anchor"></a>$nodes</h4>
<div class="sourceCode" id="cb27"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb27-1" title="1">nodes &lt;-<span class="st"> </span>rgsets_obj<span class="op">$</span>nodes</a>
<a class="sourceLine" id="cb27-2" title="2">nodes[<span class="dv">1123</span><span class="op">:</span><span class="dv">1128</span>,]</a></code></pre></div>
<pre><code>##                                         label            id length
## R-HSA-8874081    MET activates PTK2 signaling R-HSA-8874081     18
## R-HSA-8865999            MET activates PTPN11 R-HSA-8865999      5
## R-HSA-8875555     MET activates RAP1 and RAC1 R-HSA-8875555     11
## R-HSA-8851805     MET activates RAS signaling R-HSA-8851805     11
## R-HSA-8875791             MET activates STAT3 R-HSA-8875791      3
## R-HSA-8875513 MET interacts with TNS proteins R-HSA-8875513      5</code></pre>
</div>
<div id="edges" class="section level4">
<h4 class="hasAnchor">
<a href="#edges" class="anchor"></a>$edges</h4>
<div class="sourceCode" id="cb29"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb29-1" title="1">edges &lt;-<span class="st"> </span>rgsets_obj<span class="op">$</span>edges</a>
<a class="sourceLine" id="cb29-2" title="2">edges[<span class="dv">1994</span><span class="op">:</span><span class="dv">1999</span>,]</a></code></pre></div>
<pre><code>##               from            to
## 1994 R-HSA-8875878 R-HSA-8874081
## 1995 R-HSA-8875878 R-HSA-8875513
## 1996 R-HSA-8875878 R-HSA-8875555
## 1997 R-HSA-8876384 R-HSA-8875360
## 1998 R-HSA-8876384 R-HSA-8876493
## 1999 R-HSA-8878159 R-HSA-8941855</code></pre>
<p>All workflows begin with performing hyper enrichment with <strong><em>hypeR</em></strong>. Note that when passing an <strong><em>rgsets</em></strong> object, ensure <code>gsets_relational=TRUE</code>. Otherwise, <strong><em>hypeR</em></strong> will treat it as a list of unstructured gsets.</p>
<div class="sourceCode" id="cb31"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb31-1" title="1">hyp_obj &lt;-<span class="st"> </span><span class="kw"><a href="../reference/hypeR.html">hypeR</a></span>(signature, rgsets_obj, <span class="dt">gsets_relational=</span><span class="ot">TRUE</span>, <span class="dt">bg=</span><span class="dv">7842</span>, <span class="dt">fdr=</span><span class="fl">0.05</span>)</a></code></pre></div>
</div>
<div id="hierarchy-map" class="section level3">
<h3 class="hasAnchor">
<a href="#hierarchy-map" class="anchor"></a>Hierarchy map</h3>
<p>One can visualize the top enriched pathways using <strong><em>hyp_hmap</em></strong> which will return a hiearchy map. Each node represents a geneset, where the shade of the gold border indicates the normalized significance of enrichment. Hover over the <em>leaf nodes</em> to view the raw value. Double click <em>internal nodes</em> to cluster their first degree connections.s Edges represent a directed relationship between genesets in the hiearchy.</p>
<div class="sourceCode" id="cb32"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb32-1" title="1"><span class="kw"><a href="../reference/hyp_hmap.html">hyp_hmap</a></span>(hyp_obj, <span class="dt">top=</span><span class="dv">60</span>, <span class="dt">return_plots=</span><span class="ot">TRUE</span>)</a></code></pre></div>
<div id="htmlwidget-89da55dca06d772538db" style="width:648px;height:432.632880098888px;" class="visNetwork html-widget"></div>
<script type="application/json" data-for="htmlwidget-89da55dca06d772538db">{"x":{"nodes":{"label":["Abortive elongation of HIV-1 transcript in the absence of Tat","HIV Transcription Elongation","Transcription of the HIV genome","Late Phase of HIV Life Cycle","HIV Life Cycle","HIV Infection","Infectious disease","Disease","Activated PKN1 stimulates transcription of AR (androgen receptor) regulated genes KLK2 and KLK3","RHO GTPases activate PKNs","RHO GTPase Effectors","Signaling by Rho GTPases","Signal Transduction","Activation of anterior HOX genes in hindbrain development during early embryogenesis","Activation of HOX genes during differentiation","Developmental Biology","Amyloid fiber formation","Metabolism of proteins","B-WICH complex positively regulates rRNA expression","Positive epigenetic regulation of rRNA expression","Epigenetic regulation of gene expression","Gene expression (Transcription)","Cleavage of the damaged purine","Depurination","Base-Excision Repair, AP Site Formation","Base Excision Repair","DNA Repair","Condensation of Prophase Chromosomes","Mitotic Prophase","M Phase","Cell Cycle, Mitotic","Cell Cycle","DNA methylation","Deposition of new CENPA-containing nucleosomes at the centromere","Nucleosome assembly","Chromosome Maintenance","ERCC6 (CSB) and EHMT2 (G9a) positively regulate rRNA expression","Estrogen-dependent gene expression","ESR-mediated signaling","Signaling by Nuclear Receptors","FGFR2 alternative splicing","Signaling by FGFR2","Signaling by FGFR","Signaling by Receptor Tyrosine Kinases","Formation of HIV elongation complex in the absence of HIV Tat","Formation of HIV-1 elongation complex containing HIV-1 Tat","Tat-mediated elongation of the HIV-1 transcript","Formation of TC-NER Pre-Incision Complex","Transcription-Coupled Nucleotide Excision Repair (TC-NER)","Nucleotide Excision Repair","Formation of the Early Elongation Complex","RNA Polymerase II Transcription Elongation","RNA Polymerase II Transcription","Formation of the HIV-1 Early Elongation Complex","HATs acetylate histones","Chromatin modifying enzymes","Chromatin organization","HDACs deacetylate histones","HIV Transcription Initiation","HIV elongation arrest and recovery","Meiotic recombination","Meiosis","Reproduction","Meiotic synapsis","MicroRNA (miRNA) biogenesis","Gene Silencing by RNA","NoRC negatively regulates rRNA expression","Negative epigenetic regulation of rRNA expression","Nonhomologous End-Joining (NHEJ)","DNA Double-Strand Break Repair","Oxidative Stress Induced Senescence","Cellular Senescence","Cellular responses to stress","Cellular responses to external stimuli","PIWI-interacting RNA (piRNA) biogenesis","PRC2 methylates histones and DNA","Packaging Of Telomere Ends","Telomere Maintenance","Pausing and recovery of HIV elongation","Pausing and recovery of Tat-mediated HIV elongation","Pre-NOTCH Transcription and Translation","Pre-NOTCH Expression and Processing","Signaling by NOTCH","Processing of DNA double-strand break ends","HDR through Homologous Recombination (HRR) or Single Strand Annealing (SSA)","Homology Directed Repair","RMTs methylate histone arginines","RNA Pol II CTD phosphorylation and interaction with CE","RNA Pol II CTD phosphorylation and interaction with CE during HIV infection","RNA Polymerase I Promoter Escape","RNA Polymerase I Promoter Clearance","RNA Polymerase I Transcription","RNA Polymerase I Promoter Opening","RNA Polymerase II HIV Promoter Escape","RNA Polymerase II Pre-transcription Events","RNA Polymerase II Promoter Escape","RNA Polymerase II Transcription Initiation And Promoter Clearance","RNA Polymerase II Transcription Initiation","RNA Polymerase II Transcription Pre-Initiation And Promoter Opening","RNA polymerase II transcribes snRNA genes","RUNX1 regulates genes involved in megakaryocyte differentiation and platelet function","Transcriptional regulation by RUNX1","Generic Transcription Pathway","RUNX1 regulates transcription of genes involved in differentiation of HSCs","Recognition and association of DNA glycosylase with site containing an affected purine","Recognition and association of DNA glycosylase with site containing an affected pyrimidine","Depyrimidination","Recruitment and ATM-mediated phosphorylation of repair and signaling proteins at DNA double strand breaks","DNA Double Strand Break Response","SIRT1 negatively regulates rRNA expression","Senescence-Associated Secretory Phenotype (SASP)","Signaling by FGFR2 IIIa TM","FGFR2 mutant receptor activation","Signaling by FGFR2 in disease","Signaling by FGFR in disease","Diseases of signal transduction","TP53 Regulates Transcription of DNA Repair Genes","Transcriptional Regulation by TP53","Tat-mediated HIV elongation arrest and recovery","Transcriptional regulation by small RNAs","Ub-specific processing proteases","Deubiquitination","Post-translational protein modification","Viral Messenger RNA Synthesis","Influenza Viral RNA Transcription and Replication","Influenza Life Cycle","Influenza Infection","mRNA Capping","Metabolism of RNA","mRNA Splicing - Minor Pathway","mRNA Splicing","Processing of Capped Intron-Containing Pre-mRNA"],"id":["R-HSA-167242","R-HSA-167169","R-HSA-167172","R-HSA-162599","R-HSA-162587","R-HSA-162906","R-HSA-5663205","R-HSA-1643685","R-HSA-5625886","R-HSA-5625740","R-HSA-195258","R-HSA-194315","R-HSA-162582","R-HSA-5617472","R-HSA-5619507","R-HSA-1266738","R-HSA-977225","R-HSA-392499","R-HSA-5250924","R-HSA-5250913","R-HSA-212165","R-HSA-74160","R-HSA-110331","R-HSA-73927","R-HSA-73929","R-HSA-73884","R-HSA-73894","R-HSA-2299718","R-HSA-68875","R-HSA-68886","R-HSA-69278","R-HSA-1640170","R-HSA-5334118","R-HSA-606279","R-HSA-774815","R-HSA-73886","R-HSA-427389","R-HSA-9018519","R-HSA-8939211","R-HSA-9006931","R-HSA-6803529","R-HSA-5654738","R-HSA-190236","R-HSA-9006934","R-HSA-167152","R-HSA-167200","R-HSA-167246","R-HSA-6781823","R-HSA-6781827","R-HSA-5696398","R-HSA-113418","R-HSA-75955","R-HSA-73857","R-HSA-167158","R-HSA-3214847","R-HSA-3247509","R-HSA-4839726","R-HSA-3214815","R-HSA-167161","R-HSA-167287","R-HSA-912446","R-HSA-1500620","R-HSA-1474165","R-HSA-1221632","R-HSA-203927","R-HSA-211000","R-HSA-427413","R-HSA-5250941","R-HSA-5693571","R-HSA-5693532","R-HSA-2559580","R-HSA-2559583","R-HSA-2262752","R-HSA-8953897","R-HSA-5601884","R-HSA-212300","R-HSA-171306","R-HSA-157579","R-HSA-167290","R-HSA-167238","R-HSA-1912408","R-HSA-1912422","R-HSA-157118","R-HSA-5693607","R-HSA-5693567","R-HSA-5693538","R-HSA-3214858","R-HSA-77075","R-HSA-167160","R-HSA-73772","R-HSA-73854","R-HSA-73864","R-HSA-73728","R-HSA-167162","R-HSA-674695","R-HSA-73776","R-HSA-76042","R-HSA-75953","R-HSA-73779","R-HSA-6807505","R-HSA-8936459","R-HSA-8878171","R-HSA-212436","R-HSA-8939236","R-HSA-110330","R-HSA-110328","R-HSA-73928","R-HSA-5693565","R-HSA-5693606","R-HSA-427359","R-HSA-2559582","R-HSA-8851708","R-HSA-1839126","R-HSA-5655253","R-HSA-1226099","R-HSA-5663202","R-HSA-6796648","R-HSA-3700989","R-HSA-167243","R-HSA-5578749","R-HSA-5689880","R-HSA-5688426","R-HSA-597592","R-HSA-168325","R-HSA-168273","R-HSA-168255","R-HSA-168254","R-HSA-72086","R-HSA-8953854","R-HSA-72165","R-HSA-72172","R-HSA-72203"],"length":[23,47,74,74,74,74,109,112,36,36,36,36,172,91,91,91,79,259,60,76,118,410,37,37,44,44,185,43,43,43,43,129,34,54,54,61,46,122,122,122,28,28,28,28,48,47,47,53,53,53,33,33,281,33,108,171,171,60,47,36,56,88,88,60,26,104,77,83,52,109,95,132,132,132,29,42,33,33,36,35,74,74,74,81,81,81,49,27,27,60,62,62,32,47,84,47,47,47,47,81,67,133,195,99,37,42,42,59,59,40,80,19,19,19,19,19,65,65,35,76,205,205,205,49,49,49,49,29,68,55,55,55],"size":[15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,18.4143222506394,15,15,15,30,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,20.1023017902813,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15],"title":["FDR: 3.7e-14",null,null,null,null,null,null,null,"FDR: 9.8e-41",null,null,null,null,"FDR: 2.2e-42",null,null,"FDR: 1.2e-22",null,"FDR: 2.6e-38",null,null,null,"FDR: 1.2e-31",null,null,null,null,"FDR: 1.1e-33",null,null,null,null,"FDR: 3.6e-37","FDR: 6.6e-27",null,null,"FDR: 1.2e-34","FDR: 9.3e-205",null,null,"FDR: 4.8e-13",null,null,null,"FDR: 4.7e-13","FDR: 3.5e-13",null,"FDR: 2.3e-08",null,null,"FDR: 3.7e-12",null,null,"FDR: 3.7e-12","FDR: 8e-21",null,null,"FDR: 2.8e-22","FDR: 1.1e-14","FDR: 8.5e-15","FDR: 4.4e-30",null,null,"FDR: 1.2e-34","FDR: 9.3e-12",null,"FDR: 1.3e-34",null,"FDR: 1.4e-12",null,"FDR: 5.3e-30",null,null,null,"FDR: 1.4e-09","FDR: 5.4e-34","FDR: 2.8e-33",null,"FDR: 8.5e-15","FDR: 5.8e-15","FDR: 2.2e-42",null,null,"FDR: 5.4e-10",null,null,"FDR: 7.4e-16","FDR: 3.2e-13","FDR: 3.2e-13","FDR: 1.6e-36",null,null,"FDR: 3.3e-38","FDR: 1.1e-14","FDR: 1.7e-13","FDR: 1.1e-14",null,"FDR: 1.1e-14","FDR: 1.1e-14","FDR: 2.1e-16","FDR: 4.6e-44",null,null,"FDR: 1.9e-29","FDR: 1.2e-31","FDR: 5.3e-30",null,"FDR: 3.4e-13",null,"FDR: 9.3e-37","FDR: 1e-25","FDR: 2.9e-15",null,null,null,null,"FDR: 1.3e-12",null,"FDR: 5.8e-15","FDR: 1.8e-45","FDR: 1.9e-09",null,null,"FDR: 4.6e-10",null,null,null,"FDR: 7.5e-13",null,"FDR: 1.7e-09",null,null],"color.background":["rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(199,0,57,1.0)","rgba(199,0,57,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(199,0,57,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(199,0,57,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(199,0,57,1.0)","rgba(199,0,57,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(199,0,57,1.0)","rgba(199,0,57,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(199,0,57,1.0)","rgba(199,0,57,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(199,0,57,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(199,0,57,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(199,0,57,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)"],"color.highlight":["rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)"],"color.border":["rgba(255,195,0,1)","rgb(88,24,69)","rgb(88,24,69)","rgb(88,24,69)","rgb(88,24,69)","rgb(88,24,69)","rgb(88,24,69)","rgb(88,24,69)","rgba(255,195,0,1)","rgb(88,24,69)","rgb(88,24,69)","rgb(88,24,69)","rgb(88,24,69)","rgba(255,195,0,1)","rgb(88,24,69)","rgb(88,24,69)","rgba(255,195,0,1)","rgb(88,24,69)","rgba(255,195,0,1)","rgb(88,24,69)","rgb(88,24,69)","rgb(88,24,69)","rgba(255,195,0,1)","rgb(88,24,69)","rgb(88,24,69)","rgb(88,24,69)","rgb(88,24,69)","rgba(255,195,0,1)","rgb(88,24,69)","rgb(88,24,69)","rgb(88,24,69)","rgb(88,24,69)","rgba(255,195,0,1)","rgba(255,195,0,1)","rgb(88,24,69)","rgb(88,24,69)","rgba(255,195,0,1)","rgba(255,195,0,1)","rgb(88,24,69)","rgb(88,24,69)","rgba(255,195,0,1)","rgb(88,24,69)","rgb(88,24,69)","rgb(88,24,69)","rgba(255,195,0,1)","rgba(255,195,0,1)","rgb(88,24,69)","rgba(255,195,0,0)","rgb(88,24,69)","rgb(88,24,69)","rgba(255,195,0,1)","rgb(88,24,69)","rgb(88,24,69)","rgba(255,195,0,1)","rgba(255,195,0,1)","rgb(88,24,69)","rgb(88,24,69)","rgba(255,195,0,1)","rgba(255,195,0,1)","rgba(255,195,0,1)","rgba(255,195,0,1)","rgb(88,24,69)","rgb(88,24,69)","rgba(255,195,0,1)","rgba(255,195,0,1)","rgb(88,24,69)","rgba(255,195,0,1)","rgb(88,24,69)","rgba(255,195,0,1)","rgb(88,24,69)","rgba(255,195,0,1)","rgb(88,24,69)","rgb(88,24,69)","rgb(88,24,69)","rgba(255,195,0,0.939)","rgba(255,195,0,1)","rgba(255,195,0,1)","rgb(88,24,69)","rgba(255,195,0,1)","rgba(255,195,0,1)","rgba(255,195,0,1)","rgb(88,24,69)","rgb(88,24,69)","rgba(255,195,0,0.977)","rgb(88,24,69)","rgb(88,24,69)","rgba(255,195,0,1)","rgba(255,195,0,1)","rgba(255,195,0,1)","rgba(255,195,0,1)","rgb(88,24,69)","rgb(88,24,69)","rgba(255,195,0,1)","rgba(255,195,0,1)","rgba(255,195,0,1)","rgba(255,195,0,1)","rgb(88,24,69)","rgba(255,195,0,1)","rgba(255,195,0,1)","rgba(255,195,0,1)","rgba(255,195,0,1)","rgb(88,24,69)","rgb(88,24,69)","rgba(255,195,0,1)","rgba(255,195,0,1)","rgba(255,195,0,1)","rgb(88,24,69)","rgba(255,195,0,1)","rgb(88,24,69)","rgba(255,195,0,1)","rgba(255,195,0,1)","rgba(255,195,0,1)","rgb(88,24,69)","rgb(88,24,69)","rgb(88,24,69)","rgb(88,24,69)","rgba(255,195,0,1)","rgb(88,24,69)","rgba(255,195,0,1)","rgba(255,195,0,1)","rgba(255,195,0,0.917)","rgb(88,24,69)","rgb(88,24,69)","rgba(255,195,0,0.98)","rgb(88,24,69)","rgb(88,24,69)","rgb(88,24,69)","rgba(255,195,0,1)","rgb(88,24,69)","rgba(255,195,0,0.926)","rgb(88,24,69)","rgb(88,24,69)"]},"edges":{"from":["R-HSA-1226099","R-HSA-1266738","R-HSA-1474165","R-HSA-1500620","R-HSA-1500620","R-HSA-157118","R-HSA-157579","R-HSA-162582","R-HSA-162582","R-HSA-162582","R-HSA-162582","R-HSA-162587","R-HSA-162599","R-HSA-162906","R-HSA-1640170","R-HSA-1640170","R-HSA-1640170","R-HSA-1643685","R-HSA-1643685","R-HSA-167169","R-HSA-167169","R-HSA-167169","R-HSA-167172","R-HSA-167172","R-HSA-167172","R-HSA-167172","R-HSA-167172","R-HSA-167172","R-HSA-167172","R-HSA-167172","R-HSA-167172","R-HSA-167246","R-HSA-168254","R-HSA-168255","R-HSA-168273","R-HSA-1839126","R-HSA-190236","R-HSA-1912422","R-HSA-194315","R-HSA-195258","R-HSA-211000","R-HSA-211000","R-HSA-211000","R-HSA-212165","R-HSA-212165","R-HSA-212165","R-HSA-212165","R-HSA-212436","R-HSA-212436","R-HSA-2262752","R-HSA-2559583","R-HSA-2559583","R-HSA-3247509","R-HSA-3247509","R-HSA-3247509","R-HSA-3700989","R-HSA-392499","R-HSA-392499","R-HSA-4839726","R-HSA-5250913","R-HSA-5250913","R-HSA-5250941","R-HSA-5250941","R-HSA-5619507","R-HSA-5625740","R-HSA-5654738","R-HSA-5655253","R-HSA-5663202","R-HSA-5663205","R-HSA-5663205","R-HSA-5688426","R-HSA-5693532","R-HSA-5693532","R-HSA-5693532","R-HSA-5693538","R-HSA-5693567","R-HSA-5693606","R-HSA-5696398","R-HSA-597592","R-HSA-6781827","R-HSA-68875","R-HSA-68886","R-HSA-69278","R-HSA-72172","R-HSA-72203","R-HSA-73854","R-HSA-73854","R-HSA-73857","R-HSA-73857","R-HSA-73857","R-HSA-73857","R-HSA-73857","R-HSA-73857","R-HSA-73857","R-HSA-73864","R-HSA-73884","R-HSA-73886","R-HSA-73886","R-HSA-73894","R-HSA-73894","R-HSA-73894","R-HSA-73927","R-HSA-73927","R-HSA-73928","R-HSA-73929","R-HSA-73929","R-HSA-74160","R-HSA-74160","R-HSA-74160","R-HSA-74160","R-HSA-75955","R-HSA-76042","R-HSA-76042","R-HSA-774815","R-HSA-8878171","R-HSA-8878171","R-HSA-8939211","R-HSA-8953854","R-HSA-8953854","R-HSA-8953897","R-HSA-9006931","R-HSA-9006934"],"to":["R-HSA-5655253","R-HSA-5619507","R-HSA-1500620","R-HSA-1221632","R-HSA-912446","R-HSA-1912422","R-HSA-171306","R-HSA-157118","R-HSA-194315","R-HSA-9006931","R-HSA-9006934","R-HSA-162599","R-HSA-167172","R-HSA-162587","R-HSA-1500620","R-HSA-69278","R-HSA-73886","R-HSA-5663202","R-HSA-5663205","R-HSA-167158","R-HSA-167242","R-HSA-167246","R-HSA-167152","R-HSA-167160","R-HSA-167161","R-HSA-167162","R-HSA-167169","R-HSA-167238","R-HSA-167243","R-HSA-167287","R-HSA-167290","R-HSA-167200","R-HSA-168255","R-HSA-168273","R-HSA-168325","R-HSA-8851708","R-HSA-5654738","R-HSA-1912408","R-HSA-195258","R-HSA-5625740","R-HSA-203927","R-HSA-5578749","R-HSA-5601884","R-HSA-212300","R-HSA-5250913","R-HSA-5250941","R-HSA-5334118","R-HSA-3700989","R-HSA-8878171","R-HSA-2559583","R-HSA-2559580","R-HSA-2559582","R-HSA-3214815","R-HSA-3214847","R-HSA-3214858","R-HSA-6796648","R-HSA-597592","R-HSA-977225","R-HSA-3247509","R-HSA-427389","R-HSA-5250924","R-HSA-427359","R-HSA-427413","R-HSA-5617472","R-HSA-5625886","R-HSA-6803529","R-HSA-1839126","R-HSA-1226099","R-HSA-162906","R-HSA-168254","R-HSA-5689880","R-HSA-5693538","R-HSA-5693571","R-HSA-5693606","R-HSA-5693567","R-HSA-5693607","R-HSA-5693565","R-HSA-6781827","R-HSA-5688426","R-HSA-6781823","R-HSA-2299718","R-HSA-68875","R-HSA-68886","R-HSA-72165","R-HSA-72172","R-HSA-73728","R-HSA-73772","R-HSA-212436","R-HSA-674695","R-HSA-6807505","R-HSA-73779","R-HSA-75955","R-HSA-76042","R-HSA-77075","R-HSA-73854","R-HSA-73929","R-HSA-157579","R-HSA-774815","R-HSA-5693532","R-HSA-5696398","R-HSA-73884","R-HSA-110330","R-HSA-110331","R-HSA-110328","R-HSA-73927","R-HSA-73928","R-HSA-211000","R-HSA-212165","R-HSA-73857","R-HSA-73864","R-HSA-113418","R-HSA-73776","R-HSA-75953","R-HSA-606279","R-HSA-8936459","R-HSA-8939236","R-HSA-9018519","R-HSA-72086","R-HSA-72203","R-HSA-2262752","R-HSA-8939211","R-HSA-190236"]},"nodesToDataframe":true,"edgesToDataframe":true,"options":{"width":"100%","height":"100%","nodes":{"shape":"dot","borderWidth":3,"borderWidthSelected":0},"manipulation":{"enabled":false},"edges":{"selectionWidth":0,"arrows":"to"},"interaction":{"multiselect":true,"tooltipDelay":300}},"groups":null,"width":null,"height":null,"idselection":{"enabled":false,"style":"width: 150px; height: 26px","useLabels":true,"main":"Select by id"},"byselection":{"enabled":false,"style":"width: 150px; height: 26px","multiple":false,"hideColor":"rgba(200,200,200,0.5)"},"main":{"text":"","style":"font-family:Helvetica"},"submain":null,"footer":null,"background":"rgba(0, 0, 0, 0)","tooltipStay":300,"tooltipStyle":"position: fixed;visibility:hidden;padding: 5px;white-space: nowrap;font-family: verdana;font-size:14px;font-color:#000000;background-color: #f5f4ed;-moz-border-radius: 3px;-webkit-border-radius: 3px;border-radius: 3px;border: 1px solid #808074;box-shadow: 3px 3px 10px rgba(0, 0, 0, 0.2);","highlight":{"enabled":true,"hoverNearest":false,"degree":1,"algorithm":"all","hideColor":"rgba(200,200,200,0.5)","labelOnly":true},"collapse":{"enabled":true,"fit":false,"resetHighlight":true,"clusterOptions":{"color.border":"black"},"keepCoord":true,"labelSuffix":"(cluster)"}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div id="multiple-signatures" class="section level2">
<h2 class="hasAnchor">
<a href="#multiple-signatures" class="anchor"></a>Multiple signatures</h2>
<p>Other times we are looking at multiple signatures that may have been defined in a single experiment. In this case, we can feed <strong><em>hypeR</em></strong> a named list of signatures and a <strong><em>multihyp</em></strong> object will be returned. This object is essentially just multiple <strong><em>hyp</em></strong> objects. However it is recognized and handled differently by downstream visualization and analysis tools.</p>
<div class="sourceCode" id="cb33"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb33-1" title="1">signatures &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/readRDS">readRDS</a></span>(<span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/system.file">system.file</a></span>(<span class="st">"extdata/signatures.rds"</span>, <span class="dt">package=</span><span class="st">"hypeR"</span>))</a>
<a class="sourceLine" id="cb33-2" title="2">experiment &lt;-<span class="st"> </span>signatures<span class="op">$</span>experiment</a></code></pre></div>
<div class="sourceCode" id="cb34"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb34-1" title="1"><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/names">names</a></span>(experiment)</a></code></pre></div>
<pre><code>## [1] "YAP-KO Signature"      "YAP-KO Up-regulated"   "YAP-KO Down-regulated"</code></pre>
<div class="sourceCode" id="cb36"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb36-1" title="1"><span class="kw"><a href="https://www.rdocumentation.org/packages/utils/topics/head">head</a></span>(experiment[[<span class="dv">1</span>]])</a></code></pre></div>
<pre><code>## [1] "CD55"     "IGLC1"    "IGKV1-5"  "C9"       "IGLV4-60" "IGLV4-3"</code></pre>
<div class="sourceCode" id="cb38"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb38-1" title="1">multihyp_obj &lt;-<span class="st"> </span><span class="kw"><a href="../reference/hypeR.html">hypeR</a></span>(experiment, gsets, <span class="dt">bg=</span><span class="dv">35423</span>)</a></code></pre></div>
<div class="sourceCode" id="cb39"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb39-1" title="1"><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/print">print</a></span>(multihyp_obj)</a></code></pre></div>
<pre><code>## multihyp object: 
## YAP-KO Signature 
## hyp object: 
##   data: 1552 x 8
##   args: symbols
##         gsets
##         gsets_relational
##         bg
##         min_drawsize
##         pval_cutoff
##         fdr_cutoff
##         verbosemultihyp object: 
## YAP-KO Up-regulated 
## hyp object: 
##   data: 1552 x 8
##   args: symbols
##         gsets
##         gsets_relational
##         bg
##         min_drawsize
##         pval_cutoff
##         fdr_cutoff
##         verbosemultihyp object: 
## YAP-KO Down-regulated 
## hyp object: 
##   data: 1552 x 8
##   args: symbols
##         gsets
##         gsets_relational
##         bg
##         min_drawsize
##         pval_cutoff
##         fdr_cutoff
##         verbose</code></pre>
<div id="save-results-to-excel-1" class="section level3">
<h3 class="hasAnchor">
<a href="#save-results-to-excel-1" class="anchor"></a>Save results to excel</h3>
<p>When saving a <strong><em>multihyp</em></strong> object to excel, each signature is exported to its own tab.</p>
<div class="sourceCode" id="cb41"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb41-1" title="1"><span class="kw"><a href="../reference/hyp_to_excel.html">hyp_to_excel</a></span>(multihyp_obj, <span class="dt">file_path=</span><span class="st">"pathways.xlsx"</span>)</a></code></pre></div>
</div>
<div id="save-results-to-table-1" class="section level3">
<h3 class="hasAnchor">
<a href="#save-results-to-table-1" class="anchor"></a>Save results to table</h3>
<p>When saving a <strong><em>multihyp</em></strong> object to a table, each signature is exported as its own table in a single directory.</p>
<div class="sourceCode" id="cb42"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb42-1" title="1"><span class="kw"><a href="../reference/hyp_to_table.html">hyp_to_table</a></span>(multihyp_obj, <span class="dt">file_path=</span><span class="st">"pathways.txt"</span>)</a></code></pre></div>
<p>When plotting a <strong><em>multihyp</em></strong> object, each signatures is plotted and returned separately.</p>
</div>
<div id="plot-results-1" class="section level3">
<h3 class="hasAnchor">
<a href="#plot-results-1" class="anchor"></a>Plot results</h3>
<div class="sourceCode" id="cb43"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb43-1" title="1">p &lt;-<span class="st"> </span><span class="kw"><a href="../reference/hyp_plot.html">hyp_plot</a></span>(multihyp_obj, <span class="dt">return_plots=</span><span class="ot">TRUE</span>)</a></code></pre></div>
<div class="sourceCode" id="cb44"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb44-1" title="1"><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/names">names</a></span>(p)</a></code></pre></div>
<pre><code>## [1] "YAP-KO Signature"      "YAP-KO Up-regulated"   "YAP-KO Down-regulated"</code></pre>
<div class="sourceCode" id="cb46"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb46-1" title="1">p[[<span class="dv">1</span>]]</a></code></pre></div>
<div id="htmlwidget-1ad0ee8c747537d2dffa" style="width:648px;height:480px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-1ad0ee8c747537d2dffa">{"x":{"visdat":{"8bd35c91d291":["function () ","plotlyVisDat"]},"cur_data":"8bd35c91d291","attrs":{"8bd35c91d291":{"x":{},"y":{},"orientation":"h","hoverinfo":"y","marker":{"color":"#4CA1AF","line":{"color":"white","width":1}},"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"type":"bar"},"8bd35c91d291.1":{"x":{},"y":{},"orientation":"h","hoverinfo":"y","marker":{"color":"#C4E0E5","line":{"color":"white","width":1}},"alpha_stroke":1,"sizes":[10,100],"spans":[1,20],"type":"bar","inherit":true}},"layout":{"margin":{"b":40,"l":300,"t":30,"r":0},"title":"YAP-KO Signature","xaxis":{"domain":[0.16,1],"automargin":true,"title":"-log<sub>10<\/sub>(FDR)","tickvals":[1.30102999566398,2,3,5,10,15,20,25,30,35,40,45,50,55,60,65,70,75,80,85,90,95,100,105,110,115,120,125,130,135,140,145,150,155,160,165,170,175,180,185,190,195,200,205,210,215,220,225,230,235,240,245,250,255,260,265,270,275,280,285,290,295,300,305,310,315,320,325,330,335,340,345,350,355,360,365,370,375,380,385,390,395,400,405,410,415,420,425,430,435,440,445,450,455,460,465,470,475,480,485,490,495,500,505,510,515,520,525,530,535,540,545,550,555,560,565,570,575,580,585,590,595,600,605,610,615,620,625,630,635,640,645,650,655,660,665,670,675,680,685,690,695,700,705,710,715,720,725,730,735,740,745,750,755,760,765,770,775,780,785,790,795,800,805,810,815,820,825,830,835,840,845,850,855,860,865,870,875,880,885,890,895,900,905,910,915,920,925,930,935,940,945,950,955,960,965,970,975,980,985,990,995,1000],"ticktext":["-log(0.05)","-log(0.01)","-log(0.001)"],"tickfont":{"size":9},"showgrid":true,"showline":true,"showticklabels":true,"zeroline":true},"yaxis":{"domain":[0,1],"automargin":true,"title":"","categoryarray":["FCERI mediated NF-kB activation","Regulation of actin dynamics for phagocytic cup formation","FCERI mediated MAPK activation","FCERI mediated Ca+2 mobilization","Scavenging of heme from plasma","Role of LAT2/NTAL/LAB on calcium mobilization","Role of phospholipids in phagocytosis","FCGR activation","Classical antibody-mediated complement activation","Regulation of Complement cascade"],"categoryorder":"array","showgrid":true,"showline":true,"showticklabels":false,"zeroline":true,"type":"category"},"barmode":"stack","paper_bgcolor":"white","plot_bgcolor":"white","showlegend":false,"annotations":[{"text":"Regulation of Complement cascade","xref":"paper","yref":"y","x":0.15,"y":"Regulation of Complement cascade","xanchor":"right","categoryorder":"array","font":{"family":"Arial","size":10,"color":"black"},"showarrow":false,"align":"right"},{"text":"Classical antibody-mediated complement activation","xref":"paper","yref":"y","x":0.15,"y":"Classical antibody-mediated complement activation","xanchor":"right","categoryorder":"array","font":{"family":"Arial","size":10,"color":"black"},"showarrow":false,"align":"right"},{"text":"FCGR activation","xref":"paper","yref":"y","x":0.15,"y":"FCGR activation","xanchor":"right","categoryorder":"array","font":{"family":"Arial","size":10,"color":"black"},"showarrow":false,"align":"right"},{"text":"Role of phospholipids in phagocytosis","xref":"paper","yref":"y","x":0.15,"y":"Role of phospholipids in phagocytosis","xanchor":"right","categoryorder":"array","font":{"family":"Arial","size":10,"color":"black"},"showarrow":false,"align":"right"},{"text":"Role of LAT2/NTAL/LAB on calcium mobilization","xref":"paper","yref":"y","x":0.15,"y":"Role of LAT2/NTAL/LAB on calcium mobilization","xanchor":"right","categoryorder":"array","font":{"family":"Arial","size":10,"color":"black"},"showarrow":false,"align":"right"},{"text":"Scavenging of heme from plasma","xref":"paper","yref":"y","x":0.15,"y":"Scavenging of heme from plasma","xanchor":"right","categoryorder":"array","font":{"family":"Arial","size":10,"color":"black"},"showarrow":false,"align":"right"},{"text":"FCERI mediated Ca+2 mobilization","xref":"paper","yref":"y","x":0.15,"y":"FCERI mediated Ca+2 mobilization","xanchor":"right","categoryorder":"array","font":{"family":"Arial","size":10,"color":"black"},"showarrow":false,"align":"right"},{"text":"FCERI mediated MAPK activation","xref":"paper","yref":"y","x":0.15,"y":"FCERI mediated MAPK activation","xanchor":"right","categoryorder":"array","font":{"family":"Arial","size":10,"color":"black"},"showarrow":false,"align":"right"},{"text":"Regulation of actin dynamics for phagocytic cup formation","xref":"paper","yref":"y","x":0.15,"y":"Regulation of actin dynamics for phagocytic cup formation","xanchor":"right","categoryorder":"array","font":{"family":"Arial","size":10,"color":"black"},"showarrow":false,"align":"right"},{"text":"FCERI mediated NF-kB activation","xref":"paper","yref":"y","x":0.15,"y":"FCERI mediated NF-kB activation","xanchor":"right","categoryorder":"array","font":{"family":"Arial","size":10,"color":"black"},"showarrow":false,"align":"right"}],"hovermode":"closest"},"source":"A","config":{"showSendToCloud":false},"data":[{"x":[209.635096068061,151.595058035539,120.746219696765,101.957434321629,111.603586864842,111.603586864842,91.6404672108926,90.5756317674934,70.3363568141601,56.0649113498837],"y":["Regulation of Complement cascade","Classical antibody-mediated complement activation","FCGR activation","Role of phospholipids in phagocytosis","Role of LAT2/NTAL/LAB on calcium mobilization","Scavenging of heme from plasma","FCERI mediated Ca+2 mobilization","FCERI mediated MAPK activation","Regulation of actin dynamics for phagocytic cup formation","FCERI mediated NF-kB activation"],"orientation":"h","hoverinfo":["y","y","y","y","y","y","y","y","y","y"],"marker":{"color":"#4CA1AF","line":{"color":"white","width":1}},"type":"bar","error_y":{"color":"rgba(31,119,180,1)"},"error_x":{"color":"rgba(31,119,180,1)"},"xaxis":"x","yaxis":"y","frame":null},{"x":[56.6014759383764,31.1496694593574,43.7486303249149,56.15047107568,43.9650493709986,43.9650493709986,56.9281690249484,57.6390383974958,75.4331942644616,77.3016201945367],"y":["Regulation of Complement cascade","Classical antibody-mediated complement activation","FCGR activation","Role of phospholipids in phagocytosis","Role of LAT2/NTAL/LAB on calcium mobilization","Scavenging of heme from plasma","FCERI mediated Ca+2 mobilization","FCERI mediated MAPK activation","Regulation of actin dynamics for phagocytic cup formation","FCERI mediated NF-kB activation"],"orientation":"h","hoverinfo":["y","y","y","y","y","y","y","y","y","y"],"marker":{"color":"#C4E0E5","line":{"color":"white","width":1}},"type":"bar","error_y":{"color":"rgba(255,127,14,1)"},"error_x":{"color":"rgba(255,127,14,1)"},"xaxis":"x","yaxis":"y","frame":null}],"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
</div>
<div id="enrichment-map-1" class="section level3">
<h3 class="hasAnchor">
<a href="#enrichment-map-1" class="anchor"></a>Enrichment map</h3>
<div class="sourceCode" id="cb47"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb47-1" title="1">p &lt;-<span class="st"> </span><span class="kw"><a href="../reference/hyp_emap.html">hyp_emap</a></span>(multihyp_obj, <span class="dt">top=</span><span class="dv">100</span>, <span class="dt">similarity_cutoff=</span><span class="fl">0.2</span>, <span class="dt">val=</span><span class="st">"pval"</span>, <span class="dt">multihyp_titles=</span><span class="ot">FALSE</span>, <span class="dt">return_plots=</span><span class="ot">TRUE</span>)</a></code></pre></div>
<div class="sourceCode" id="cb48"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb48-1" title="1"><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/names">names</a></span>(p)</a></code></pre></div>
<pre><code>## [1] "YAP-KO Signature"      "YAP-KO Up-regulated"   "YAP-KO Down-regulated"</code></pre>
<div class="sourceCode" id="cb50"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb50-1" title="1">p[[<span class="dv">1</span>]]</a></code></pre></div>
<div id="htmlwidget-9175a73a1f7f05b35f82" style="width:648px;height:432.632880098888px;" class="visNetwork html-widget"></div>
<script type="application/json" data-for="htmlwidget-9175a73a1f7f05b35f82">{"x":{"nodes":{"id":["Regulation of Complement cascade","Classical antibody-mediated complement activation","FCGR activation","Role of phospholipids in phagocytosis","Role of LAT2/NTAL/LAB on calcium mobilization","Scavenging of heme from plasma","FCERI mediated Ca+2 mobilization","FCERI mediated MAPK activation","Regulation of actin dynamics for phagocytic cup formation","FCERI mediated NF-kB activation","CD22 mediated BCR regulation","Immunoregulatory interactions between a Lymphoid and a non-Lymphoid cell","Antigen activates B Cell Receptor (BCR) leading to generation of second messengers","Terminal pathway of complement","Activation of C3 and C5","Transport of gamma-carboxylated protein precursors from the endoplasmic reticulum to the Golgi apparatus","Gamma-carboxylation of protein precursors","Removal of aminoterminal propeptides from gamma-carboxylated proteins","Common Pathway of Fibrin Clot Formation","Intrinsic Pathway of Fibrin Clot Formation","Regulation of Insulin-like Growth Factor (IGF) transport and uptake by Insulin-like Growth Factor Binding Proteins (IGFBPs)","Alternative complement activation","Neutrophil degranulation","COPI-mediated anterograde transport","Post-translational protein phosphorylation","Interleukin-4 and Interleukin-13 signaling","Metabolism of Angiotensinogen to Angiotensins","Syndecan interactions","Molecules associated with elastic fibres","Thrombin signalling through proteinase activated receptors (PARs)","Activation of Matrix Metalloproteinases","Cargo concentration in the ER","Collagen degradation","ECM proteoglycans","Integrin cell surface interactions","COPII-mediated vesicle transport","2-LTR circle formation","5-Phosphoribose 1-diphosphate biosynthesis","A tetrasaccharide linker sequence is required for GAG synthesis","ABC transporters in lipid homeostasis","ABO blood group biosynthesis","ADP signalling through P2Y purinoceptor 1","ADP signalling through P2Y purinoceptor 12","AKT phosphorylates targets in the cytosol","AKT phosphorylates targets in the nucleus","AKT-mediated inactivation of FOXO1A","ALKBH2 mediated reversal of alkylation damage","ALKBH3 mediated reversal of alkylation damage","AMPK inhibits chREBP transcriptional activation activity","APC truncation mutants are not K63 polyubiquitinated","APC truncation mutants have impaired AXIN binding","APC-Cdc20 mediated degradation of Nek2A","APC/C:Cdc20 mediated degradation of Cyclin B","APC/C:Cdc20 mediated degradation of Securin","APC/C:Cdh1 mediated degradation of Cdc20 and other APC/C:Cdh1 targeted proteins in late mitosis/early G1","APEX1-Independent Resolution of AP Sites via the Single Nucleotide Replacement Pathway","APOBEC3G mediated resistance to HIV-1 infection","ARL13B-mediated ciliary trafficking of INPP5E","ARMS-mediated activation","ATF4 activates genes in response to endoplasmic reticulum  stress","ATF6 (ATF6-alpha) activates chaperone genes","ATP sensitive Potassium channels","AUF1 (hnRNP D0) binds and destabilizes mRNA","AURKA Activation by TPX2","AXIN missense mutants destabilize the destruction complex","Abacavir metabolism","Abacavir transmembrane transport","Abasic sugar-phosphate removal via the single-nucleotide replacement pathway","Abnormal conversion of 2-oxoglutarate to 2-hydroxyglutarate","Abortive elongation of HIV-1 transcript in the absence of Tat","Acetylation","Acetylcholine Neurotransmitter Release Cycle","Acetylcholine regulates insulin secretion","Acrosome Reaction and Sperm:Oocyte Membrane Binding","Activated NOTCH1 Transmits Signal to the Nucleus","Activated NTRK2 signals through CDK5","Activated NTRK2 signals through FRS2 and FRS3","Activated NTRK2 signals through PI3K","Activated NTRK2 signals through PLCG1","Activated NTRK2 signals through RAS","Activated NTRK3 signals through PI3K","Activated NTRK3 signals through PLCG1","Activated NTRK3 signals through RAS","Activated PKN1 stimulates transcription of AR (androgen receptor) regulated genes KLK2 and KLK3","Activated point mutants of FGFR2","Activation and oligomerization of BAK protein","Activation of AKT2","Activation of AMPA receptors","Activation of AMPK downstream of NMDARs","Activation of ATR in response to replication stress","Activation of BMF and translocation to mitochondria","Activation of Ca-permeable Kainate Receptor","Activation of G protein gated Potassium channels","Activation of IRF3/IRF7 mediated by TBK1/IKK epsilon","Activation of NF-kappaB in B cells","Activation of NIMA Kinases NEK9, NEK6, NEK7","Activation of NOXA and translocation to mitochondria","Activation of Na-permeable kainate receptors","Activation of PPARGC1A (PGC-1alpha) by phosphorylation","Activation of PUMA and translocation to mitochondria"],"label":["Regulation of Complement cascade","Classical antibody-mediated complement activation","FCGR activation","Role of phospholipids in phagocytosis","Role of LAT2/NTAL/LAB on calcium mobilization","Scavenging of heme from plasma","FCERI mediated Ca+2 mobilization","FCERI mediated MAPK activation","Regulation of actin dynamics for phagocytic cup formation","FCERI mediated NF-kB activation","CD22 mediated BCR regulation","Immunoregulatory interactions between a Lymphoid and a non-Lymphoid cell","Antigen activates B Cell Receptor (BCR) leading to generation of second messengers","Terminal pathway of complement","Activation of C3 and C5","Transport of gamma-carboxylated protein precursors from the endoplasmic reticulum to the Golgi apparatus","Gamma-carboxylation of protein precursors","Removal of aminoterminal propeptides from gamma-carboxylated proteins","Common Pathway of Fibrin Clot Formation","Intrinsic Pathway of Fibrin Clot Formation","Regulation of Insulin-like Growth Factor (IGF) transport and uptake by Insulin-like Growth Factor Binding Proteins (IGFBPs)","Alternative complement activation","Neutrophil degranulation","COPI-mediated anterograde transport","Post-translational protein phosphorylation","Interleukin-4 and Interleukin-13 signaling","Metabolism of Angiotensinogen to Angiotensins","Syndecan interactions","Molecules associated with elastic fibres","Thrombin signalling through proteinase activated receptors (PARs)","Activation of Matrix Metalloproteinases","Cargo concentration in the ER","Collagen degradation","ECM proteoglycans","Integrin cell surface interactions","COPII-mediated vesicle transport","2-LTR circle formation","5-Phosphoribose 1-diphosphate biosynthesis","A tetrasaccharide linker sequence is required for GAG synthesis","ABC transporters in lipid homeostasis","ABO blood group biosynthesis","ADP signalling through P2Y purinoceptor 1","ADP signalling through P2Y purinoceptor 12","AKT phosphorylates targets in the cytosol","AKT phosphorylates targets in the nucleus","AKT-mediated inactivation of FOXO1A","ALKBH2 mediated reversal of alkylation damage","ALKBH3 mediated reversal of alkylation damage","AMPK inhibits chREBP transcriptional activation activity","APC truncation mutants are not K63 polyubiquitinated","APC truncation mutants have impaired AXIN binding","APC-Cdc20 mediated degradation of Nek2A","APC/C:Cdc20 mediated degradation of Cyclin B","APC/C:Cdc20 mediated degradation of Securin","APC/C:Cdh1 mediated degradation of Cdc20 and other APC/C:Cdh1 targeted proteins in late mitosis/early G1","APEX1-Independent Resolution of AP Sites via the Single Nucleotide Replacement Pathway","APOBEC3G mediated resistance to HIV-1 infection","ARL13B-mediated ciliary trafficking of INPP5E","ARMS-mediated activation","ATF4 activates genes in response to endoplasmic reticulum  stress","ATF6 (ATF6-alpha) activates chaperone genes","ATP sensitive Potassium channels","AUF1 (hnRNP D0) binds and destabilizes mRNA","AURKA Activation by TPX2","AXIN missense mutants destabilize the destruction complex","Abacavir metabolism","Abacavir transmembrane transport","Abasic sugar-phosphate removal via the single-nucleotide replacement pathway","Abnormal conversion of 2-oxoglutarate to 2-hydroxyglutarate","Abortive elongation of HIV-1 transcript in the absence of Tat","Acetylation","Acetylcholine Neurotransmitter Release Cycle","Acetylcholine regulates insulin secretion","Acrosome Reaction and Sperm:Oocyte Membrane Binding","Activated NOTCH1 Transmits Signal to the Nucleus","Activated NTRK2 signals through CDK5","Activated NTRK2 signals through FRS2 and FRS3","Activated NTRK2 signals through PI3K","Activated NTRK2 signals through PLCG1","Activated NTRK2 signals through RAS","Activated NTRK3 signals through PI3K","Activated NTRK3 signals through PLCG1","Activated NTRK3 signals through RAS","Activated PKN1 stimulates transcription of AR (androgen receptor) regulated genes KLK2 and KLK3","Activated point mutants of FGFR2","Activation and oligomerization of BAK protein","Activation of AKT2","Activation of AMPA receptors","Activation of AMPK downstream of NMDARs","Activation of ATR in response to replication stress","Activation of BMF and translocation to mitochondria","Activation of Ca-permeable Kainate Receptor","Activation of G protein gated Potassium channels","Activation of IRF3/IRF7 mediated by TBK1/IKK epsilon","Activation of NF-kappaB in B cells","Activation of NIMA Kinases NEK9, NEK6, NEK7","Activation of NOXA and translocation to mitochondria","Activation of Na-permeable kainate receptors","Activation of PPARGC1A (PGC-1alpha) by phosphorylation","Activation of PUMA and translocation to mitochondria"],"size":[27.907949790795,25.4602510460251,25.836820083682,26.652719665272,25.7112970711297,25.7112970711297,26.652719665272,26.7154811715481,28.9121338912134,29.7907949790795,24.3305439330544,33.5564853556485,25.8995815899582,20.4393305439331,20.3138075313808,20.5020920502092,20.5648535564854,20.5648535564854,21.3179916317992,21.3179916317992,27.7196652719665,20.1882845188285,50,25.1464435146444,26.652719665272,26.9665271966527,21.0669456066946,21.1924686192469,21.8200836820084,21.9456066945607,22.0083682008368,22.0083682008368,22.4476987447699,23.4518828451883,24.0794979079498,24.2050209205021,20.7531380753138,20.1255230125523,21.5690376569038,21.0669456066946,20.1255230125523,21.5062761506276,21.3179916317992,20.81589958159,20.5648535564854,20.1882845188285,20,20.1882845188285,20.4393305439331,20,20.81589958159,21.5062761506276,21.3807531380753,24.1422594142259,24.4560669456067,20.3765690376569,20.6276150627615,20.1255230125523,20.3765690376569,21.5690376569038,20.5648535564854,20.1882845188285,23.3891213389121,24.4560669456067,20.81589958159,20.2510460251046,20.2510460251046,20.0627615062761,20,21.3807531380753,20.0627615062761,21.0041841004184,20.5648535564854,20.3138075313808,21.8828451882845,20.3138075313808,20.6276150627615,20.3765690376569,20.1882845188285,20.5020920502092,20.3138075313808,20.1255230125523,20.4393305439331,22.1966527196653,21.0041841004184,20.0627615062761,20.1882845188285,20.1882845188285,20.5648535564854,22.2594142259414,20.1255230125523,20.5648535564854,21.5062761506276,21.0041841004184,24.1422594142259,20.3765690376569,20.2510460251046,20.0627615062761,20.5648535564854,20.5020920502092],"title":["P-Value: 3.8e-270","P-Value: 2.3e-186","P-Value: 6.1e-168","P-Value: 2e-161","P-Value: 1e-158","P-Value: 1e-158","P-Value: 1.2e-151","P-Value: 3.1e-151","P-Value: 9.7e-149","P-Value: 2.7e-136","P-Value: 9.4e-131","P-Value: 3.4e-130","P-Value: 4.9e-119","P-Value: 9.2e-18","P-Value: 8.9e-10","P-Value: 0.00028","P-Value: 0.00035","P-Value: 0.00035","P-Value: 0.0018","P-Value: 0.0018","P-Value: 0.0053","P-Value: 0.011","P-Value: 0.012","P-Value: 0.023","P-Value: 0.037","P-Value: 0.04","P-Value: 0.05","P-Value: 0.055","P-Value: 0.081","P-Value: 0.087","P-Value: 0.089","P-Value: 0.089","P-Value: 0.11","P-Value: 0.15","P-Value: 0.17","P-Value: 0.18","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1","P-Value: 1"],"color.border":["rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)"],"color.highlight":["rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)"],"color.background":["rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,1)","rgba(199,0,57,0.998)","rgba(199,0,57,0.998)","rgba(199,0,57,0.995)","rgba(199,0,57,0.989)","rgba(199,0,57,0.988)","rgba(199,0,57,0.977)","rgba(199,0,57,0.963)","rgba(199,0,57,0.96)","rgba(199,0,57,0.95)","rgba(199,0,57,0.945)","rgba(199,0,57,0.919)","rgba(199,0,57,0.913)","rgba(199,0,57,0.911)","rgba(199,0,57,0.911)","rgba(199,0,57,0.89)","rgba(199,0,57,0.85)","rgba(199,0,57,0.83)","rgba(199,0,57,0.82)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)","rgba(199,0,57,0)"],"x":[-0.749143825603696,-0.642264090556224,-0.75066837262187,-0.716682753860463,-0.78407550281311,-0.798247201452195,-0.717235925408753,-0.695118889569854,-0.694321705001024,-0.625246278455787,-0.653020669686361,-0.772690493914406,-0.648418926044192,0.443276329567978,-0.307583768744108,0.607492799848525,0.713311724449506,0.662801485625088,-0.0773288055544032,-0.18540108979339,-0.680163397047814,-0.399367820250134,0.803874827622088,0.431213087751133,-0.785803645483421,0.0223271735209036,0.633487131727551,0.149125677361414,0.837088705385326,0.134733611833389,0.159805198036105,0.302479806741189,0.183590902272567,-0.224163170295793,-0.112769736870141,0.389274748655608,0.00764529404236969,0.781751892185453,0.692587372681464,0.939503753999365,0.583463694979437,0.0580014366010413,0.0703701641116912,-0.507613516974268,-0.574276060697928,-0.517870956881989,0.893184499434002,0.748603643941779,-0.12701812121242,0.641942915322011,0.402625879086429,-0.322499428897884,-0.389417074006908,-0.437423269018092,-0.480112603654728,0.583553938713905,0.0347039035748864,0.815986139094824,0.591396823926519,0.421549643242323,0.514141687801411,0.822718604634676,-0.566893370622825,0.0207387270489521,0.419297124393193,1,-0.274353847774755,-0.0535114534392646,-0.289138511079443,0.442291938979569,-0.597985164237666,-0.132848651811945,-0.154206463597438,0.0593709772196824,0.993971326027517,-1,-0.932562343841602,-0.846535222273156,-0.946866065162486,-0.846808579869979,-0.69519557308316,-0.810555729874644,-0.874493600691013,0.638327485745728,0.274551359676346,0.235224377800636,0.957983707538221,0.45365863774921,-0.0807285332349291,0.217492643721524,-0.461882812642914,0.218959570600287,-0.00629992747148511,0.399358612331362,-0.46316232258284,-0.441498437168873,0.287243936034692,0.165755301614758,-0.184067821353204,0.253655328476796],"y":[-0.636384779460512,-0.627188762293324,-0.537619748155309,-0.483106415178455,-0.599535643415447,-0.550414662263846,-0.593953744163798,-0.644782610962508,-0.528753139658105,-0.477915105351378,-0.58968513371295,-0.494687078067654,-0.54751534309659,0.888703758547817,0.450912493661606,-0.537443396924757,-0.536767635428287,-0.440085607231543,0.232871389494045,0.255128563898272,0.667626820757343,0.502098960420078,0.496681897069204,0.485737574455207,0.668897847170409,0.439806461275061,-0.00861622708182996,0.990463071578843,0.0835327737021605,-0.942747791974557,-0.567597316213945,-0.736919560967764,-0.467393892777122,0.0253612258447742,-0.00754094903568614,-0.674455154474441,-0.333921278436654,-0.272861440366218,-0.731104375342177,0.370175267845402,0.591243086336348,-1,-0.879583537779703,0.312738074425052,0.104245553596717,0.203553733050139,-0.475240783130477,0.655933850181367,-0.664331106477876,0.391714435912301,-0.416559760274577,-0.280057380858498,-0.201343184618075,-0.34456668817459,-0.293299726479255,-0.188475573499869,-0.230751680291467,-0.0938837980634479,0.778257805786382,-0.897162221818344,-0.839906904267399,0.264858635691437,-0.31183928403844,0.644249413020211,-0.311328296743169,0.129391021846851,0.963345860303358,1,0.752699621826814,0.286116265539843,0.821289903563507,0.841398167109074,0.595560757834109,0.828328555591324,-0.08954680508713,-0.000298021899435397,0.23087806968847,0.123124450708715,0.116035629623078,0.220837012522956,0.480707321308251,0.451942894327684,0.344125342565233,0.19526994469062,0.878999653483294,0.477113035346747,-0.283158376592219,0.0657537530418359,-0.747605579364105,0.680256807110388,0.678697935454059,0.251332156300824,-0.941240721721324,0.692843807621282,-0.418309951156614,0.896303847931081,-0.0478337229785559,0.161011680780962,-0.773811043335791,-0.145659870243276]},"edges":{"from":["Regulation of Complement cascade","Regulation of Complement cascade","Regulation of Complement cascade","Regulation of Complement cascade","Regulation of Complement cascade","Regulation of Complement cascade","Regulation of Complement cascade","Regulation of Complement cascade","Regulation of Complement cascade","Regulation of Complement cascade","Regulation of Complement cascade","Regulation of Complement cascade","Classical antibody-mediated complement activation","Classical antibody-mediated complement activation","Classical antibody-mediated complement activation","Classical antibody-mediated complement activation","Classical antibody-mediated complement activation","Classical antibody-mediated complement activation","Classical antibody-mediated complement activation","Classical antibody-mediated complement activation","Classical antibody-mediated complement activation","Classical antibody-mediated complement activation","Classical antibody-mediated complement activation","FCGR activation","FCGR activation","FCGR activation","FCGR activation","FCGR activation","FCGR activation","FCGR activation","FCGR activation","FCGR activation","FCGR activation","Role of phospholipids in phagocytosis","Role of phospholipids in phagocytosis","Role of phospholipids in phagocytosis","Role of phospholipids in phagocytosis","Role of phospholipids in phagocytosis","Role of phospholipids in phagocytosis","Role of phospholipids in phagocytosis","Role of phospholipids in phagocytosis","Role of phospholipids in phagocytosis","Role of LAT2/NTAL/LAB on calcium mobilization","Role of LAT2/NTAL/LAB on calcium mobilization","Role of LAT2/NTAL/LAB on calcium mobilization","Role of LAT2/NTAL/LAB on calcium mobilization","Role of LAT2/NTAL/LAB on calcium mobilization","Role of LAT2/NTAL/LAB on calcium mobilization","Role of LAT2/NTAL/LAB on calcium mobilization","Role of LAT2/NTAL/LAB on calcium mobilization","Scavenging of heme from plasma","Scavenging of heme from plasma","Scavenging of heme from plasma","Scavenging of heme from plasma","Scavenging of heme from plasma","Scavenging of heme from plasma","Scavenging of heme from plasma","FCERI mediated Ca+2 mobilization","FCERI mediated Ca+2 mobilization","FCERI mediated Ca+2 mobilization","FCERI mediated Ca+2 mobilization","FCERI mediated Ca+2 mobilization","FCERI mediated Ca+2 mobilization","FCERI mediated MAPK activation","FCERI mediated MAPK activation","FCERI mediated MAPK activation","FCERI mediated MAPK activation","FCERI mediated MAPK activation","Regulation of actin dynamics for phagocytic cup formation","Regulation of actin dynamics for phagocytic cup formation","Regulation of actin dynamics for phagocytic cup formation","Regulation of actin dynamics for phagocytic cup formation","FCERI mediated NF-kB activation","FCERI mediated NF-kB activation","FCERI mediated NF-kB activation","FCERI mediated NF-kB activation","FCERI mediated NF-kB activation","FCERI mediated NF-kB activation","FCERI mediated NF-kB activation","CD22 mediated BCR regulation","CD22 mediated BCR regulation","Immunoregulatory interactions between a Lymphoid and a non-Lymphoid cell","Activation of C3 and C5","Transport of gamma-carboxylated protein precursors from the endoplasmic reticulum to the Golgi apparatus","Transport of gamma-carboxylated protein precursors from the endoplasmic reticulum to the Golgi apparatus","Gamma-carboxylation of protein precursors","Common Pathway of Fibrin Clot Formation","Regulation of Insulin-like Growth Factor (IGF) transport and uptake by Insulin-like Growth Factor Binding Proteins (IGFBPs)","Thrombin signalling through proteinase activated receptors (PARs)","Thrombin signalling through proteinase activated receptors (PARs)","Thrombin signalling through proteinase activated receptors (PARs)","Activation of Matrix Metalloproteinases","Cargo concentration in the ER","ECM proteoglycans","2-LTR circle formation","ADP signalling through P2Y purinoceptor 1","ADP signalling through P2Y purinoceptor 1","ADP signalling through P2Y purinoceptor 12","AKT phosphorylates targets in the cytosol","AKT phosphorylates targets in the nucleus","AMPK inhibits chREBP transcriptional activation activity","AMPK inhibits chREBP transcriptional activation activity","APC truncation mutants have impaired AXIN binding","APC-Cdc20 mediated degradation of Nek2A","APC-Cdc20 mediated degradation of Nek2A","APC-Cdc20 mediated degradation of Nek2A","APC/C:Cdc20 mediated degradation of Cyclin B","APC/C:Cdc20 mediated degradation of Cyclin B","APC/C:Cdc20 mediated degradation of Securin","APC/C:Cdc20 mediated degradation of Securin","APC/C:Cdc20 mediated degradation of Securin","APC/C:Cdh1 mediated degradation of Cdc20 and other APC/C:Cdh1 targeted proteins in late mitosis/early G1","APC/C:Cdh1 mediated degradation of Cdc20 and other APC/C:Cdh1 targeted proteins in late mitosis/early G1","ATF4 activates genes in response to endoplasmic reticulum  stress","AUF1 (hnRNP D0) binds and destabilizes mRNA","Activated NTRK2 signals through CDK5","Activated NTRK2 signals through FRS2 and FRS3","Activated NTRK2 signals through FRS2 and FRS3","Activated NTRK2 signals through FRS2 and FRS3","Activated NTRK2 signals through FRS2 and FRS3","Activated NTRK2 signals through PI3K","Activated NTRK2 signals through PI3K","Activated NTRK2 signals through PLCG1","Activated NTRK2 signals through RAS","Activated NTRK3 signals through PI3K","Activated NTRK3 signals through PLCG1","Activation of AMPK downstream of NMDARs","Activation of Ca-permeable Kainate Receptor","Activation of NOXA and translocation to mitochondria"],"to":["Classical antibody-mediated complement activation","FCGR activation","Role of phospholipids in phagocytosis","Role of LAT2/NTAL/LAB on calcium mobilization","Scavenging of heme from plasma","FCERI mediated Ca+2 mobilization","FCERI mediated MAPK activation","Regulation of actin dynamics for phagocytic cup formation","FCERI mediated NF-kB activation","CD22 mediated BCR regulation","Immunoregulatory interactions between a Lymphoid and a non-Lymphoid cell","Antigen activates B Cell Receptor (BCR) leading to generation of second messengers","FCGR activation","Role of phospholipids in phagocytosis","Role of LAT2/NTAL/LAB on calcium mobilization","Scavenging of heme from plasma","FCERI mediated Ca+2 mobilization","FCERI mediated MAPK activation","Regulation of actin dynamics for phagocytic cup formation","FCERI mediated NF-kB activation","CD22 mediated BCR regulation","Immunoregulatory interactions between a Lymphoid and a non-Lymphoid cell","Antigen activates B Cell Receptor (BCR) leading to generation of second messengers","Role of phospholipids in phagocytosis","Role of LAT2/NTAL/LAB on calcium mobilization","Scavenging of heme from plasma","FCERI mediated Ca+2 mobilization","FCERI mediated MAPK activation","Regulation of actin dynamics for phagocytic cup formation","FCERI mediated NF-kB activation","CD22 mediated BCR regulation","Immunoregulatory interactions between a Lymphoid and a non-Lymphoid cell","Antigen activates B Cell Receptor (BCR) leading to generation of second messengers","Role of LAT2/NTAL/LAB on calcium mobilization","Scavenging of heme from plasma","FCERI mediated Ca+2 mobilization","FCERI mediated MAPK activation","Regulation of actin dynamics for phagocytic cup formation","FCERI mediated NF-kB activation","CD22 mediated BCR regulation","Immunoregulatory interactions between a Lymphoid and a non-Lymphoid cell","Antigen activates B Cell Receptor (BCR) leading to generation of second messengers","Scavenging of heme from plasma","FCERI mediated Ca+2 mobilization","FCERI mediated MAPK activation","Regulation of actin dynamics for phagocytic cup formation","FCERI mediated NF-kB activation","CD22 mediated BCR regulation","Immunoregulatory interactions between a Lymphoid and a non-Lymphoid cell","Antigen activates B Cell Receptor (BCR) leading to generation of second messengers","FCERI mediated Ca+2 mobilization","FCERI mediated MAPK activation","Regulation of actin dynamics for phagocytic cup formation","FCERI mediated NF-kB activation","CD22 mediated BCR regulation","Immunoregulatory interactions between a Lymphoid and a non-Lymphoid cell","Antigen activates B Cell Receptor (BCR) leading to generation of second messengers","FCERI mediated MAPK activation","Regulation of actin dynamics for phagocytic cup formation","FCERI mediated NF-kB activation","CD22 mediated BCR regulation","Immunoregulatory interactions between a Lymphoid and a non-Lymphoid cell","Antigen activates B Cell Receptor (BCR) leading to generation of second messengers","Regulation of actin dynamics for phagocytic cup formation","FCERI mediated NF-kB activation","CD22 mediated BCR regulation","Immunoregulatory interactions between a Lymphoid and a non-Lymphoid cell","Antigen activates B Cell Receptor (BCR) leading to generation of second messengers","FCERI mediated NF-kB activation","CD22 mediated BCR regulation","Immunoregulatory interactions between a Lymphoid and a non-Lymphoid cell","Antigen activates B Cell Receptor (BCR) leading to generation of second messengers","CD22 mediated BCR regulation","Immunoregulatory interactions between a Lymphoid and a non-Lymphoid cell","Antigen activates B Cell Receptor (BCR) leading to generation of second messengers","APC/C:Cdc20 mediated degradation of Securin","APC/C:Cdh1 mediated degradation of Cdc20 and other APC/C:Cdh1 targeted proteins in late mitosis/early G1","AUF1 (hnRNP D0) binds and destabilizes mRNA","Activation of NF-kappaB in B cells","Immunoregulatory interactions between a Lymphoid and a non-Lymphoid cell","Antigen activates B Cell Receptor (BCR) leading to generation of second messengers","Antigen activates B Cell Receptor (BCR) leading to generation of second messengers","Alternative complement activation","Gamma-carboxylation of protein precursors","Removal of aminoterminal propeptides from gamma-carboxylated proteins","Removal of aminoterminal propeptides from gamma-carboxylated proteins","Intrinsic Pathway of Fibrin Clot Formation","Post-translational protein phosphorylation","ADP signalling through P2Y purinoceptor 1","ADP signalling through P2Y purinoceptor 12","Activation of G protein gated Potassium channels","Collagen degradation","COPII-mediated vesicle transport","Integrin cell surface interactions","APOBEC3G mediated resistance to HIV-1 infection","ADP signalling through P2Y purinoceptor 12","Activation of G protein gated Potassium channels","Activation of G protein gated Potassium channels","AKT-mediated inactivation of FOXO1A","AKT-mediated inactivation of FOXO1A","Activation of AMPK downstream of NMDARs","Activation of PPARGC1A (PGC-1alpha) by phosphorylation","AXIN missense mutants destabilize the destruction complex","APC/C:Cdc20 mediated degradation of Cyclin B","APC/C:Cdc20 mediated degradation of Securin","APC/C:Cdh1 mediated degradation of Cdc20 and other APC/C:Cdh1 targeted proteins in late mitosis/early G1","APC/C:Cdc20 mediated degradation of Securin","APC/C:Cdh1 mediated degradation of Cdc20 and other APC/C:Cdh1 targeted proteins in late mitosis/early G1","APC/C:Cdh1 mediated degradation of Cdc20 and other APC/C:Cdh1 targeted proteins in late mitosis/early G1","AUF1 (hnRNP D0) binds and destabilizes mRNA","Activation of NF-kappaB in B cells","AUF1 (hnRNP D0) binds and destabilizes mRNA","Activation of NF-kappaB in B cells","ATF6 (ATF6-alpha) activates chaperone genes","Activation of NF-kappaB in B cells","Activated NTRK2 signals through PLCG1","Activated NTRK2 signals through PI3K","Activated NTRK2 signals through PLCG1","Activated NTRK2 signals through RAS","Activated NTRK3 signals through RAS","Activated NTRK2 signals through PLCG1","Activated NTRK2 signals through RAS","Activated NTRK2 signals through RAS","Activated NTRK3 signals through RAS","Activated NTRK3 signals through PLCG1","Activated NTRK3 signals through RAS","Activation of PPARGC1A (PGC-1alpha) by phosphorylation","Activation of Na-permeable kainate receptors","Activation of PUMA and translocation to mitochondria"],"weight":[0.6796875,0.589928057553957,0.539473684210526,0.553191489361702,0.553191489361702,0.5,0.496815286624204,0.436170212765957,0.378640776699029,0.470149253731343,0.307984790874525,0.405063291139241,0.82,0.725663716814159,0.764705882352941,0.764705882352941,0.666666666666667,0.661016949152542,0.550335570469799,0.467065868263473,0.663157894736842,0.343612334801762,0.525,0.778761061946903,0.771428571428571,0.722222222222222,0.661157024793388,0.655737704918033,0.590604026845638,0.459302325581395,0.64,0.358078602620087,0.536585365853659,0.71551724137931,0.644628099173554,0.65891472868217,0.604477611940298,0.54320987654321,0.419354838709677,0.552631578947368,0.338842975206612,0.53030303030303,0.735849056603774,0.730434782608696,0.724137931034483,0.516129032258065,0.482142857142857,0.653061224489796,0.337662337662338,0.584745762711864,0.644628099173554,0.639344262295082,0.496815286624204,0.456140350877193,0.636363636363636,0.337662337662338,0.508064516129032,0.747967479674797,0.506024096385542,0.434782608695652,0.566371681415929,0.317073170731707,0.590551181102362,0.530487804878049,0.432432432432432,0.56140350877193,0.315789473684211,0.514925373134328,0.351351351351351,0.42,0.294964028776978,0.4,0.392638036809816,0.263513513513513,0.340425531914894,0.28735632183908,0.279329608938547,0.300613496932515,0.391304347826087,0.286995515695067,0.736842105263158,0.263157894736842,0.25,0.9,0.9,0.818181818181818,0.222222222222222,0.862903225806452,0.628571428571429,0.459459459459459,0.295454545454545,0.280701754385965,0.442857142857143,0.207920792079208,0.6,0.566666666666667,0.351351351351351,0.382352941176471,0.2,0.4,0.2,0.2,1,0.777777777777778,0.295774647887324,0.276315789473684,0.304347826086957,0.283783783783784,0.930555555555556,0.671232876712329,0.576470588235294,0.628205128205128,0.544444444444444,0.2,0.671232876712329,0.25,0.285714285714286,0.25,0.666666666666667,0.357142857142857,0.375,0.333333333333333,0.3,0.545454545454545,0.285714285714286,0.222222222222222,0.428571428571429,0.2,0.4],"value":[0.6796875,0.589928057553957,0.539473684210526,0.553191489361702,0.553191489361702,0.5,0.496815286624204,0.436170212765957,0.378640776699029,0.470149253731343,0.307984790874525,0.405063291139241,0.82,0.725663716814159,0.764705882352941,0.764705882352941,0.666666666666667,0.661016949152542,0.550335570469799,0.467065868263473,0.663157894736842,0.343612334801762,0.525,0.778761061946903,0.771428571428571,0.722222222222222,0.661157024793388,0.655737704918033,0.590604026845638,0.459302325581395,0.64,0.358078602620087,0.536585365853659,0.71551724137931,0.644628099173554,0.65891472868217,0.604477611940298,0.54320987654321,0.419354838709677,0.552631578947368,0.338842975206612,0.53030303030303,0.735849056603774,0.730434782608696,0.724137931034483,0.516129032258065,0.482142857142857,0.653061224489796,0.337662337662338,0.584745762711864,0.644628099173554,0.639344262295082,0.496815286624204,0.456140350877193,0.636363636363636,0.337662337662338,0.508064516129032,0.747967479674797,0.506024096385542,0.434782608695652,0.566371681415929,0.317073170731707,0.590551181102362,0.530487804878049,0.432432432432432,0.56140350877193,0.315789473684211,0.514925373134328,0.351351351351351,0.42,0.294964028776978,0.4,0.392638036809816,0.263513513513513,0.340425531914894,0.28735632183908,0.279329608938547,0.300613496932515,0.391304347826087,0.286995515695067,0.736842105263158,0.263157894736842,0.25,0.9,0.9,0.818181818181818,0.222222222222222,0.862903225806452,0.628571428571429,0.459459459459459,0.295454545454545,0.280701754385965,0.442857142857143,0.207920792079208,0.6,0.566666666666667,0.351351351351351,0.382352941176471,0.2,0.4,0.2,0.2,1,0.777777777777778,0.295774647887324,0.276315789473684,0.304347826086957,0.283783783783784,0.930555555555556,0.671232876712329,0.576470588235294,0.628205128205128,0.544444444444444,0.2,0.671232876712329,0.25,0.285714285714286,0.25,0.666666666666667,0.357142857142857,0.375,0.333333333333333,0.3,0.545454545454545,0.285714285714286,0.222222222222222,0.428571428571429,0.2,0.4]},"nodesToDataframe":true,"edgesToDataframe":true,"options":{"width":"100%","height":"100%","nodes":{"shape":"dot","borderWidth":1,"borderWidthSelected":0,"physics":false},"manipulation":{"enabled":false},"edges":{"color":"rgb(88,24,69)","smooth":false},"interaction":{"multiselect":true,"tooltipDelay":300},"physics":{"stabilization":false}},"groups":null,"width":null,"height":null,"idselection":{"enabled":false,"style":"width: 150px; height: 26px","useLabels":true,"main":"Select by id"},"byselection":{"enabled":false,"style":"width: 150px; height: 26px","multiple":false,"hideColor":"rgba(200,200,200,0.5)"},"main":{"text":"","style":"font-family:Helvetica"},"submain":null,"footer":null,"background":"rgba(0, 0, 0, 0)","highlight":{"enabled":true,"hoverNearest":false,"degree":1,"algorithm":"all","hideColor":"rgba(200,200,200,0.5)","labelOnly":true},"collapse":{"enabled":false,"fit":false,"resetHighlight":true,"clusterOptions":null,"keepCoord":true,"labelSuffix":"(cluster)"},"tooltipStay":300,"tooltipStyle":"position: fixed;visibility:hidden;padding: 5px;white-space: nowrap;font-family: verdana;font-size:14px;font-color:#000000;background-color: #f5f4ed;-moz-border-radius: 3px;-webkit-border-radius: 3px;border-radius: 3px;border: 1px solid #808074;box-shadow: 3px 3px 10px rgba(0, 0, 0, 0.2);","igraphlayout":{"type":"square"}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div id="multiple-experiments" class="section level2">
<h2 class="hasAnchor">
<a href="#multiple-experiments" class="anchor"></a>Multiple experiments</h2>
<p>Sometimes, we are generating reports for mutliple signatures across multiple experiments. Typically, the best way to manage this data is to process it all at once and generate a markdown report to analyze ourselves or share with our collaborators.</p>
<div class="sourceCode" id="cb51"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb51-1" title="1">signatures &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/readRDS">readRDS</a></span>(<span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/system.file">system.file</a></span>(<span class="st">"extdata/signatures.rds"</span>, <span class="dt">package=</span><span class="st">"hypeR"</span>))</a>
<a class="sourceLine" id="cb51-2" title="2">project &lt;-<span class="st"> </span>signatures<span class="op">$</span>project</a></code></pre></div>
<div class="sourceCode" id="cb52"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb52-1" title="1"><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/names">names</a></span>(project)</a></code></pre></div>
<pre><code>## [1] "YAP-KO Experiment" "TNF-KO Experiment" "TAZ-KO Experiment"</code></pre>
<div class="sourceCode" id="cb54"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb54-1" title="1"><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/names">names</a></span>(project[[<span class="dv">1</span>]])</a></code></pre></div>
<pre><code>## [1] "YAP-KO Signature"      "YAP-KO Up-regulated"   "YAP-KO Down-regulated"</code></pre>
<div class="sourceCode" id="cb56"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb56-1" title="1"><span class="kw"><a href="https://www.rdocumentation.org/packages/utils/topics/head">head</a></span>(project[[<span class="dv">1</span>]][[<span class="dv">1</span>]])</a></code></pre></div>
<pre><code>## [1] "ELOC"   "TRIM71" "KLHL13" "HECTD3" "PSMD6"  "WSB1"</code></pre>
<div class="sourceCode" id="cb58"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb58-1" title="1">lmultihyp_obj &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/lapply">lapply</a></span>(project, <span class="cf">function</span>(x) <span class="kw"><a href="../reference/hypeR.html">hypeR</a></span>(x, gsets, <span class="dt">bg=</span><span class="dv">35423</span>))</a></code></pre></div>
<div class="sourceCode" id="cb59"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb59-1" title="1"><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/names">names</a></span>(lmultihyp_obj)</a></code></pre></div>
<pre><code>## [1] "YAP-KO Experiment" "TNF-KO Experiment" "TAZ-KO Experiment"</code></pre>
<div class="sourceCode" id="cb61"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb61-1" title="1"><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/print">print</a></span>(lmultihyp_obj[[<span class="dv">1</span>]])</a></code></pre></div>
<pre><code>## multihyp object: 
## YAP-KO Signature 
## hyp object: 
##   data: 1552 x 8
##   args: symbols
##         gsets
##         gsets_relational
##         bg
##         min_drawsize
##         pval_cutoff
##         fdr_cutoff
##         verbosemultihyp object: 
## YAP-KO Up-regulated 
## hyp object: 
##   data: 1552 x 8
##   args: symbols
##         gsets
##         gsets_relational
##         bg
##         min_drawsize
##         pval_cutoff
##         fdr_cutoff
##         verbosemultihyp object: 
## YAP-KO Down-regulated 
## hyp object: 
##   data: 1552 x 8
##   args: symbols
##         gsets
##         gsets_relational
##         bg
##         min_drawsize
##         pval_cutoff
##         fdr_cutoff
##         verbose</code></pre>
</div>
<div id="markdown-reports" class="section level2">
<h2 class="hasAnchor">
<a href="#markdown-reports" class="anchor"></a>Markdown reports</h2>
<p><strong><em>hyp_to_rmd</em></strong> will accept multiple formats, including a single <strong><em>hyp</em></strong> or <strong><em>multihyp</em></strong> objects as well as a list of either, including a list of <strong><em>hyp</em></strong> or <strong><em>multihyp</em></strong> objects together. When a list of <strong><em>multihyp</em></strong> objects are passed for example, each experiment will become its own section, while each signature becomes its own tab within that section. Lists of keyword arguments can be passed for <strong><em>hyp_plot</em></strong>, <strong><em>hyp_emap</em></strong>, and <strong><em>hyp_hmap</em></strong>, allowing customization of their functionality per report.</p>
<div class="sourceCode" id="cb63"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb63-1" title="1"><span class="kw"><a href="../reference/hyp_to_rmd.html">hyp_to_rmd</a></span>(lmultihyp_obj,</a>
<a class="sourceLine" id="cb63-2" title="2">           <span class="dt">file_path=</span><span class="st">"hyper-enrichment.rmd"</span>,</a>
<a class="sourceLine" id="cb63-3" title="3">           <span class="dt">title=</span><span class="st">"Hyper Enrichment (hypeR)"</span>,</a>
<a class="sourceLine" id="cb63-4" title="4">           <span class="dt">subtitle=</span><span class="st">"YAP, TNF, and TAZ Knockout Experiments"</span>,</a>
<a class="sourceLine" id="cb63-5" title="5">           <span class="dt">author=</span><span class="st">"Anthony Federico, Stefano Monti"</span>,</a>
<a class="sourceLine" id="cb63-6" title="6">           <span class="dt">show_plots=</span>T,</a>
<a class="sourceLine" id="cb63-7" title="7">           <span class="dt">show_emaps=</span>T,</a>
<a class="sourceLine" id="cb63-8" title="8">           <span class="dt">show_hmaps=</span>T,</a>
<a class="sourceLine" id="cb63-9" title="9">           <span class="dt">show_tables=</span>T)</a></code></pre></div>
</div>
</div>
<div id="session-info" class="section level1">
<h1 class="hasAnchor">
<a href="#session-info" class="anchor"></a>Session Info</h1>
<div class="sourceCode" id="cb64"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb64-1" title="1"><span class="kw"><a href="https://www.rdocumentation.org/packages/utils/topics/sessionInfo">sessionInfo</a></span>()</a></code></pre></div>
<pre><code>## R version 3.6.0 (2019-04-26)
## Platform: x86_64-apple-darwin15.6.0 (64-bit)
## Running under: macOS Mojave 10.14.3
## 
## Matrix products: default
## BLAS:   /Library/Frameworks/R.framework/Versions/3.6/Resources/lib/libRblas.0.dylib
## LAPACK: /Library/Frameworks/R.framework/Versions/3.6/Resources/lib/libRlapack.dylib
## 
## locale:
## [1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8
## 
## attached base packages:
## [1] stats     graphics  grDevices utils     datasets  methods   base     
## 
## other attached packages:
## [1] hypeR_1.10.1     testthat_2.1.1   BiocStyle_2.10.0
## 
## loaded via a namespace (and not attached):
##  [1] httr_1.4.0         pkgload_1.0.2      msigdbr_6.2.1     
##  [4] tidyr_0.8.3        jsonlite_1.6       viridisLite_0.3.0 
##  [7] shiny_1.3.2        assertthat_0.2.1   BiocManager_1.30.4
## [10] yaml_2.2.0         remotes_2.0.4      sessioninfo_1.1.1 
## [13] pillar_1.3.1       backports_1.1.4    glue_1.3.1        
## [16] digest_0.6.18      promises_1.0.1     colorspace_1.4-1  
## [19] htmltools_0.3.6    httpuv_1.5.1       plyr_1.8.4        
## [22] pkgconfig_2.0.2    devtools_2.0.2     bookdown_0.9      
## [25] xtable_1.8-4       purrr_0.3.2        scales_1.0.0      
## [28] processx_3.3.0     openxlsx_4.1.0     later_0.8.0       
## [31] tibble_2.1.1       ggplot2_3.1.1      usethis_1.5.0     
## [34] DT_0.5             withr_2.1.2        lazyeval_0.2.2    
## [37] cli_1.1.0          magrittr_1.5       crayon_1.3.4      
## [40] mime_0.6           memoise_1.1.0      evaluate_0.13     
## [43] ps_1.3.0           fs_1.2.7           MASS_7.3-51.4     
## [46] xml2_1.2.0         pkgbuild_1.0.3     tools_3.6.0       
## [49] gh_1.0.1           data.table_1.12.2  prettyunits_1.0.2 
## [52] stringr_1.4.0      plotly_4.9.0       munsell_0.5.0     
## [55] zip_2.0.1          callr_3.2.0        compiler_3.6.0    
## [58] pkgdown_1.3.0      rlang_0.3.4        grid_3.6.0        
## [61] rstudioapi_0.10    htmlwidgets_1.3    visNetwork_2.0.6  
## [64] crosstalk_1.0.0    igraph_1.2.4.1     rmarkdown_1.12    
## [67] gtable_0.3.0       roxygen2_6.1.1     R6_2.4.0          
## [70] knitr_1.22         dplyr_0.8.0.1      commonmark_1.7    
## [73] rprojroot_1.3-2    desc_1.2.0         stringi_1.4.3     
## [76] Rcpp_1.0.1         tidyselect_0.2.5   xfun_0.6</code></pre>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="sidebar">
        <div id="tocnav">
      <h2 class="hasAnchor">
<a href="#tocnav" class="anchor"></a>Contents</h2>
      <ul class="nav nav-pills nav-stacked">
<li><a href="#introduction">Introduction</a></li>
      <li><a href="#installation">Installation</a></li>
      <li>
<a href="#basics">Basics</a><ul class="nav nav-pills nav-stacked">
<li><a href="#example-data">Example Data</a></li>
      <li><a href="#hyper-enrichment">Hyper enrichment</a></li>
      <li><a href="#visualizations">Visualizations</a></li>
      </ul>
</li>
      <li>
<a href="#advanced">Advanced</a><ul class="nav nav-pills nav-stacked">
<li><a href="#relational-genesets">Relational genesets</a></li>
      <li><a href="#multiple-signatures">Multiple signatures</a></li>
      <li><a href="#multiple-experiments">Multiple experiments</a></li>
      <li><a href="#markdown-reports">Markdown reports</a></li>
      </ul>
</li>
      <li><a href="#session-info">Session Info</a></li>
      </ul>
</div>
      </div>

</div>


      <footer><div class="copyright">
  <p>Developed by <a href="https://github.com/anfederico">Anthony Federico</a>, <a href="https://github.com/montilab">Stefano Monti</a>.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.3.0.</p>
</div>
      </footer>
</div>

  

  </body>
</html>
