<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>hypeR • hypeR</title>
<!-- favicons --><link rel="icon" type="image/png" sizes="16x16" href="../favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="../favicon-32x32.png">
<link rel="apple-touch-icon" type="image/png" sizes="180x180" href="../apple-touch-icon.png">
<link rel="apple-touch-icon" type="image/png" sizes="120x120" href="../apple-touch-icon-120x120.png">
<link rel="apple-touch-icon" type="image/png" sizes="76x76" href="../apple-touch-icon-76x76.png">
<link rel="apple-touch-icon" type="image/png" sizes="60x60" href="../apple-touch-icon-60x60.png">
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script><!-- Bootstrap --><link href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.3.7/cosmo/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha256-U5ZEeKfGNOja007MMD3YBI0A3OSZOQbeG6z2f2Y0hu8=" crossorigin="anonymous"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" integrity="sha256-eZrrJcwDc/3uDhsdt61sL2oOBY362qM3lon1gyExkL0=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.4/clipboard.min.js" integrity="sha256-FiZwavyI2V6+EXO1U+xzLG3IKldpiTFf3153ea9zikQ=" crossorigin="anonymous"></script><!-- sticky kit --><script src="https://cdnjs.cloudflare.com/ajax/libs/sticky-kit/1.1.3/sticky-kit.min.js" integrity="sha256-c4Rlo1ZozqTPE2RLuvbusY3+SU1pQaJC0TjuhygMipw=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="hypeR">
<meta property="og:description" content="">
<meta name="twitter:card" content="summary">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body>
    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">hypeR</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">1.1.9</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../index.html">Home</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Docs
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/docs/installation.html">Installation</a>
    </li>
    <li>
      <a href="../articles/docs/basics.html">Basic Usage</a>
    </li>
    <li>
      <a href="../articles/docs/data.html">Downloading Data</a>
    </li>
    <li>
      <a href="../articles/docs/visualize.html">Visualize Results</a>
    </li>
    <li>
      <a href="../articles/docs/rgsets.html">Constructing Relational Genesets</a>
    </li>
    <li>
      <a href="../articles/docs/multihyp.html">Multiple Signatures</a>
    </li>
    <li>
      <a href="../articles/docs/reporting.html">Generating Reports</a>
    </li>
    <li>
      <a href="../articles/docs/shiny.html">Shiny Applications</a>
    </li>
    <li>
      <a href="../articles/docs/extending.html">Extending hypeR</a>
    </li>
  </ul>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Examples
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/examples/workflow.html">Co-Expression Modules</a>
    </li>
  </ul>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/montilab/hypeR">
    <span class="fa fa-github"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      
      </header><script src="hypeR_files/htmlwidgets-1.3/htmlwidgets.js"></script><link href="hypeR_files/vis-4.20.1/vis.css" rel="stylesheet">
<script src="hypeR_files/vis-4.20.1/vis.min.js"></script><script src="hypeR_files/visNetwork-binding-2.0.7/visNetwork.js"></script><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1>hypeR</h1>
                        <h4 class="author">Anthony Federico</h4>
            <address class="author_afil">
      Boston University School of Medicine, Boston, MA<br><h4 class="author">Stefano Monti</h4>
            <address class="author_afil">
      Boston University School of Medicine, Boston, MA<br><h4 class="date">August 11, 2019</h4>
      
      <small class="dont-index">Source: <a href="https://github.com/montilab/hypeR/blob/master/vignettes/hypeR.Rmd"><code>vignettes/hypeR.Rmd</code></a></small>
      <div class="hidden name"><code>hypeR.Rmd</code></div>

    </address>
</address>
</div>

    
    
<div id="introduction" class="section level1">
<h1 class="hasAnchor">
<a href="#introduction" class="anchor"></a>Introduction</h1>
<p>Geneset enrichment is an important step in biological data analysis workflows, particularly in bioinformatics and computational biology. At a basic level, one is performing a hypergeometric or Kol-mogorov–Smirnov test to determine if a group of genes is over-represented or enriched, respectively, in pre-defined sets of genes, which suggests some biological relevance. The R package hypeR brings a fresh take to geneset enrichment, focusing on the analysis, visualization, and reporting of enriched genesets. While similar tools exists - such as Enrichr (Kuleshov et al., 2016), fgsea (Sergushichev, 2016), and clusterProfiler (Wang et al., 2012), among others - hypeR excels in the downstream analysis of gene-set enrichment workflows – in addition to sometimes overlooked upstream analysis methods such as allowing for a flexible back-ground population size or reducing genesets to a background distribution of genes. Finding relevant biological meaning from a large number of often obscurely labeled genesets may be challenging for researchers. hypeR overcomes this barrier by incorporating hierarchical ontologies - also referred to as relational genesets - into its workflows, allowing researchers to visualize and summarize their data at varying levels of biological resolution. All analysis methods are compatible with hypeR’s markdown features, enabling concise and reproducible reports easily shareable with collaborators. Additionally, users can import custom genesets that are easily defined, extending the analysis of genes to other areas of interest such as proteins, microbes, metabolites, etc. The hypeR package goes beyond performing basic enrichment, by providing a suite of methods designed to make routine geneset enrichment seamless for scientists working in R.</p>
</div>
<div id="installation" class="section level1">
<h1 class="hasAnchor">
<a href="#installation" class="anchor"></a>Installation</h1>
<p>Download the package from Bioconductor.</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1"></a><span class="cf">if</span> (<span class="op">!</span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/ns-load">requireNamespace</a></span>(<span class="st">"BiocManager"</span>, <span class="dt">quietly =</span> <span class="ot">TRUE</span>))</span>
<span id="cb1-2"><a href="#cb1-2"></a><span class="kw"><a href="https://www.rdocumentation.org/packages/utils/topics/install.packages">install.packages</a></span>(<span class="st">"BiocManager"</span>)</span>
<span id="cb1-3"><a href="#cb1-3"></a>BiocManager<span class="op">::</span><span class="kw"><a href="https://www.rdocumentation.org/packages/BiocManager/topics/install">install</a></span>(<span class="st">"hypeR"</span>)</span></code></pre></div>
<p>Install the development version of the package from Github.</p>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb2-1"><a href="#cb2-1"></a>devtools<span class="op">::</span><span class="kw"><a href="https://www.rdocumentation.org/packages/devtools/topics/reexports">install_github</a></span>(<span class="st">"montilab/hypeR"</span>)</span></code></pre></div>
<p>Or install the development version of the package from Bioconductor.</p>
<div class="sourceCode" id="cb3"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb3-1"><a href="#cb3-1"></a>BiocManager<span class="op">::</span><span class="kw"><a href="https://www.rdocumentation.org/packages/BiocManager/topics/install">install</a></span>(<span class="st">"montilab/hypeR"</span>, <span class="dt">version=</span><span class="st">'devel'</span>)</span></code></pre></div>
<p>Load the package into R session.</p>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb4-1"><a href="#cb4-1"></a><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/library">library</a></span>(hypeR)</span></code></pre></div>
</div>
<div id="basics" class="section level1">
<h1 class="hasAnchor">
<a href="#basics" class="anchor"></a>Basics</h1>
<div id="terminology" class="section level2">
<h2 class="hasAnchor">
<a href="#terminology" class="anchor"></a>Terminology</h2>
<div id="signature" class="section level3">
<h3 class="hasAnchor">
<a href="#signature" class="anchor"></a>Signature</h3>
<p><strong>hypeR</strong> employs multiple types of enrichment analyses (e.g. hypergeometric, kstest, gsea). Depending on the type, different kinds of signatures are expected. There are three types of signatures <code><a href="../reference/hypeR.html">hypeR()</a></code> expects.</p>
<div class="sourceCode" id="cb5"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb5-1"><a href="#cb5-1"></a><span class="co"># Simply a character vector of symbols (hypergeometric)</span></span>
<span id="cb5-2"><a href="#cb5-2"></a>signature &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/c">c</a></span>(<span class="st">"GENE1"</span>, <span class="st">"GENE2"</span>, <span class="st">"GENE3"</span>)</span>
<span id="cb5-3"><a href="#cb5-3"></a></span>
<span id="cb5-4"><a href="#cb5-4"></a><span class="co"># A pre-ranked character vector of symbols (kstest)</span></span>
<span id="cb5-5"><a href="#cb5-5"></a>ranked.signature &lt;-<span class="st">  </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/c">c</a></span>(<span class="st">"GENE1"</span>, <span class="st">"GENE2"</span>, <span class="st">"GENE3"</span>)</span>
<span id="cb5-6"><a href="#cb5-6"></a></span>
<span id="cb5-7"><a href="#cb5-7"></a><span class="co"># A pre-ranked named numerical vector of symbols with ranking weights (gsea)</span></span>
<span id="cb5-8"><a href="#cb5-8"></a>weighted.signature &lt;-<span class="st">  </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/c">c</a></span>(<span class="st">"GENE1"</span>=<span class="fl">1.22</span>, <span class="st">"GENE2"</span>=<span class="fl">0.94</span>, <span class="st">"GENE3"</span>=<span class="fl">0.77</span>)</span></code></pre></div>
</div>
<div id="geneset" class="section level3">
<h3 class="hasAnchor">
<a href="#geneset" class="anchor"></a>Geneset</h3>
<p>A geneset is simply a list of vectors, therefore, one can use any custom geneset in their analyses, as long as it’s appropriately defined. In our tutorials, we will use genesets from <a href="https://reactome.org/">REACTOME</a>. There is also what is called relational genesets, whereby genesets are organized into a hiearchy; we will explore these in later tutorials.</p>
<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb6-1"><a href="#cb6-1"></a>genesets &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/list">list</a></span>(<span class="st">"GSET1"</span> =<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/c">c</a></span>(<span class="st">"GENE1"</span>, <span class="st">"GENE2"</span>, <span class="st">"GENE3"</span>),</span>
<span id="cb6-2"><a href="#cb6-2"></a>                 <span class="st">"GSET2"</span> =<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/c">c</a></span>(<span class="st">"GENE4"</span>, <span class="st">"GENE5"</span>, <span class="st">"GENE6"</span>),</span>
<span id="cb6-3"><a href="#cb6-3"></a>                 <span class="st">"GSET3"</span> =<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/c">c</a></span>(<span class="st">"GENE7"</span>, <span class="st">"GENE8"</span>, <span class="st">"GENE9"</span>))</span></code></pre></div>
</div>
</div>
<div id="example-data" class="section level2">
<h2 class="hasAnchor">
<a href="#example-data" class="anchor"></a>Example Data</h2>
<p>In these tutorials, we will use example data. The example data includes an expression set object as well as pre-computed results from common workflows such as diffential expression and weighted gene co-expression analyses.</p>
<div class="sourceCode" id="cb7"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb7-1"><a href="#cb7-1"></a>hypdat &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/readRDS">readRDS</a></span>(<span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/file.path">file.path</a></span>(<span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/system.file">system.file</a></span>(<span class="st">"extdata"</span>, <span class="dt">package=</span><span class="st">"hypeR"</span>), <span class="st">"hypdat.rds"</span>))</span></code></pre></div>
<div id="defining-signatures-and-genesets" class="section level3">
<h3 class="hasAnchor">
<a href="#defining-signatures-and-genesets" class="anchor"></a>Defining Signatures and Genesets</h3>
<p>Using a differential expression dataframe created with limma, we will extract a signature of upregulated genes for use with a hypergeometric test and rank genes descending by their differential expression level for use with a kstest. We’ll also import genesets from <a href="https://www.kegg.jp">KEGG</a>.</p>
<div class="sourceCode" id="cb8"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb8-1"><a href="#cb8-1"></a>limma &lt;-<span class="st"> </span>hypdat<span class="op">$</span>limma</span>
<span id="cb8-2"><a href="#cb8-2"></a>signature &lt;-<span class="st"> </span>limma <span class="op">%&gt;%</span><span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/stats/topics/filter">filter</a></span>(t <span class="op">&gt;</span><span class="st"> </span><span class="dv">0</span> <span class="op">&amp;</span><span class="st"> </span>fdr <span class="op">&lt;</span><span class="st"> </span><span class="fl">0.001</span>) <span class="op">%&gt;%</span><span class="st"> </span><span class="kw">use_series</span>(symbol)</span>
<span id="cb8-3"><a href="#cb8-3"></a>ranked.signature &lt;-<span class="st"> </span>limma <span class="op">%&gt;%</span><span class="st"> </span><span class="kw">arrange</span>(<span class="kw">desc</span>(t)) <span class="op">%&gt;%</span><span class="st"> </span><span class="kw">use_series</span>(symbol)</span>
<span id="cb8-4"><a href="#cb8-4"></a></span>
<span id="cb8-5"><a href="#cb8-5"></a>gsets &lt;-<span class="st"> </span><span class="kw"><a href="../reference/hyperdb_fetch.html">hyperdb_fetch</a></span>(<span class="dt">type=</span><span class="st">"gsets"</span>, <span class="st">"KEGG_2019_Human"</span>)</span></code></pre></div>
</div>
</div>
<div id="basic-usage" class="section level2">
<h2 class="hasAnchor">
<a href="#basic-usage" class="anchor"></a>Basic Usage</h2>
<p>All workflows begin with performing hyper enrichment with <code><a href="../reference/hypeR.html">hypeR()</a></code>. Often we’re just interested in a single signature, as described above. In this case, <code><a href="../reference/hypeR.html">hypeR()</a></code> will return a <code>hyp</code> object. This object contains relevant information to the enrichment results and is recognized by downstream methods.</p>
<div id="overrepresentation-analysis" class="section level3">
<h3 class="hasAnchor">
<a href="#overrepresentation-analysis" class="anchor"></a>Overrepresentation analysis</h3>
<div class="sourceCode" id="cb9"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb9-1"><a href="#cb9-1"></a>hyp_obj &lt;-<span class="st"> </span><span class="kw"><a href="../reference/hypeR.html">hypeR</a></span>(signature, gsets, <span class="dt">test=</span><span class="st">"hypergeometric"</span>, <span class="dt">bg=</span><span class="dv">50000</span>, <span class="dt">fdr_cutoff=</span><span class="fl">0.01</span>, <span class="dt">do_plots=</span><span class="ot">TRUE</span>)</span></code></pre></div>
<div class="sourceCode" id="cb10"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb10-1"><a href="#cb10-1"></a><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/print">print</a></span>(hyp_obj)</span></code></pre></div>
<pre><code>## hyp object: 
##   data: 67 x 6
##   plots: 67
##   args: signature
##         gsets
##         test
##         bg
##         pval_cutoff
##         fdr_cutoff
##         weights_pwr
##         absolute
##         do_plots
##         verbose</code></pre>
<div class="sourceCode" id="cb12"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb12-1"><a href="#cb12-1"></a>hyp_df &lt;-<span class="st"> </span>hyp_obj<span class="op">$</span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/as.data.frame">as.data.frame</a></span>()</span>
<span id="cb12-2"><a href="#cb12-2"></a><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/print">print</a></span>(<span class="kw"><a href="https://www.rdocumentation.org/packages/utils/topics/head">head</a></span>(hyp_df[,<span class="dv">1</span><span class="op">:</span><span class="dv">3</span>]), <span class="dt">row.names=</span><span class="ot">FALSE</span>)</span></code></pre></div>
<pre><code>##                           label    pval     fdr
##                  Tight junction 1.2e-11 3.7e-09
##            Salmonella infection 3.8e-09 5.8e-07
##          MAPK signaling pathway 6.6e-09 6.8e-07
##                     Endocytosis 9.9e-09 7.6e-07
##  Neurotrophin signaling pathway 5.0e-08 3.1e-06
##           Ras signaling pathway 7.2e-08 3.7e-06</code></pre>
<div class="sourceCode" id="cb14"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb14-1"><a href="#cb14-1"></a>hyp_obj<span class="op">$</span>plots[[<span class="dv">1</span>]]</span></code></pre></div>
<p><img src="hypeR_files/figure-html/unnamed-chunk-10-1.png" width="648"></p>
</div>
<div id="enrichment-analysis" class="section level3">
<h3 class="hasAnchor">
<a href="#enrichment-analysis" class="anchor"></a>Enrichment analysis</h3>
<div class="sourceCode" id="cb15"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb15-1"><a href="#cb15-1"></a>hyp_obj &lt;-<span class="st"> </span><span class="kw"><a href="../reference/hypeR.html">hypeR</a></span>(ranked.signature, gsets, <span class="dt">test=</span><span class="st">"kstest"</span>, <span class="dt">fdr_cutoff=</span><span class="fl">0.01</span>, <span class="dt">do_plots=</span><span class="ot">TRUE</span>)</span></code></pre></div>
<div class="sourceCode" id="cb16"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb16-1"><a href="#cb16-1"></a><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/print">print</a></span>(hyp_obj)</span></code></pre></div>
<pre><code>## hyp object: 
##   data: 6 x 6
##   plots: 6
##   args: signature
##         gsets
##         test
##         bg
##         pval_cutoff
##         fdr_cutoff
##         weights_pwr
##         absolute
##         do_plots
##         verbose</code></pre>
<div class="sourceCode" id="cb18"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb18-1"><a href="#cb18-1"></a>hyp_df &lt;-<span class="st"> </span>hyp_obj<span class="op">$</span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/as.data.frame">as.data.frame</a></span>()</span>
<span id="cb18-2"><a href="#cb18-2"></a><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/print">print</a></span>(<span class="kw"><a href="https://www.rdocumentation.org/packages/utils/topics/head">head</a></span>(hyp_df[,<span class="dv">1</span><span class="op">:</span><span class="dv">3</span>]), <span class="dt">row.names=</span><span class="ot">FALSE</span>)</span></code></pre></div>
<pre><code>##                           label    pval     fdr
##                  Tight junction 1.4e-09 4.3e-07
##                      Proteasome 1.7e-05 2.6e-03
##            Viral carcinogenesis 4.1e-05 3.9e-03
##  Sphingolipid signaling pathway 5.3e-05 3.9e-03
##            Steroid biosynthesis 6.4e-05 3.9e-03
##    Longevity regulating pathway 1.0e-04 5.1e-03</code></pre>
<div class="sourceCode" id="cb20"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb20-1"><a href="#cb20-1"></a>hyp_obj<span class="op">$</span>plots[[<span class="dv">1</span>]]</span></code></pre></div>
<p><img src="hypeR_files/figure-html/unnamed-chunk-12-1.png" width="648"></p>
</div>
<div id="save-results-to-excel" class="section level3">
<h3 class="hasAnchor">
<a href="#save-results-to-excel" class="anchor"></a>Save results to excel</h3>
<div class="sourceCode" id="cb21"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb21-1"><a href="#cb21-1"></a><span class="kw"><a href="../reference/hyp_to_excel.html">hyp_to_excel</a></span>(hyp_obj, <span class="dt">file_path=</span><span class="st">"hyper.xlsx"</span>)</span></code></pre></div>
</div>
<div id="save-results-to-table" class="section level3">
<h3 class="hasAnchor">
<a href="#save-results-to-table" class="anchor"></a>Save results to table</h3>
<div class="sourceCode" id="cb22"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb22-1"><a href="#cb22-1"></a><span class="kw"><a href="../reference/hyp_to_table.html">hyp_to_table</a></span>(hyp_obj, <span class="dt">file_path=</span><span class="st">"hyper.txt"</span>)</span></code></pre></div>
</div>
</div>
</div>
<div id="downloading-data" class="section level1">
<h1 class="hasAnchor">
<a href="#downloading-data" class="anchor"></a>Downloading Data</h1>
<div id="downloading-genesets-from-msigdb" class="section level2">
<h2 class="hasAnchor">
<a href="#downloading-genesets-from-msigdb" class="anchor"></a>Downloading genesets from msigdb</h2>
<p>For most purposes, the genesets hosted by <a href="https://software.broadinstitute.org/gsea/msigdb/collections.jsp">msigdb</a> are more than adequate to perform geneset enrichment analysis. There are various types of genesets available across many species. Therefore, we have added convenient functions for retrieving msigdb data compatible with <strong>hypeR</strong>.</p>
<div id="available-genesets" class="section level3">
<h3 class="hasAnchor">
<a href="#available-genesets" class="anchor"></a>Available genesets</h3>
<div class="sourceCode" id="cb23"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb23-1"><a href="#cb23-1"></a><span class="kw"><a href="../reference/msigdb_info.html">msigdb_info</a></span>()</span></code></pre></div>
<pre><code>## |------------------------------------------------------------|
## | Available Gene Sets                                 v6.2.1 |
## |------------------------------------------------------------|
## | C1             | Positional (326)                          |
## | C2.CGP         | Chemical and Genetic Perturbations (3433) |
## | C2.CP          | Canonical Pathways (252)                  |
## | C2.CP.BIOCARTA | Canonical BIOCARTA (217)                  |
## | C2.CP.KEGG     | Canonical KEGG (186)                      |
## | C2.CP.REACTOME | Canonical REACTOME (674)                  |
## | C3.MIR         | Motif miRNA Targets (221)                 |
## | C3.TFT         | Motif Transcription Factor Targets (615)  |
## | C4.CGN         | Cancer Gene Neighborhoods (427)           |
## | C4.CM          | Cancer Modules (431)                      |
## | C5.BP          | GO Biological Process (4436)              |
## | C5.CC          | GO Cellular Component (580)               |
## | C5.MF          | GO Molecular Function (901)               |
## | C6             | Oncogenic Signatures (189)                |
## | C7             | Immunologic Signatures (4872)             |
## | H              | Hallmark (50)                             |
## |------------------------------------------------------------|
## | Source: http://software.broadinstitute.org/gsea/msigdb     |
## |------------------------------------------------------------|</code></pre>
</div>
<div id="downloading-and-loading-genesets" class="section level3">
<h3 class="hasAnchor">
<a href="#downloading-and-loading-genesets" class="anchor"></a>Downloading and loading genesets</h3>
<p>Use <code><a href="../reference/msigdb_download_one.html">msigdb_download_one()</a></code> to download a single geneset into memory.</p>
<div class="sourceCode" id="cb25"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb25-1"><a href="#cb25-1"></a>HALLMARK &lt;-<span class="st"> </span><span class="kw"><a href="../reference/msigdb_download_one.html">msigdb_download_one</a></span>(<span class="dt">species=</span><span class="st">"Homo sapiens"</span>, <span class="dt">category=</span><span class="st">"H"</span>)</span></code></pre></div>
<div class="sourceCode" id="cb26"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb26-1"><a href="#cb26-1"></a><span class="kw"><a href="https://www.rdocumentation.org/packages/utils/topics/head">head</a></span>(<span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/names">names</a></span>(HALLMARK))</span></code></pre></div>
<pre><code>## [1] "HALLMARK_ADIPOGENESIS"        "HALLMARK_ALLOGRAFT_REJECTION"
## [3] "HALLMARK_ANDROGEN_RESPONSE"   "HALLMARK_ANGIOGENESIS"       
## [5] "HALLMARK_APICAL_JUNCTION"     "HALLMARK_APICAL_SURFACE"</code></pre>
<div class="sourceCode" id="cb28"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb28-1"><a href="#cb28-1"></a><span class="kw"><a href="https://www.rdocumentation.org/packages/utils/topics/head">head</a></span>(HALLMARK[[<span class="dv">1</span>]])</span></code></pre></div>
<pre><code>## [1] "ABCA1" "ABCB8" "ACAA2" "ACADL" "ACADM" "ACADS"</code></pre>
<p>Use <code><a href="../reference/msigdb_download_all.html">msigdb_download_all()</a></code> to retrieve all genesets for a given species. By default, genesets are cached in a temporary directory, therefore a path object is returned. Use the path object with <code><a href="../reference/msigdb_fetch.html">msigdb_fetch()</a></code> to load genesets into memory. Users can also specify a directory to download to for repeated use.</p>
<div class="sourceCode" id="cb30"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb30-1"><a href="#cb30-1"></a>msigdb_path &lt;-<span class="st"> </span><span class="kw"><a href="../reference/msigdb_download_all.html">msigdb_download_all</a></span>(<span class="dt">species=</span><span class="st">"Homo sapiens"</span>)</span></code></pre></div>
<pre><code>## v6.2.1
## Downloading genesets to...
## /var/folders/7j/bg701dss38lc2xq26vt_by240000gn/T//Rtmpy9oYdX
## - C1 -&gt; 326 Gene Sets 
## - C2.CGP -&gt; 3433 Gene Sets 
## - C2.CP -&gt; 252 Gene Sets 
## - C2.CP.BIOCARTA -&gt; 217 Gene Sets 
## - C2.CP.KEGG -&gt; 186 Gene Sets 
## - C2.CP.REACTOME -&gt; 674 Gene Sets 
## - C3.MIR -&gt; 221 Gene Sets 
## - C3.TFT -&gt; 615 Gene Sets 
## - C4.CGN -&gt; 427 Gene Sets 
## - C4.CM -&gt; 431 Gene Sets 
## - C5.BP -&gt; 4436 Gene Sets 
## - C5.CC -&gt; 580 Gene Sets 
## - C5.MF -&gt; 901 Gene Sets 
## - C6 -&gt; 189 Gene Sets 
## - C7 -&gt; 4872 Gene Sets 
## - H -&gt; 50 Gene Sets</code></pre>
<div class="sourceCode" id="cb32"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb32-1"><a href="#cb32-1"></a>BIOCARTA &lt;-<span class="st"> </span><span class="kw"><a href="../reference/msigdb_fetch.html">msigdb_fetch</a></span>(msigdb_path, <span class="st">"C2.CP.BIOCARTA"</span>)</span>
<span id="cb32-2"><a href="#cb32-2"></a>KEGG     &lt;-<span class="st"> </span><span class="kw"><a href="../reference/msigdb_fetch.html">msigdb_fetch</a></span>(msigdb_path, <span class="st">"C2.CP.KEGG"</span>)</span>
<span id="cb32-3"><a href="#cb32-3"></a>REACTOME &lt;-<span class="st"> </span><span class="kw"><a href="../reference/msigdb_fetch.html">msigdb_fetch</a></span>(msigdb_path, <span class="st">"C2.CP.REACTOME"</span>)</span></code></pre></div>
<p>In this example, we are interested in all three of the following genesets, therefore we concatenate them. A geneset is simply a named list of vectors, therefore, one can use any custom genesets in their analysis, as long as it’s appropriately defined.</p>
<div class="sourceCode" id="cb33"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb33-1"><a href="#cb33-1"></a>gsets &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/c">c</a></span>(BIOCARTA, KEGG, REACTOME)</span></code></pre></div>
</div>
</div>
<div id="defining-custom-genesets" class="section level2">
<h2 class="hasAnchor">
<a href="#defining-custom-genesets" class="anchor"></a>Defining custom genesets</h2>
<p>As mentioned previously, one can use custom genesets with <strong>hypeR</strong>. In this example, we download one of the many publicly available genesets hosted by <a href="https://amp.pharm.mssm.edu/Enrichr/">Enrichr</a>. Once downloaded, one performs enrichment as normal.</p>
<div class="sourceCode" id="cb34"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb34-1"><a href="#cb34-1"></a>url =<span class="st"> "http://amp.pharm.mssm.edu/Enrichr/geneSetLibrary?mode=text&amp;libraryName=Cancer_Cell_Line_Encyclopedia"</span></span>
<span id="cb34-2"><a href="#cb34-2"></a>r &lt;-<span class="st"> </span>httr<span class="op">::</span><span class="kw"><a href="https://httr.r-lib.org/reference/GET.html">GET</a></span>(url)</span>
<span id="cb34-3"><a href="#cb34-3"></a>text &lt;-<span class="st"> </span>httr<span class="op">::</span><span class="kw"><a href="https://httr.r-lib.org/reference/content.html">content</a></span>(r, <span class="st">"text"</span>, <span class="dt">encoding=</span><span class="st">"ISO-8859-1"</span>)</span>
<span id="cb34-4"><a href="#cb34-4"></a>text.split &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/strsplit">strsplit</a></span>(text, <span class="st">"</span><span class="ch">\n</span><span class="st">"</span>)[[<span class="dv">1</span>]]</span>
<span id="cb34-5"><a href="#cb34-5"></a>gsets &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/lapply">sapply</a></span>(text.split, <span class="cf">function</span>(x) {</span>
<span id="cb34-6"><a href="#cb34-6"></a>    genes &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/strsplit">strsplit</a></span>(x, <span class="st">"</span><span class="ch">\t</span><span class="st">"</span>)[[<span class="dv">1</span>]]</span>
<span id="cb34-7"><a href="#cb34-7"></a>    <span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/function">return</a></span>(genes[<span class="dv">3</span><span class="op">:</span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/length">length</a></span>(genes)])</span>
<span id="cb34-8"><a href="#cb34-8"></a>})</span>
<span id="cb34-9"><a href="#cb34-9"></a><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/names">names</a></span>(gsets) &lt;-<span class="st"> </span><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/unlist">unlist</a></span>(<span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/lapply">lapply</a></span>(text.split, <span class="cf">function</span>(x) <span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/strsplit">strsplit</a></span>(x, <span class="st">"</span><span class="ch">\t</span><span class="st">"</span>)[[<span class="dv">1</span>]][<span class="dv">1</span>]))</span></code></pre></div>
</div>
<div id="downloading-genesets-from-hyperdb" class="section level2">
<h2 class="hasAnchor">
<a href="#downloading-genesets-from-hyperdb" class="anchor"></a>Downloading genesets from hyperdb</h2>
<p>If msigdb genesets are not sufficient, we have also provided another set of functions for downloading and loading other open source genesets. Essentially, we wrap the method used in the example defining custom genesets. This is facilitated by interfacing with the publicly available <a href="https://amp.pharm.mssm.edu/Enrichr/#stats">libraries</a> hosted by <a href="https://amp.pharm.mssm.edu/Enrichr/">Enrichr</a>.</p>
<div id="downloading-and-loading-genesets-1" class="section level4">
<h4 class="hasAnchor">
<a href="#downloading-and-loading-genesets-1" class="anchor"></a>Downloading and loading genesets</h4>
<div class="sourceCode" id="cb35"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb35-1"><a href="#cb35-1"></a>gsets &lt;-<span class="st"> </span><span class="kw"><a href="../reference/hyperdb_fetch.html">hyperdb_fetch</a></span>(<span class="dt">type=</span><span class="st">"gsets"</span>, <span class="st">"Cancer_Cell_Line_Encyclopedia"</span>)</span></code></pre></div>
</div>
<div id="available-genesets-1" class="section level4">
<h4 class="hasAnchor">
<a href="#available-genesets-1" class="anchor"></a>Available genesets</h4>
<div class="sourceCode" id="cb36"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb36-1"><a href="#cb36-1"></a><span class="kw"><a href="../reference/hyperdb_info.html">hyperdb_info</a></span>(<span class="dt">type=</span><span class="st">"gsets"</span>, <span class="dt">quiet=</span><span class="ot">TRUE</span>)[<span class="dv">1</span><span class="op">:</span><span class="dv">15</span>]</span></code></pre></div>
<pre><code>## Getting available gsets ...</code></pre>
<pre><code>##  [1] "ARCHS4_Cell-lines"                
##  [2] "ARCHS4_IDG_Coexp"                 
##  [3] "ARCHS4_Kinases_Coexp"             
##  [4] "ARCHS4_TFs_Coexp"                 
##  [5] "ARCHS4_Tissues"                   
##  [6] "Achilles_fitness_decrease"        
##  [7] "Achilles_fitness_increase"        
##  [8] "Aging_Perturbations_from_GEO_down"
##  [9] "Aging_Perturbations_from_GEO_up"  
## [10] "Allen_Brain_Atlas_down"           
## [11] "Allen_Brain_Atlas_up"             
## [12] "BioCarta_2013"                    
## [13] "BioCarta_2015"                    
## [14] "BioCarta_2016"                    
## [15] "BioPlex_2017"</code></pre>
</div>
<div id="available-relational-genesets" class="section level4">
<h4 class="hasAnchor">
<a href="#available-relational-genesets" class="anchor"></a>Available relational genesets</h4>
<div class="sourceCode" id="cb39"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb39-1"><a href="#cb39-1"></a><span class="kw"><a href="../reference/hyperdb_info.html">hyperdb_info</a></span>(<span class="dt">type=</span><span class="st">"rgsets"</span>, <span class="dt">quiet=</span><span class="ot">TRUE</span>)</span></code></pre></div>
<pre><code>## Getting available rgsets ...</code></pre>
<pre><code>## [1] "KEGG"     "REACTOME"</code></pre>
</div>
</div>
</div>
<div id="visualize-results" class="section level1">
<h1 class="hasAnchor">
<a href="#visualize-results" class="anchor"></a>Visualize Results</h1>
<div id="example-data-1" class="section level2">
<h2 class="hasAnchor">
<a href="#example-data-1" class="anchor"></a>Example Data</h2>
<div class="sourceCode" id="cb42"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb42-1"><a href="#cb42-1"></a>limma &lt;-<span class="st"> </span>hypdat<span class="op">$</span>limma</span>
<span id="cb42-2"><a href="#cb42-2"></a>ranked.signature &lt;-<span class="st"> </span>limma <span class="op">%&gt;%</span><span class="st"> </span><span class="kw">arrange</span>(<span class="kw">desc</span>(t)) <span class="op">%&gt;%</span><span class="st"> </span><span class="kw">use_series</span>(symbol)</span>
<span id="cb42-3"><a href="#cb42-3"></a></span>
<span id="cb42-4"><a href="#cb42-4"></a>rgsets &lt;-<span class="st"> </span><span class="kw"><a href="../reference/hyperdb_fetch.html">hyperdb_fetch</a></span>(<span class="dt">type=</span><span class="st">"rgsets"</span>, <span class="st">"REACTOME"</span>)</span>
<span id="cb42-5"><a href="#cb42-5"></a>gsets &lt;-<span class="st"> </span>rgsets<span class="op">$</span>gsets</span></code></pre></div>
</div>
<div id="hyper-enrichment" class="section level2">
<h2 class="hasAnchor">
<a href="#hyper-enrichment" class="anchor"></a>Hyper enrichment</h2>
<p>All workflows begin with performing hyper enrichment with <code><a href="../reference/hypeR.html">hypeR()</a></code>.</p>
<div class="sourceCode" id="cb43"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb43-1"><a href="#cb43-1"></a>hyp_obj &lt;-<span class="st"> </span><span class="kw"><a href="../reference/hypeR.html">hypeR</a></span>(ranked.signature, gsets, <span class="dt">test=</span><span class="st">"kstest"</span>, <span class="dt">fdr_cutoff=</span><span class="fl">0.01</span>)</span></code></pre></div>
</div>
<div id="dots-plot" class="section level2">
<h2 class="hasAnchor">
<a href="#dots-plot" class="anchor"></a>Dots plot</h2>
<p>One can visualize the top enriched genesets using <code><a href="../reference/hyp_dots.html">hyp_dots()</a></code> which returns a horizontal dots plot. Each dot is a geneset, where the color represents the significance and the size signifies the geneset size.</p>
<div class="sourceCode" id="cb44"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb44-1"><a href="#cb44-1"></a><span class="kw"><a href="../reference/hyp_dots.html">hyp_dots</a></span>(hyp_obj, <span class="dt">show_plots=</span><span class="ot">FALSE</span>, <span class="dt">return_plots=</span><span class="ot">TRUE</span>)</span></code></pre></div>
<p><img src="hypeR_files/figure-html/unnamed-chunk-26-1.png" width="648"></p>
</div>
<div id="enrichment-map" class="section level2">
<h2 class="hasAnchor">
<a href="#enrichment-map" class="anchor"></a>Enrichment map</h2>
<p>One can visualize the top enriched genesets using <code><a href="../reference/hyp_emap.html">hyp_emap()</a></code> which will return an enrichment map. Each node represents a geneset, where the shade of red indicates the normalized significance of enrichment. Hover over the node to view the raw value. Edges represent geneset similarity, calculated by either jaccard or overlap similarity metrics.</p>
<div class="sourceCode" id="cb45"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb45-1"><a href="#cb45-1"></a><span class="kw"><a href="../reference/hyp_emap.html">hyp_emap</a></span>(hyp_obj, <span class="dt">similarity_cutoff=</span><span class="fl">0.8</span>, <span class="dt">show_plots=</span><span class="ot">FALSE</span>, <span class="dt">return_plots=</span><span class="ot">TRUE</span>)</span></code></pre></div>
<div id="htmlwidget-4d5c487c0b93ae52dbc1" style="width:648px;height:672px;" class="visNetwork html-widget"></div>
<script type="application/json" data-for="htmlwidget-4d5c487c0b93ae52dbc1">{"x":{"nodes":{"id":["MAPK6/MAPK4 signaling","Separation of Sister Chromatids","SCF-beta-TrCP mediated degradation of Emi1","NIK-->noncanonical NF-kB signaling","The role of GTSE1 in G2/M progression after G2 checkpoint","AUF1 (hnRNP D0) binds and destabilizes mRNA","Vpu mediated degradation of CD4","Negative regulation of NOTCH4 signaling","Dectin-1 mediated noncanonical NF-kB signaling","Ub-specific processing proteases","Autodegradation of the E3 ubiquitin ligase COP1","Regulation of activated PAK-2p34 by proteasome mediated degradation","APC/C:Cdh1 mediated degradation of Cdc20 and other APC/C:Cdh1 targeted proteins in late mitosis/early G1","Degradation of AXIN","Oxygen-dependent proline hydroxylation of Hypoxia-inducible Factor Alpha","Degradation of DVL","FBXL7 down-regulates AURKA during mitotic entry and in early mitosis","Ubiquitin-dependent degradation of Cyclin D","Vif-mediated degradation of APOBEC3G","Asymmetric localization of PCP proteins","Regulation of PLK1 Activity at G2/M Transition","Ubiquitin Mediated Degradation of Phosphorylated Cdc25A","Regulation of RAS by GAPs","Degradation of GLI1 by the proteasome","Degradation of GLI2 by the proteasome","GLI3 is processed to GLI3R by the proteasome","RHO GTPases Activate Formins","Regulation of RUNX3 expression and activity","FCERI mediated NF-kB activation","Regulation of ornithine decarboxylase (ODC)","Cross-presentation of soluble exogenous antigens (endosomes)","Autodegradation of Cdh1 by Cdh1:APC/C","Cdc20:Phospho-APC/C mediated degradation of Cyclin A","UCH proteinases","APC/C:Cdc20 mediated degradation of Securin","Hh mutants that don't undergo autocatalytic processing are degraded by ERAD","CDT1 association with the CDC6:ORC:origin complex","Activation of NF-kappaB in B cells","Orc1 removal from chromatin","AURKA Activation by TPX2","MHC class II antigen presentation","Translocation of SLC2A4 (GLUT4) to the plasma membrane","Neutrophil degranulation","Loss of Nlp from mitotic centrosomes","Regulation of PTEN stability and activity","CDK-mediated phosphorylation and removal of Cdc6","Recycling pathway of L1","Defective CFTR causes cystic fibrosis","Amplification  of signal from unattached  kinetochores via a MAD2  inhibitory signal","PKMTs methylate histone lysines","Downstream TCR signaling","ER-Phagosome pathway","Formation of tubulin folding intermediates by CCT/TriC","SCF(Skp2)-mediated degradation of p27/p21","Resolution of Sister Chromatid Cohesion","Recruitment of NuMA to mitotic centrosomes","Recruitment of mitotic centrosome proteins and complexes","Activation of gene expression by SREBF (SREBP)","TBC/RABGAPs","Basigin interactions","Signaling by BRAF and RAF fusions","Antigen processing: Ubiquitination & Proteasome degradation"],"label":["MAPK6/MAPK4 signaling","Separation of Sister Chromatids","SCF-beta-TrCP mediated degradation of Emi1","NIK-->noncanonical NF-kB signaling","The role of GTSE1 in G2/M progression after G2 checkpoint","AUF1 (hnRNP D0) binds and destabilizes mRNA","Vpu mediated degradation of CD4","Negative regulation of NOTCH4 signaling","Dectin-1 mediated noncanonical NF-kB signaling","Ub-specific processing proteases","Autodegradation of the E3 ubiquitin ligase COP1","Regulation of activated PAK-2p34 by proteasome mediated degradation","APC/C:Cdh1 mediated degradation of Cdc20 and other APC/C:Cdh1 targeted proteins in late mitosis/early G1","Degradation of AXIN","Oxygen-dependent proline hydroxylation of Hypoxia-inducible Factor Alpha","Degradation of DVL","FBXL7 down-regulates AURKA during mitotic entry and in early mitosis","Ubiquitin-dependent degradation of Cyclin D","Vif-mediated degradation of APOBEC3G","Asymmetric localization of PCP proteins","Regulation of PLK1 Activity at G2/M Transition","Ubiquitin Mediated Degradation of Phosphorylated Cdc25A","Regulation of RAS by GAPs","Degradation of GLI1 by the proteasome","Degradation of GLI2 by the proteasome","GLI3 is processed to GLI3R by the proteasome","RHO GTPases Activate Formins","Regulation of RUNX3 expression and activity","FCERI mediated NF-kB activation","Regulation of ornithine decarboxylase (ODC)","Cross-presentation of soluble exogenous antigens (endosomes)","Autodegradation of Cdh1 by Cdh1:APC/C","Cdc20:Phospho-APC/C mediated degradation of Cyclin A","UCH proteinases","APC/C:Cdc20 mediated degradation of Securin","Hh mutants that don't undergo autocatalytic processing are degraded by ERAD","CDT1 association with the CDC6:ORC:origin complex","Activation of NF-kappaB in B cells","Orc1 removal from chromatin","AURKA Activation by TPX2","MHC class II antigen presentation","Translocation of SLC2A4 (GLUT4) to the plasma membrane","Neutrophil degranulation","Loss of Nlp from mitotic centrosomes","Regulation of PTEN stability and activity","CDK-mediated phosphorylation and removal of Cdc6","Recycling pathway of L1","Defective CFTR causes cystic fibrosis","Amplification  of signal from unattached  kinetochores via a MAD2  inhibitory signal","PKMTs methylate histone lysines","Downstream TCR signaling","ER-Phagosome pathway","Formation of tubulin folding intermediates by CCT/TriC","SCF(Skp2)-mediated degradation of p27/p21","Resolution of Sister Chromatid Cohesion","Recruitment of NuMA to mitotic centrosomes","Recruitment of mitotic centrosome proteins and complexes","Activation of gene expression by SREBF (SREBP)","TBC/RABGAPs","Basigin interactions","Signaling by BRAF and RAF fusions","Antigen processing: Ubiquitination & Proteasome degradation"],"size":[24.5594713656388,29.647577092511,21.9823788546256,22.2466960352423,22.3127753303965,21.9823788546256,21.8502202643172,21.9162995594714,22.4449339207048,31.8942731277533,21.784140969163,21.6519823788546,23.1057268722467,21.9823788546256,22.7092511013216,22.1145374449339,21.9823788546256,21.784140969163,21.9823788546256,22.5770925110132,24.0969162995595,21.784140969163,22.84140969163,22.3127753303965,22.3127753303965,22.3127753303965,26.3436123348018,21.9823788546256,28.7224669603524,21.7180616740088,21.6519823788546,22.511013215859,23.1057268722467,24.6916299559471,22.7753303964758,22.0484581497797,22.2466960352423,22.7753303964758,23.0396475770925,23.1057268722467,25.4845814977974,21.7180616740088,50,22.9074889867841,22.9074889867841,23.1057268722467,20.1982378854626,22.3788546255507,24.5594713656388,21.4537444933921,25.2202643171806,24.0308370044053,20.0660792951542,22.3127753303965,25.4185022026432,23.6343612334802,23.7004405286344,21.1894273127753,21.3215859030837,20,22.511013215859,38.7665198237885],"title":["FDR: 4.8e-08","FDR: 8.3e-06","FDR: 8.3e-06","FDR: 1e-05","FDR: 1e-05","FDR: 1e-05","FDR: 1e-05","FDR: 1.4e-05","FDR: 1.4e-05","FDR: 1.7e-05","FDR: 1.7e-05","FDR: 1.7e-05","FDR: 1.7e-05","FDR: 4e-05","FDR: 7e-05","FDR: 7e-05","FDR: 7.1e-05","FDR: 7.1e-05","FDR: 7.6e-05","FDR: 8.5e-05","FDR: 8.5e-05","FDR: 8.5e-05","FDR: 8.9e-05","FDR: 8.9e-05","FDR: 8.9e-05","FDR: 8.9e-05","FDR: 8.9e-05","FDR: 8.9e-05","FDR: 9.7e-05","FDR: 9.9e-05","FDR: 0.00013","FDR: 0.00014","FDR: 0.00015","FDR: 0.00015","FDR: 0.00017","FDR: 0.00018","FDR: 0.00018","FDR: 0.00019","FDR: 0.00034","FDR: 0.00036","FDR: 0.00056","FDR: 0.00056","FDR: 0.00058","FDR: 0.00067","FDR: 0.00069","FDR: 0.00093","FDR: 0.00093","FDR: 0.001","FDR: 0.0014","FDR: 0.0016","FDR: 0.0021","FDR: 0.0029","FDR: 0.0029","FDR: 0.0029","FDR: 0.0068","FDR: 0.0071","FDR: 0.0071","FDR: 0.0072","FDR: 0.0072","FDR: 0.0089","FDR: 0.01","FDR: 0.01"],"color.border":["rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)","rgb(0,0,0)"],"color.highlight":["rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)"],"color.background":["rgba(199,0,57,1)","rgba(199,0,57,0.999)","rgba(199,0,57,0.999)","rgba(199,0,57,0.999)","rgba(199,0,57,0.999)","rgba(199,0,57,0.999)","rgba(199,0,57,0.999)","rgba(199,0,57,0.999)","rgba(199,0,57,0.999)","rgba(199,0,57,0.998)","rgba(199,0,57,0.998)","rgba(199,0,57,0.998)","rgba(199,0,57,0.998)","rgba(199,0,57,0.996)","rgba(199,0,57,0.993)","rgba(199,0,57,0.993)","rgba(199,0,57,0.993)","rgba(199,0,57,0.993)","rgba(199,0,57,0.992)","rgba(199,0,57,0.992)","rgba(199,0,57,0.992)","rgba(199,0,57,0.992)","rgba(199,0,57,0.991)","rgba(199,0,57,0.991)","rgba(199,0,57,0.991)","rgba(199,0,57,0.991)","rgba(199,0,57,0.991)","rgba(199,0,57,0.991)","rgba(199,0,57,0.99)","rgba(199,0,57,0.99)","rgba(199,0,57,0.987)","rgba(199,0,57,0.986)","rgba(199,0,57,0.985)","rgba(199,0,57,0.985)","rgba(199,0,57,0.983)","rgba(199,0,57,0.982)","rgba(199,0,57,0.982)","rgba(199,0,57,0.981)","rgba(199,0,57,0.966)","rgba(199,0,57,0.964)","rgba(199,0,57,0.944)","rgba(199,0,57,0.944)","rgba(199,0,57,0.942)","rgba(199,0,57,0.933)","rgba(199,0,57,0.931)","rgba(199,0,57,0.907)","rgba(199,0,57,0.907)","rgba(199,0,57,0.9)","rgba(199,0,57,0.86)","rgba(199,0,57,0.84)","rgba(199,0,57,0.79)","rgba(199,0,57,0.71)","rgba(199,0,57,0.71)","rgba(199,0,57,0.71)","rgba(199,0,57,0.32)","rgba(199,0,57,0.29)","rgba(199,0,57,0.29)","rgba(199,0,57,0.28)","rgba(199,0,57,0.28)","rgba(199,0,57,0.11)","rgba(199,0,57,0)","rgba(199,0,57,0)"],"x":[0.360254429766065,-0.159964606208776,-0.426364978843743,-0.295648701585871,-0.75904145513177,-0.53572130191972,-0.516479574926521,-0.346669313762858,-0.153676426816774,-0.334461096965296,-0.656373428452346,-0.54102279254912,0.418344900464257,-0.601378621730938,-0.107230438881253,-0.74559921933049,-0.483438062955762,-0.580103753909472,-0.657567896853395,0.0908726685735677,0.519305140285909,-0.662395449677413,0.794761337501694,-0.304337842077834,-0.363089281010603,-0.406805875022067,-0.400003500776135,0.161579354970627,0.754613148073271,-0.669537868202803,-0.546987522346899,0.312041215385695,0.29240250275371,0.753681494218897,0.384927284255291,-0.770968523218173,-0.333066778552332,0.567209026348377,-0.210580162813538,-0.845758639532966,0.904327983278828,0.968690650856855,0.199923704402104,-0.73460208234569,0.787495941987996,0.47201243778764,0.552982738458862,-1,0.34357302036294,-0.0628958005550894,0.717917892499206,0.224636747439855,0.455729974484527,-0.611921397711901,0.290756642882198,-0.908938886862889,-0.799214892061873,1,-0.399808701788304,0.699435899476006,0.452637518024387,0.972144673218631],"y":[0.937151760058446,0.999974103231821,-0.602929645384383,-0.633471227056216,-0.252546339249521,-0.700688704498716,-0.530068685683591,-0.948613065287562,-0.75278171751994,0.724392785701659,-0.494735033614329,-0.411340552916974,-0.99386332361773,-0.702632793139653,0.456594158978207,-0.562782318153557,-0.632737950752588,-0.5353021119053,-0.661765212936234,1,0.322511535889779,-0.583867806539282,0.0956900187795036,-0.444694469667459,-0.507948794382363,-0.409894310562876,0.488684108764023,0.518760324940025,-0.754021385105535,-0.0862131084773555,-0.0788553403151152,-1,-0.88853574263512,0.6217179646851,-0.824473118887236,-0.395184815248753,-0.0977907956875396,0.788012413163646,0.101144400494003,0.588076755525435,-0.554566606546476,-0.296302351895197,0.24899025412964,0.532738317615577,0.401411763422704,-0.901371529867083,0.060318567270937,-0.205038173477403,-0.196004144897851,0.753624091919721,-0.182962491829958,0.754963304268818,0.560040051082303,0.829784870431307,-0.0570390356861857,0.469398676617825,0.419446999652126,-0.0444093839579687,0.959623481234508,-0.441514437474266,-0.473474774946159,0.259152723630093]},"edges":{"from":["SCF-beta-TrCP mediated degradation of Emi1","SCF-beta-TrCP mediated degradation of Emi1","SCF-beta-TrCP mediated degradation of Emi1","SCF-beta-TrCP mediated degradation of Emi1","SCF-beta-TrCP mediated degradation of Emi1","SCF-beta-TrCP mediated degradation of Emi1","SCF-beta-TrCP mediated degradation of Emi1","SCF-beta-TrCP mediated degradation of Emi1","SCF-beta-TrCP mediated degradation of Emi1","SCF-beta-TrCP mediated degradation of Emi1","SCF-beta-TrCP mediated degradation of Emi1","SCF-beta-TrCP mediated degradation of Emi1","SCF-beta-TrCP mediated degradation of Emi1","SCF-beta-TrCP mediated degradation of Emi1","NIK-->noncanonical NF-kB signaling","NIK-->noncanonical NF-kB signaling","NIK-->noncanonical NF-kB signaling","NIK-->noncanonical NF-kB signaling","The role of GTSE1 in G2/M progression after G2 checkpoint","The role of GTSE1 in G2/M progression after G2 checkpoint","AUF1 (hnRNP D0) binds and destabilizes mRNA","AUF1 (hnRNP D0) binds and destabilizes mRNA","AUF1 (hnRNP D0) binds and destabilizes mRNA","AUF1 (hnRNP D0) binds and destabilizes mRNA","AUF1 (hnRNP D0) binds and destabilizes mRNA","AUF1 (hnRNP D0) binds and destabilizes mRNA","AUF1 (hnRNP D0) binds and destabilizes mRNA","AUF1 (hnRNP D0) binds and destabilizes mRNA","Vpu mediated degradation of CD4","Vpu mediated degradation of CD4","Vpu mediated degradation of CD4","Vpu mediated degradation of CD4","Vpu mediated degradation of CD4","Vpu mediated degradation of CD4","Vpu mediated degradation of CD4","Vpu mediated degradation of CD4","Vpu mediated degradation of CD4","Vpu mediated degradation of CD4","Vpu mediated degradation of CD4","Vpu mediated degradation of CD4","Negative regulation of NOTCH4 signaling","Autodegradation of the E3 ubiquitin ligase COP1","Autodegradation of the E3 ubiquitin ligase COP1","Autodegradation of the E3 ubiquitin ligase COP1","Autodegradation of the E3 ubiquitin ligase COP1","Autodegradation of the E3 ubiquitin ligase COP1","Autodegradation of the E3 ubiquitin ligase COP1","Autodegradation of the E3 ubiquitin ligase COP1","Autodegradation of the E3 ubiquitin ligase COP1","Regulation of activated PAK-2p34 by proteasome mediated degradation","Regulation of activated PAK-2p34 by proteasome mediated degradation","Regulation of activated PAK-2p34 by proteasome mediated degradation","Regulation of activated PAK-2p34 by proteasome mediated degradation","Regulation of activated PAK-2p34 by proteasome mediated degradation","Regulation of activated PAK-2p34 by proteasome mediated degradation","Regulation of activated PAK-2p34 by proteasome mediated degradation","Regulation of activated PAK-2p34 by proteasome mediated degradation","Regulation of activated PAK-2p34 by proteasome mediated degradation","Regulation of activated PAK-2p34 by proteasome mediated degradation","Regulation of activated PAK-2p34 by proteasome mediated degradation","Regulation of activated PAK-2p34 by proteasome mediated degradation","Regulation of activated PAK-2p34 by proteasome mediated degradation","APC/C:Cdh1 mediated degradation of Cdc20 and other APC/C:Cdh1 targeted proteins in late mitosis/early G1","APC/C:Cdh1 mediated degradation of Cdc20 and other APC/C:Cdh1 targeted proteins in late mitosis/early G1","APC/C:Cdh1 mediated degradation of Cdc20 and other APC/C:Cdh1 targeted proteins in late mitosis/early G1","APC/C:Cdh1 mediated degradation of Cdc20 and other APC/C:Cdh1 targeted proteins in late mitosis/early G1","Degradation of AXIN","Degradation of AXIN","Degradation of AXIN","Degradation of AXIN","Degradation of DVL","Degradation of DVL","Degradation of DVL","Degradation of DVL","FBXL7 down-regulates AURKA during mitotic entry and in early mitosis","FBXL7 down-regulates AURKA during mitotic entry and in early mitosis","FBXL7 down-regulates AURKA during mitotic entry and in early mitosis","FBXL7 down-regulates AURKA during mitotic entry and in early mitosis","FBXL7 down-regulates AURKA during mitotic entry and in early mitosis","FBXL7 down-regulates AURKA during mitotic entry and in early mitosis","Ubiquitin-dependent degradation of Cyclin D","Ubiquitin-dependent degradation of Cyclin D","Ubiquitin-dependent degradation of Cyclin D","Ubiquitin-dependent degradation of Cyclin D","Ubiquitin-dependent degradation of Cyclin D","Vif-mediated degradation of APOBEC3G","Ubiquitin Mediated Degradation of Phosphorylated Cdc25A","Degradation of GLI1 by the proteasome","Degradation of GLI1 by the proteasome","Degradation of GLI2 by the proteasome","Regulation of ornithine decarboxylase (ODC)","Autodegradation of Cdh1 by Cdh1:APC/C","Autodegradation of Cdh1 by Cdh1:APC/C","Autodegradation of Cdh1 by Cdh1:APC/C","Cdc20:Phospho-APC/C mediated degradation of Cyclin A","Cdc20:Phospho-APC/C mediated degradation of Cyclin A","APC/C:Cdc20 mediated degradation of Securin","Hh mutants that don't undergo autocatalytic processing are degraded by ERAD","CDT1 association with the CDC6:ORC:origin complex","AURKA Activation by TPX2","AURKA Activation by TPX2","AURKA Activation by TPX2","Loss of Nlp from mitotic centrosomes","Loss of Nlp from mitotic centrosomes","Amplification  of signal from unattached  kinetochores via a MAD2  inhibitory signal","Recruitment of NuMA to mitotic centrosomes"],"to":["NIK-->noncanonical NF-kB signaling","AUF1 (hnRNP D0) binds and destabilizes mRNA","Vpu mediated degradation of CD4","Dectin-1 mediated noncanonical NF-kB signaling","Autodegradation of the E3 ubiquitin ligase COP1","Regulation of activated PAK-2p34 by proteasome mediated degradation","Degradation of AXIN","FBXL7 down-regulates AURKA during mitotic entry and in early mitosis","Ubiquitin-dependent degradation of Cyclin D","Vif-mediated degradation of APOBEC3G","Ubiquitin Mediated Degradation of Phosphorylated Cdc25A","Degradation of GLI1 by the proteasome","Degradation of GLI2 by the proteasome","GLI3 is processed to GLI3R by the proteasome","Vpu mediated degradation of CD4","Dectin-1 mediated noncanonical NF-kB signaling","Regulation of activated PAK-2p34 by proteasome mediated degradation","FBXL7 down-regulates AURKA during mitotic entry and in early mitosis","Autodegradation of the E3 ubiquitin ligase COP1","Regulation of activated PAK-2p34 by proteasome mediated degradation","Vpu mediated degradation of CD4","Autodegradation of the E3 ubiquitin ligase COP1","Regulation of activated PAK-2p34 by proteasome mediated degradation","Degradation of AXIN","FBXL7 down-regulates AURKA during mitotic entry and in early mitosis","Ubiquitin-dependent degradation of Cyclin D","Vif-mediated degradation of APOBEC3G","Ubiquitin Mediated Degradation of Phosphorylated Cdc25A","Autodegradation of the E3 ubiquitin ligase COP1","Regulation of activated PAK-2p34 by proteasome mediated degradation","Degradation of AXIN","Degradation of DVL","FBXL7 down-regulates AURKA during mitotic entry and in early mitosis","Ubiquitin-dependent degradation of Cyclin D","Vif-mediated degradation of APOBEC3G","Ubiquitin Mediated Degradation of Phosphorylated Cdc25A","Degradation of GLI1 by the proteasome","Degradation of GLI2 by the proteasome","GLI3 is processed to GLI3R by the proteasome","Hh mutants that don't undergo autocatalytic processing are degraded by ERAD","FBXL7 down-regulates AURKA during mitotic entry and in early mitosis","Regulation of activated PAK-2p34 by proteasome mediated degradation","Degradation of AXIN","Degradation of DVL","FBXL7 down-regulates AURKA during mitotic entry and in early mitosis","Ubiquitin-dependent degradation of Cyclin D","Vif-mediated degradation of APOBEC3G","Ubiquitin Mediated Degradation of Phosphorylated Cdc25A","Hh mutants that don't undergo autocatalytic processing are degraded by ERAD","Degradation of AXIN","Degradation of DVL","FBXL7 down-regulates AURKA during mitotic entry and in early mitosis","Ubiquitin-dependent degradation of Cyclin D","Vif-mediated degradation of APOBEC3G","Ubiquitin Mediated Degradation of Phosphorylated Cdc25A","Degradation of GLI1 by the proteasome","Degradation of GLI2 by the proteasome","GLI3 is processed to GLI3R by the proteasome","Regulation of ornithine decarboxylase (ODC)","Cross-presentation of soluble exogenous antigens (endosomes)","Hh mutants that don't undergo autocatalytic processing are degraded by ERAD","CDT1 association with the CDC6:ORC:origin complex","Autodegradation of Cdh1 by Cdh1:APC/C","Cdc20:Phospho-APC/C mediated degradation of Cyclin A","APC/C:Cdc20 mediated degradation of Securin","CDK-mediated phosphorylation and removal of Cdc6","FBXL7 down-regulates AURKA during mitotic entry and in early mitosis","Ubiquitin-dependent degradation of Cyclin D","Vif-mediated degradation of APOBEC3G","Ubiquitin Mediated Degradation of Phosphorylated Cdc25A","FBXL7 down-regulates AURKA during mitotic entry and in early mitosis","Ubiquitin-dependent degradation of Cyclin D","Vif-mediated degradation of APOBEC3G","Ubiquitin Mediated Degradation of Phosphorylated Cdc25A","Ubiquitin-dependent degradation of Cyclin D","Vif-mediated degradation of APOBEC3G","Ubiquitin Mediated Degradation of Phosphorylated Cdc25A","Degradation of GLI1 by the proteasome","Degradation of GLI2 by the proteasome","GLI3 is processed to GLI3R by the proteasome","Vif-mediated degradation of APOBEC3G","Ubiquitin Mediated Degradation of Phosphorylated Cdc25A","Degradation of GLI2 by the proteasome","GLI3 is processed to GLI3R by the proteasome","Hh mutants that don't undergo autocatalytic processing are degraded by ERAD","Ubiquitin Mediated Degradation of Phosphorylated Cdc25A","Hh mutants that don't undergo autocatalytic processing are degraded by ERAD","Degradation of GLI2 by the proteasome","GLI3 is processed to GLI3R by the proteasome","GLI3 is processed to GLI3R by the proteasome","Cross-presentation of soluble exogenous antigens (endosomes)","Cdc20:Phospho-APC/C mediated degradation of Cyclin A","APC/C:Cdc20 mediated degradation of Securin","CDK-mediated phosphorylation and removal of Cdc6","APC/C:Cdc20 mediated degradation of Securin","CDK-mediated phosphorylation and removal of Cdc6","CDK-mediated phosphorylation and removal of Cdc6","Defective CFTR causes cystic fibrosis","Orc1 removal from chromatin","Loss of Nlp from mitotic centrosomes","Recruitment of NuMA to mitotic centrosomes","Recruitment of mitotic centrosome proteins and complexes","Recruitment of NuMA to mitotic centrosomes","Recruitment of mitotic centrosome proteins and complexes","Resolution of Sister Chromatid Cohesion","Recruitment of mitotic centrosome proteins and complexes"],"weight":[0.838709677419355,0.80327868852459,0.894736842105263,0.8,0.844827586206897,0.875,0.80327868852459,0.864406779661017,0.844827586206897,0.80327868852459,0.844827586206897,0.825396825396825,0.825396825396825,0.825396825396825,0.836065573770492,0.951612903225806,0.816666666666667,0.80952380952381,0.806451612903226,0.80327868852459,0.830508474576271,0.844827586206897,0.875,0.80327868852459,0.80327868852459,0.844827586206897,0.80327868852459,0.844827586206897,0.875,0.907407407407407,0.830508474576271,0.80327868852459,0.862068965517241,0.875,0.830508474576271,0.875,0.82258064516129,0.82258064516129,0.82258064516129,0.816666666666667,0.816666666666667,0.924528301886792,0.844827586206897,0.816666666666667,0.844827586206897,0.890909090909091,0.844827586206897,0.890909090909091,0.830508474576271,0.875,0.844827586206897,0.875,0.924528301886792,0.875,0.924528301886792,0.80327868852459,0.80327868852459,0.80327868852459,0.803571428571429,0.818181818181818,0.859649122807018,0.816666666666667,0.875,0.846153846153846,0.930555555555556,0.846153846153846,0.80327868852459,0.844827586206897,0.80327868852459,0.844827586206897,0.806451612903226,0.816666666666667,0.806451612903226,0.816666666666667,0.844827586206897,0.833333333333333,0.844827586206897,0.825396825396825,0.825396825396825,0.825396825396825,0.844827586206897,0.890909090909091,0.806451612903226,0.806451612903226,0.830508474576271,0.844827586206897,0.830508474576271,0.904761904761905,0.904761904761905,0.967213114754098,0.803571428571429,0.849315068493151,0.911764705882353,0.875,0.904109589041096,0.87012987012987,0.878378378378378,0.857142857142857,0.805555555555556,0.958333333333333,0.831325301204819,0.821428571428571,0.8625,0.851851851851852,0.878504672897196,0.963414634146341],"value":[0.838709677419355,0.80327868852459,0.894736842105263,0.8,0.844827586206897,0.875,0.80327868852459,0.864406779661017,0.844827586206897,0.80327868852459,0.844827586206897,0.825396825396825,0.825396825396825,0.825396825396825,0.836065573770492,0.951612903225806,0.816666666666667,0.80952380952381,0.806451612903226,0.80327868852459,0.830508474576271,0.844827586206897,0.875,0.80327868852459,0.80327868852459,0.844827586206897,0.80327868852459,0.844827586206897,0.875,0.907407407407407,0.830508474576271,0.80327868852459,0.862068965517241,0.875,0.830508474576271,0.875,0.82258064516129,0.82258064516129,0.82258064516129,0.816666666666667,0.816666666666667,0.924528301886792,0.844827586206897,0.816666666666667,0.844827586206897,0.890909090909091,0.844827586206897,0.890909090909091,0.830508474576271,0.875,0.844827586206897,0.875,0.924528301886792,0.875,0.924528301886792,0.80327868852459,0.80327868852459,0.80327868852459,0.803571428571429,0.818181818181818,0.859649122807018,0.816666666666667,0.875,0.846153846153846,0.930555555555556,0.846153846153846,0.80327868852459,0.844827586206897,0.80327868852459,0.844827586206897,0.806451612903226,0.816666666666667,0.806451612903226,0.816666666666667,0.844827586206897,0.833333333333333,0.844827586206897,0.825396825396825,0.825396825396825,0.825396825396825,0.844827586206897,0.890909090909091,0.806451612903226,0.806451612903226,0.830508474576271,0.844827586206897,0.830508474576271,0.904761904761905,0.904761904761905,0.967213114754098,0.803571428571429,0.849315068493151,0.911764705882353,0.875,0.904109589041096,0.87012987012987,0.878378378378378,0.857142857142857,0.805555555555556,0.958333333333333,0.831325301204819,0.821428571428571,0.8625,0.851851851851852,0.878504672897196,0.963414634146341]},"nodesToDataframe":true,"edgesToDataframe":true,"options":{"width":"100%","height":"100%","nodes":{"shape":"dot","borderWidth":1,"borderWidthSelected":0,"physics":false},"manipulation":{"enabled":false},"edges":{"color":"rgb(88,24,69)","smooth":false},"interaction":{"multiselect":true,"tooltipDelay":300},"physics":{"stabilization":false}},"groups":null,"width":null,"height":null,"idselection":{"enabled":false,"style":"width: 150px; height: 26px","useLabels":true,"main":"Select by id"},"byselection":{"enabled":false,"style":"width: 150px; height: 26px","multiple":false,"hideColor":"rgba(200,200,200,0.5)"},"main":{"text":"","style":"font-family:Helvetica"},"submain":null,"footer":null,"background":"rgba(0, 0, 0, 0)","highlight":{"enabled":true,"hoverNearest":false,"degree":1,"algorithm":"all","hideColor":"rgba(200,200,200,0.5)","labelOnly":true},"collapse":{"enabled":false,"fit":false,"resetHighlight":true,"clusterOptions":null,"keepCoord":true,"labelSuffix":"(cluster)"},"tooltipStay":300,"tooltipStyle":"position: fixed;visibility:hidden;padding: 5px;white-space: nowrap;font-family: verdana;font-size:14px;font-color:#000000;background-color: #f5f4ed;-moz-border-radius: 3px;-webkit-border-radius: 3px;border-radius: 3px;border: 1px solid #808074;box-shadow: 3px 3px 10px rgba(0, 0, 0, 0.2);","igraphlayout":{"type":"square"}},"evals":[],"jsHooks":[]}</script>
</div>
<div id="hiearchy-map" class="section level2">
<h2 class="hasAnchor">
<a href="#hiearchy-map" class="anchor"></a>Hiearchy map</h2>
<div id="relational-genesets" class="section level3">
<h3 class="hasAnchor">
<a href="#relational-genesets" class="anchor"></a>Relational genesets</h3>
<p>When dealing with hundreds of genesets, it’s often useful to understand the relationships between them. This allows researchers to summarize many enriched pathways as more general biological processes. To do this, we rely on curated relationships defined between them. For example, <a href="https://reactome.org/">REACTOME</a> conveniently defines their genesets in a <a href="https://reactome.org/PathwayBrowser/">hiearchy of pathways</a>. This data can be formatted into a relational genesets object called <code>rgsets</code>.</p>
<div class="sourceCode" id="cb46"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb46-1"><a href="#cb46-1"></a>rgsets &lt;-<span class="st"> </span><span class="kw"><a href="../reference/hyperdb_fetch.html">hyperdb_fetch</a></span>(<span class="dt">type=</span><span class="st">"rgsets"</span>, <span class="st">"REACTOME"</span>)</span></code></pre></div>
<p>Relational genesets have three data atrributes including gsets, nodes, and edges. The <code>gsets</code> attribute includes the geneset information for the leaf nodes of the hiearchy, the <code>nodes</code> attribute describes all nodes in the hiearchical, including internal nodes, and the <code>edges</code> attribute describes the edges in the hiearchy.</p>
<div id="gsets" class="section level4">
<h4 class="hasAnchor">
<a href="#gsets" class="anchor"></a>$gsets</h4>
<div class="sourceCode" id="cb47"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb47-1"><a href="#cb47-1"></a>gsets &lt;-<span class="st"> </span>rgsets<span class="op">$</span>gsets</span>
<span id="cb47-2"><a href="#cb47-2"></a><span class="kw"><a href="https://www.rdocumentation.org/packages/base/topics/names">names</a></span>(gsets)[<span class="dv">800</span><span class="op">:</span><span class="dv">805</span>]</span></code></pre></div>
<pre><code>## [1] "MAPK3 (ERK1) activation"                                          
## [2] "MAPK6/MAPK4 signaling"                                            
## [3] "MASTL Facilitates Mitotic Progression"                            
## [4] "MECP2 regulates neuronal receptors and channels"                  
## [5] "MECP2 regulates transcription factors"                            
## [6] "MECP2 regulates transcription of genes involved in GABA signaling"</code></pre>
</div>
<div id="nodes" class="section level4">
<h4 class="hasAnchor">
<a href="#nodes" class="anchor"></a>$nodes</h4>
<div class="sourceCode" id="cb49"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb49-1"><a href="#cb49-1"></a>nodes &lt;-<span class="st"> </span>rgsets<span class="op">$</span>nodes</span>
<span id="cb49-2"><a href="#cb49-2"></a>nodes[<span class="dv">1123</span><span class="op">:</span><span class="dv">1128</span>,]</span></code></pre></div>
<pre><code>##                                                              label
## R-HSA-450282  MAPK targets/ Nuclear events mediated by MAP kinases
## R-HSA-112411                               MAPK1 (ERK2) activation
## R-HSA-5684996                                MAPK1/MAPK3 signaling
## R-HSA-110056                               MAPK3 (ERK1) activation
## R-HSA-5687128                                MAPK6/MAPK4 signaling
## R-HSA-2465910                MASTL Facilitates Mitotic Progression
##                          id length
## R-HSA-450282   R-HSA-450282     28
## R-HSA-112411   R-HSA-112411      9
## R-HSA-5684996 R-HSA-5684996    128
## R-HSA-110056   R-HSA-110056     10
## R-HSA-5687128 R-HSA-5687128     94
## R-HSA-2465910 R-HSA-2465910     10</code></pre>
</div>
<div id="edges" class="section level4">
<h4 class="hasAnchor">
<a href="#edges" class="anchor"></a>$edges</h4>
<div class="sourceCode" id="cb51"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb51-1"><a href="#cb51-1"></a>edges &lt;-<span class="st"> </span>rgsets<span class="op">$</span>edges</span>
<span id="cb51-2"><a href="#cb51-2"></a>edges[<span class="dv">1994</span><span class="op">:</span><span class="dv">1999</span>,]</span></code></pre></div>
<pre><code>##               from            to
## 1994 R-HSA-8868766 R-HSA-6793080
## 1995 R-HSA-8868773 R-HSA-6790901
## 1996 R-HSA-8868773 R-HSA-6791226
## 1997 R-HSA-8875878 R-HSA-8874081
## 1998 R-HSA-8875878 R-HSA-8875513
## 1999 R-HSA-8875878 R-HSA-8875555</code></pre>
<p>All workflows begin with performing hyper enrichment with <code><a href="../reference/hypeR.html">hypeR()</a></code>.</p>
<div class="sourceCode" id="cb53"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb53-1"><a href="#cb53-1"></a>hyp_obj &lt;-<span class="st"> </span><span class="kw"><a href="../reference/hypeR.html">hypeR</a></span>(ranked.signature, rgsets, <span class="dt">test=</span><span class="st">"kstest"</span>, <span class="dt">fdr_cutoff=</span><span class="fl">0.01</span>)</span></code></pre></div>
</div>
</div>
</div>
<div id="hierarchy-map" class="section level2">
<h2 class="hasAnchor">
<a href="#hierarchy-map" class="anchor"></a>Hierarchy map</h2>
<p>One can visualize the top enriched genesets using <code><a href="../reference/hyp_hmap.html">hyp_hmap()</a></code> which will return a hiearchy map. Each node represents a geneset, where the shade of the gold border indicates the normalized significance of enrichment. Hover over the leaf nodes to view the raw value. Double click internal nodes to cluster their first degree connections. Edges represent a directed relationship between genesets in the hiearchy.</p>
<div class="sourceCode" id="cb54"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb54-1"><a href="#cb54-1"></a><span class="kw"><a href="../reference/hyp_hmap.html">hyp_hmap</a></span>(hyp_obj, <span class="dt">top=</span><span class="dv">30</span>, <span class="dt">show_plots=</span><span class="ot">FALSE</span>, <span class="dt">return_plots=</span><span class="ot">TRUE</span>)</span></code></pre></div>
<div id="htmlwidget-22fe12c7ac94f1dcf16f" style="width:648px;height:672px;" class="visNetwork html-widget"></div>
<script type="application/json" data-for="htmlwidget-22fe12c7ac94f1dcf16f">{"x":{"nodes":{"label":["APC/C:Cdh1 mediated degradation of Cdc20 and other APC/C:Cdh1 targeted proteins in late mitosis/early G1","APC/C-mediated degradation of cell cycle proteins","Regulation of mitotic cell cycle","Cell Cycle, Mitotic","Cell Cycle","AUF1 (hnRNP D0) binds and destabilizes mRNA","Regulation of mRNA stability by proteins that bind AU-rich elements","Metabolism of RNA","Asymmetric localization of PCP proteins","PCP/CE pathway","Beta-catenin independent WNT signaling","Signaling by WNT","Signal Transduction","Autodegradation of the E3 ubiquitin ligase COP1","Stabilization of p53","p53-Dependent G1 DNA Damage Response","p53-Dependent G1/S DNA damage checkpoint","G1/S DNA Damage Checkpoints","Cell Cycle Checkpoints","Dectin-1 mediated noncanonical NF-kB signaling","CLEC7A (Dectin-1) signaling","C-type lectin receptors (CLRs)","Innate Immune System","Immune System","Degradation of AXIN","TCF dependent signaling in response to WNT","Degradation of DVL","Degradation of GLI1 by the proteasome","Hedgehog 'off' state","Signaling by Hedgehog","Degradation of GLI2 by the proteasome","FBXL7 down-regulates AURKA during mitotic entry and in early mitosis","G2/M Transition","Mitotic G2-G2/M phases","FCERI mediated NF-kB activation","Fc epsilon receptor (FCERI) signaling","GLI3 is processed to GLI3R by the proteasome","MAPK6/MAPK4 signaling","MAPK family signaling cascades","NIK-->noncanonical NF-kB signaling","TNFR2 non-canonical NF-kB pathway","Cytokine Signaling in Immune system","Negative regulation of NOTCH4 signaling","Signaling by NOTCH4","Signaling by NOTCH","Oxygen-dependent proline hydroxylation of Hypoxia-inducible Factor Alpha","Cellular response to hypoxia","Cellular responses to stress","Cellular responses to external stimuli","RHO GTPases Activate Formins","RHO GTPase Effectors","Signaling by Rho GTPases","Regulation of PLK1 Activity at G2/M Transition","Regulation of RAS by GAPs","RAF/MAP kinase cascade","MAPK1/MAPK3 signaling","FLT3 Signaling","Other interleukin signaling","Signaling by Interleukins","Regulation of RUNX3 expression and activity","Transcriptional regulation by RUNX3","Generic Transcription Pathway","RNA Polymerase II Transcription","Gene expression (Transcription)","Regulation of activated PAK-2p34 by proteasome mediated degradation","Regulation of Apoptosis","Apoptosis","Programmed Cell Death","Regulation of ornithine decarboxylase (ODC)","Metabolism of polyamines","Metabolism of amino acids and derivatives","Metabolism","SCF-beta-TrCP mediated degradation of Emi1","Regulation of APC/C activators between G1/S and early anaphase","Separation of Sister Chromatids","Mitotic Anaphase","Mitotic Metaphase and Anaphase","M Phase","The role of GTSE1 in G2/M progression after G2 checkpoint","Ub-specific processing proteases","Deubiquitination","Post-translational protein modification","Metabolism of proteins","Ubiquitin Mediated Degradation of Phosphorylated Cdc25A","p53-Independent DNA Damage Response","p53-Independent G1/S DNA damage checkpoint","Ubiquitin-dependent degradation of Cyclin D","S Phase","Vif-mediated degradation of APOBEC3G","Host Interactions of HIV factors","HIV Infection","Infectious disease","Disease","Vpu mediated degradation of CD4"],"id":["R-HSA-174178","R-HSA-174143","R-HSA-453276","R-HSA-69278","R-HSA-1640170","R-HSA-450408","R-HSA-450531","R-HSA-8953854","R-HSA-4608870","R-HSA-4086400","R-HSA-3858494","R-HSA-195721","R-HSA-162582","R-HSA-349425","R-HSA-69541","R-HSA-69563","R-HSA-69580","R-HSA-69615","R-HSA-69620","R-HSA-5607761","R-HSA-5607764","R-HSA-5621481","R-HSA-168249","R-HSA-168256","R-HSA-4641257","R-HSA-201681","R-HSA-4641258","R-HSA-5610780","R-HSA-5610787","R-HSA-5358351","R-HSA-5610783","R-HSA-8854050","R-HSA-69275","R-HSA-453274","R-HSA-2871837","R-HSA-2454202","R-HSA-5610785","R-HSA-5687128","R-HSA-5683057","R-HSA-5676590","R-HSA-5668541","R-HSA-1280215","R-HSA-9604323","R-HSA-9013694","R-HSA-157118","R-HSA-1234176","R-HSA-1234174","R-HSA-2262752","R-HSA-8953897","R-HSA-5663220","R-HSA-195258","R-HSA-194315","R-HSA-2565942","R-HSA-5658442","R-HSA-5673001","R-HSA-5684996","R-HSA-9607240","R-HSA-449836","R-HSA-449147","R-HSA-8941858","R-HSA-8878159","R-HSA-212436","R-HSA-73857","R-HSA-74160","R-HSA-211733","R-HSA-169911","R-HSA-109581","R-HSA-5357801","R-HSA-350562","R-HSA-351202","R-HSA-71291","R-HSA-1430728","R-HSA-174113","R-HSA-176408","R-HSA-2467813","R-HSA-68882","R-HSA-2555396","R-HSA-68886","R-HSA-8852276","R-HSA-5689880","R-HSA-5688426","R-HSA-597592","R-HSA-392499","R-HSA-69601","R-HSA-69610","R-HSA-69613","R-HSA-75815","R-HSA-69242","R-HSA-180585","R-HSA-162909","R-HSA-162906","R-HSA-5663205","R-HSA-1643685","R-HSA-180534"],"length":[72,76,76,258,263,55,55,55,64,64,64,76,269,52,52,52,52,55,55,62,62,62,164,183,55,63,57,60,64,64,60,55,140,140,157,157,60,94,113,59,59,78,54,54,54,66,66,66,66,121,121,121,87,68,68,68,68,68,68,55,55,55,55,55,50,50,50,50,51,51,51,51,55,55,171,171,171,171,60,205,205,205,205,52,52,52,52,52,55,59,59,59,59,53],"size":[15,15,15,28.4931506849315,29.1780821917808,15,15,15,15,15,15,15,30,15,15,15,15,15,15,15,15,15,15.6164383561644,18.2191780821918,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,15,16.5753424657534,16.5753424657534,16.5753424657534,16.5753424657534,15,21.2328767123288,21.2328767123288,21.2328767123288,21.2328767123288,15,15,15,15,15,15,15,15,15,15,15],"title":["FDR: 1.7e-05",null,null,null,null,"FDR: 1e-05",null,null,"FDR: 8.5e-05",null,null,null,null,"FDR: 1.7e-05",null,null,null,null,null,"FDR: 1.4e-05",null,null,null,null,"FDR: 4e-05",null,"FDR: 7e-05","FDR: 8.9e-05",null,null,"FDR: 8.9e-05","FDR: 7.1e-05",null,null,"FDR: 9.7e-05",null,"FDR: 8.9e-05","FDR: 4.8e-08",null,"FDR: 1e-05",null,null,"FDR: 1.4e-05",null,null,"FDR: 7e-05",null,null,null,"FDR: 8.9e-05",null,null,"FDR: 8.5e-05","FDR: 8.9e-05",null,null,null,null,null,"FDR: 8.9e-05",null,null,null,null,"FDR: 1.7e-05",null,null,null,"FDR: 9.9e-05",null,null,null,"FDR: 8.3e-06",null,"FDR: 8.3e-06",null,null,null,"FDR: 1e-05","FDR: 1.7e-05",null,null,null,"FDR: 8.5e-05",null,null,"FDR: 7.1e-05",null,"FDR: 7.6e-05",null,null,null,null,"FDR: 1e-05"],"color.background":["rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(88,24,69,1.0)","rgba(199,0,57,1.0)"],"color.highlight":["rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(88,24,69,0.9)","rgba(199,0,57,0.9)"],"color.border":["rgba(255,195,0,0.829)","rgb(88,24,69)","rgb(88,24,69)","rgb(88,24,69)","rgb(88,24,69)","rgba(255,195,0,0.899)","rgb(88,24,69)","rgb(88,24,69)","rgba(255,195,0,0.141)","rgb(88,24,69)","rgb(88,24,69)","rgb(88,24,69)","rgb(88,24,69)","rgba(255,195,0,0.829)","rgb(88,24,69)","rgb(88,24,69)","rgb(88,24,69)","rgb(88,24,69)","rgb(88,24,69)","rgba(255,195,0,0.859)","rgb(88,24,69)","rgb(88,24,69)","rgb(88,24,69)","rgb(88,24,69)","rgba(255,195,0,0.596)","rgb(88,24,69)","rgba(255,195,0,0.293)","rgba(255,195,0,0.101)","rgb(88,24,69)","rgb(88,24,69)","rgba(255,195,0,0.101)","rgba(255,195,0,0.283)","rgb(88,24,69)","rgb(88,24,69)","rgba(255,195,0,0.02)","rgb(88,24,69)","rgba(255,195,0,0.101)","rgba(255,195,0,1)","rgb(88,24,69)","rgba(255,195,0,0.899)","rgb(88,24,69)","rgb(88,24,69)","rgba(255,195,0,0.859)","rgb(88,24,69)","rgb(88,24,69)","rgba(255,195,0,0.293)","rgb(88,24,69)","rgb(88,24,69)","rgb(88,24,69)","rgba(255,195,0,0.101)","rgb(88,24,69)","rgb(88,24,69)","rgba(255,195,0,0.141)","rgba(255,195,0,0.101)","rgb(88,24,69)","rgb(88,24,69)","rgb(88,24,69)","rgb(88,24,69)","rgb(88,24,69)","rgba(255,195,0,0.101)","rgb(88,24,69)","rgb(88,24,69)","rgb(88,24,69)","rgb(88,24,69)","rgba(255,195,0,0.829)","rgb(88,24,69)","rgb(88,24,69)","rgb(88,24,69)","rgba(255,195,0,0)","rgb(88,24,69)","rgb(88,24,69)","rgb(88,24,69)","rgba(255,195,0,0.917)","rgb(88,24,69)","rgba(255,195,0,0.917)","rgb(88,24,69)","rgb(88,24,69)","rgb(88,24,69)","rgba(255,195,0,0.899)","rgba(255,195,0,0.829)","rgb(88,24,69)","rgb(88,24,69)","rgb(88,24,69)","rgba(255,195,0,0.141)","rgb(88,24,69)","rgb(88,24,69)","rgba(255,195,0,0.283)","rgb(88,24,69)","rgba(255,195,0,0.232)","rgb(88,24,69)","rgb(88,24,69)","rgb(88,24,69)","rgb(88,24,69)","rgba(255,195,0,0.899)"]},"edges":{"from":["R-HSA-109581","R-HSA-1234174","R-HSA-1280215","R-HSA-1280215","R-HSA-1430728","R-HSA-157118","R-HSA-162582","R-HSA-162582","R-HSA-162582","R-HSA-162582","R-HSA-162582","R-HSA-162906","R-HSA-162909","R-HSA-162909","R-HSA-1640170","R-HSA-1640170","R-HSA-1643685","R-HSA-168249","R-HSA-168249","R-HSA-168256","R-HSA-168256","R-HSA-169911","R-HSA-174143","R-HSA-174143","R-HSA-176408","R-HSA-194315","R-HSA-195258","R-HSA-195721","R-HSA-195721","R-HSA-201681","R-HSA-201681","R-HSA-212436","R-HSA-2262752","R-HSA-2454202","R-HSA-2555396","R-HSA-351202","R-HSA-3858494","R-HSA-392499","R-HSA-4086400","R-HSA-449147","R-HSA-449836","R-HSA-450531","R-HSA-453274","R-HSA-453276","R-HSA-5357801","R-HSA-5358351","R-HSA-5607764","R-HSA-5610787","R-HSA-5610787","R-HSA-5610787","R-HSA-5621481","R-HSA-5663205","R-HSA-5668541","R-HSA-5673001","R-HSA-5683057","R-HSA-5683057","R-HSA-5684996","R-HSA-5688426","R-HSA-597592","R-HSA-68882","R-HSA-68886","R-HSA-69242","R-HSA-69275","R-HSA-69275","R-HSA-69275","R-HSA-69278","R-HSA-69278","R-HSA-69278","R-HSA-69278","R-HSA-69541","R-HSA-69563","R-HSA-69580","R-HSA-69610","R-HSA-69613","R-HSA-69615","R-HSA-69615","R-HSA-69620","R-HSA-71291","R-HSA-73857","R-HSA-74160","R-HSA-8878159","R-HSA-8953854","R-HSA-8953897","R-HSA-9013694","R-HSA-9607240"],"to":["R-HSA-169911","R-HSA-1234176","R-HSA-449147","R-HSA-5668541","R-HSA-71291","R-HSA-9013694","R-HSA-157118","R-HSA-194315","R-HSA-195721","R-HSA-5358351","R-HSA-5683057","R-HSA-162909","R-HSA-180534","R-HSA-180585","R-HSA-69278","R-HSA-69620","R-HSA-5663205","R-HSA-2454202","R-HSA-5621481","R-HSA-1280215","R-HSA-168249","R-HSA-211733","R-HSA-174178","R-HSA-176408","R-HSA-174113","R-HSA-195258","R-HSA-5663220","R-HSA-201681","R-HSA-3858494","R-HSA-4641257","R-HSA-4641258","R-HSA-8878159","R-HSA-1234174","R-HSA-2871837","R-HSA-68882","R-HSA-350562","R-HSA-4086400","R-HSA-597592","R-HSA-4608870","R-HSA-449836","R-HSA-9607240","R-HSA-450408","R-HSA-69275","R-HSA-174143","R-HSA-109581","R-HSA-5610787","R-HSA-5607761","R-HSA-5610780","R-HSA-5610783","R-HSA-5610785","R-HSA-5607764","R-HSA-162906","R-HSA-5676590","R-HSA-5658442","R-HSA-5684996","R-HSA-5687128","R-HSA-5673001","R-HSA-5689880","R-HSA-5688426","R-HSA-2467813","R-HSA-2555396","R-HSA-75815","R-HSA-2565942","R-HSA-8852276","R-HSA-8854050","R-HSA-453274","R-HSA-453276","R-HSA-68886","R-HSA-69242","R-HSA-349425","R-HSA-69541","R-HSA-69563","R-HSA-69601","R-HSA-69610","R-HSA-69580","R-HSA-69613","R-HSA-69615","R-HSA-351202","R-HSA-212436","R-HSA-73857","R-HSA-8941858","R-HSA-450531","R-HSA-2262752","R-HSA-9604323","R-HSA-5673001"]},"nodesToDataframe":true,"edgesToDataframe":true,"options":{"width":"100%","height":"100%","nodes":{"shape":"dot","borderWidth":3,"borderWidthSelected":0},"manipulation":{"enabled":false},"edges":{"selectionWidth":0,"arrows":"to"},"interaction":{"multiselect":true,"tooltipDelay":300}},"groups":null,"width":null,"height":null,"idselection":{"enabled":false,"style":"width: 150px; height: 26px","useLabels":true,"main":"Select by id"},"byselection":{"enabled":false,"style":"width: 150px; height: 26px","multiple":false,"hideColor":"rgba(200,200,200,0.5)"},"main":{"text":"","style":"font-family:Helvetica"},"submain":null,"footer":null,"background":"rgba(0, 0, 0, 0)","tooltipStay":300,"tooltipStyle":"position: fixed;visibility:hidden;padding: 5px;white-space: nowrap;font-family: verdana;font-size:14px;font-color:#000000;background-color: #f5f4ed;-moz-border-radius: 3px;-webkit-border-radius: 3px;border-radius: 3px;border: 1px solid #808074;box-shadow: 3px 3px 10px rgba(0, 0, 0, 0.2);","highlight":{"enabled":true,"hoverNearest":false,"degree":1,"algorithm":"all","hideColor":"rgba(200,200,200,0.5)","labelOnly":true},"collapse":{"enabled":true,"fit":false,"resetHighlight":true,"clusterOptions":{"color":"rgba(199,0,57,1.0)"},"keepCoord":true,"labelSuffix":"(cluster)"}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
<div id="more-examples" class="section level1">
<h1 class="hasAnchor">
<a href="#more-examples" class="anchor"></a>More Examples</h1>
<p>For more examples, tutorials, and extensive documentation, visit the official documentation page at <a href="https://montilab.github.io/hypeR-docs" class="uri">https://montilab.github.io/hypeR-docs</a>.</p>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="sidebar">
        <div id="tocnav">
      <h2 class="hasAnchor">
<a href="#tocnav" class="anchor"></a>Contents</h2>
      <ul class="nav nav-pills nav-stacked">
<li><a href="#introduction">Introduction</a></li>
      <li><a href="#installation">Installation</a></li>
      <li>
<a href="#basics">Basics</a><ul class="nav nav-pills nav-stacked">
<li><a href="#terminology">Terminology</a></li>
      <li><a href="#example-data">Example Data</a></li>
      <li><a href="#basic-usage">Basic Usage</a></li>
      </ul>
</li>
      <li>
<a href="#downloading-data">Downloading Data</a><ul class="nav nav-pills nav-stacked">
<li><a href="#downloading-genesets-from-msigdb">Downloading genesets from msigdb</a></li>
      <li><a href="#defining-custom-genesets">Defining custom genesets</a></li>
      <li><a href="#downloading-genesets-from-hyperdb">Downloading genesets from hyperdb</a></li>
      </ul>
</li>
      <li>
<a href="#visualize-results">Visualize Results</a><ul class="nav nav-pills nav-stacked">
<li><a href="#example-data-1">Example Data</a></li>
      <li><a href="#hyper-enrichment">Hyper enrichment</a></li>
      <li><a href="#dots-plot">Dots plot</a></li>
      <li><a href="#enrichment-map">Enrichment map</a></li>
      <li><a href="#hiearchy-map">Hiearchy map</a></li>
      <li><a href="#hierarchy-map">Hierarchy map</a></li>
      </ul>
</li>
      <li><a href="#more-examples">More Examples</a></li>
      </ul>
</div>
      </div>

</div>


      <footer><div class="copyright">
  <p>Developed by <a href="https://github.com/anfederico">Anthony Federico</a>, <a href="https://github.com/montilab">Stefano Monti</a>.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.3.0.</p>
</div>
      </footer>
</div>

  

  </body>
</html>
