<!-- Generated by pkgdown: do not edit by hand -->
<!DOCTYPE html>
<html lang="en">
  <head>
  <meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<title>Print hypeR-db gsets — hyperdb_info • hypeR</title>

<!-- favicons -->
<link rel="icon" type="image/png" sizes="16x16" href="../favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="../favicon-32x32.png">
<link rel="apple-touch-icon" type="image/png" sizes="180x180" href="../apple-touch-icon.png" />
<link rel="apple-touch-icon" type="image/png" sizes="120x120" href="../apple-touch-icon-120x120.png" />
<link rel="apple-touch-icon" type="image/png" sizes="76x76" href="../apple-touch-icon-76x76.png" />
<link rel="apple-touch-icon" type="image/png" sizes="60x60" href="../apple-touch-icon-60x60.png" />
<!-- jquery -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
<!-- Bootstrap -->
<link href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.3.7/cosmo/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous" />

<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha256-U5ZEeKfGNOja007MMD3YBI0A3OSZOQbeG6z2f2Y0hu8=" crossorigin="anonymous"></script>

<!-- Font Awesome icons -->
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" integrity="sha256-eZrrJcwDc/3uDhsdt61sL2oOBY362qM3lon1gyExkL0=" crossorigin="anonymous" />

<!-- clipboard.js -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.4/clipboard.min.js" integrity="sha256-FiZwavyI2V6+EXO1U+xzLG3IKldpiTFf3153ea9zikQ=" crossorigin="anonymous"></script>

<!-- sticky kit -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/sticky-kit/1.1.3/sticky-kit.min.js" integrity="sha256-c4Rlo1ZozqTPE2RLuvbusY3+SU1pQaJC0TjuhygMipw=" crossorigin="anonymous"></script>

<!-- pkgdown -->
<link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script>



<meta property="og:title" content="Print hypeR-db gsets — hyperdb_info" />

<meta property="og:description" content="Print hypeR-db gsets" />
<meta name="twitter:card" content="summary" />



<!-- mathjax -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script>

<!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->


  </head>

  <body>
    <div class="container template-reference-topic">
      <header>
      <div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">hypeR</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">1.10.1</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="../index.html">Home</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Tutorials
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="../articles/installation.html">Installation</a>
    </li>
    <li>
      <a href="../articles/basics.html">Basic Usage</a>
    </li>
    <li>
      <a href="../articles/data.html">Downloading Data</a>
    </li>
    <li>
      <a href="../articles/visualize.html">Visualize Results</a>
    </li>
    <li>
      <a href="../articles/relational.html">Relational Genesets</a>
    </li>
    <li>
      <a href="../articles/multihyp.html">Multiple Signatures</a>
    </li>
    <li>
      <a href="../articles/reporting.html">Generating Reports</a>
    </li>
  </ul>
</li>
      </ul>
      
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/montilab/hypeR">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
      
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

      
      </header>

<div class="row">
  <div class="col-md-9 contents">
    <div class="page-header">
    <h1>Print hypeR-db gsets</h1>
    <small class="dont-index">Source: <a href='https://github.com/montilab/hypeR/blob/master/R/hyperdb.R'><code>R/hyperdb.R</code></a></small>
    <div class="hidden name"><code>hyperdb_info.Rd</code></div>
    </div>

    <div class="ref-description">
    
    <p>Print hypeR-db gsets</p>
    
    </div>

    <pre class="usage"><span class='fu'>hyperdb_info</span>(<span class='kw'>quiet</span> <span class='kw'>=</span> <span class='fl'>FALSE</span>)</pre>
    
    <h2 class="hasAnchor" id="arguments"><a class="anchor" href="#arguments"></a>Arguments</h2>
    <table class="ref-arguments">
    <colgroup><col class="name" /><col class="desc" /></colgroup>
    <tr>
      <th>quiet</th>
      <td><p>Use true to suppress printing of available genesets</p></td>
    </tr>
    </table>
    
    <h2 class="hasAnchor" id="value"><a class="anchor" href="#value"></a>Value</h2>

    <p>A character vector of available genesets</p>
    

    <h2 class="hasAnchor" id="examples"><a class="anchor" href="#examples"></a>Examples</h2>
    <pre class="examples"><div class='input'><span class='fu'>hyperdb_info</span>()</div><div class='output co'>#&gt;   [1] "ARCHS4_Cell-lines"                                
#&gt;   [2] "ARCHS4_IDG_Coexp"                                 
#&gt;   [3] "ARCHS4_Kinases_Coexp"                             
#&gt;   [4] "ARCHS4_TFs_Coexp"                                 
#&gt;   [5] "ARCHS4_Tissues"                                   
#&gt;   [6] "Achilles_fitness_decrease"                        
#&gt;   [7] "Achilles_fitness_increase"                        
#&gt;   [8] "Aging_Perturbations_from_GEO_down"                
#&gt;   [9] "Aging_Perturbations_from_GEO_up"                  
#&gt;  [10] "Allen_Brain_Atlas_down"                           
#&gt;  [11] "Allen_Brain_Atlas_up"                             
#&gt;  [12] "BioCarta_2013"                                    
#&gt;  [13] "BioCarta_2015"                                    
#&gt;  [14] "BioCarta_2016"                                    
#&gt;  [15] "BioPlex_2017"                                     
#&gt;  [16] "CORUM"                                            
#&gt;  [17] "Cancer_Cell_Line_Encyclopedia"                    
#&gt;  [18] "ChEA_2013"                                        
#&gt;  [19] "ChEA_2015"                                        
#&gt;  [20] "ChEA_2016"                                        
#&gt;  [21] "Chromosome_Location"                              
#&gt;  [22] "Chromosome_Location_hg19"                         
#&gt;  [23] "DSigDB"                                           
#&gt;  [24] "Data_Acquisition_Method_Most_Popular_Genes"       
#&gt;  [25] "Disease_Perturbations_from_GEO_down"              
#&gt;  [26] "Disease_Perturbations_from_GEO_up"                
#&gt;  [27] "Disease_Signatures_from_GEO_down_2014"            
#&gt;  [28] "Disease_Signatures_from_GEO_up_2014"              
#&gt;  [29] "DrugMatrix"                                       
#&gt;  [30] "Drug_Perturbations_from_GEO_2014"                 
#&gt;  [31] "Drug_Perturbations_from_GEO_down"                 
#&gt;  [32] "Drug_Perturbations_from_GEO_up"                   
#&gt;  [33] "ENCODE_Histone_Modifications_2013"                
#&gt;  [34] "ENCODE_Histone_Modifications_2015"                
#&gt;  [35] "ENCODE_TF_ChIP-seq_2014"                          
#&gt;  [36] "ENCODE_TF_ChIP-seq_2015"                          
#&gt;  [37] "ENCODE_and_ChEA_Consensus_TFs_from_ChIP-X"        
#&gt;  [38] "ESCAPE"                                           
#&gt;  [39] "Enrichr_Libraries_Most_Popular_Genes"             
#&gt;  [40] "Enrichr_Submissions_TF-Gene_Coocurrence"          
#&gt;  [41] "Epigenomics_Roadmap_HM_ChIP-seq"                  
#&gt;  [42] "GO_Biological_Process_2013"                       
#&gt;  [43] "GO_Biological_Process_2015"                       
#&gt;  [44] "GO_Biological_Process_2017"                       
#&gt;  [45] "GO_Biological_Process_2017b"                      
#&gt;  [46] "GO_Biological_Process_2018"                       
#&gt;  [47] "GO_Cellular_Component_2013"                       
#&gt;  [48] "GO_Cellular_Component_2015"                       
#&gt;  [49] "GO_Cellular_Component_2017"                       
#&gt;  [50] "GO_Cellular_Component_2017b"                      
#&gt;  [51] "GO_Cellular_Component_2018"                       
#&gt;  [52] "GO_Molecular_Function_2013"                       
#&gt;  [53] "GO_Molecular_Function_2015"                       
#&gt;  [54] "GO_Molecular_Function_2017"                       
#&gt;  [55] "GO_Molecular_Function_2017b"                      
#&gt;  [56] "GO_Molecular_Function_2018"                       
#&gt;  [57] "GTEx_Tissue_Sample_Gene_Expression_Profiles_down" 
#&gt;  [58] "GTEx_Tissue_Sample_Gene_Expression_Profiles_up"   
#&gt;  [59] "GWAS_Catalog_2019"                                
#&gt;  [60] "GeneSigDB"                                        
#&gt;  [61] "Genes_Associated_with_NIH_Grants"                 
#&gt;  [62] "Genome_Browser_PWMs"                              
#&gt;  [63] "HMDB_Metabolites"                                 
#&gt;  [64] "HomoloGene"                                       
#&gt;  [65] "HumanCyc_2015"                                    
#&gt;  [66] "HumanCyc_2016"                                    
#&gt;  [67] "Human_Gene_Atlas"                                 
#&gt;  [68] "Human_Phenotype_Ontology"                         
#&gt;  [69] "Jensen_COMPARTMENTS"                              
#&gt;  [70] "Jensen_DISEASES"                                  
#&gt;  [71] "Jensen_TISSUES"                                   
#&gt;  [72] "KEA_2013"                                         
#&gt;  [73] "KEA_2015"                                         
#&gt;  [74] "KEGG_2013"                                        
#&gt;  [75] "KEGG_2015"                                        
#&gt;  [76] "KEGG_2016"                                        
#&gt;  [77] "Kinase_Perturbations_from_GEO_down"               
#&gt;  [78] "Kinase_Perturbations_from_GEO_up"                 
#&gt;  [79] "LINCS_L1000_Chem_Pert_down"                       
#&gt;  [80] "LINCS_L1000_Chem_Pert_up"                         
#&gt;  [81] "LINCS_L1000_Kinase_Perturbations_down"            
#&gt;  [82] "LINCS_L1000_Kinase_Perturbations_up"              
#&gt;  [83] "LINCS_L1000_Ligand_Perturbations_down"            
#&gt;  [84] "LINCS_L1000_Ligand_Perturbations_up"              
#&gt;  [85] "Ligand_Perturbations_from_GEO_down"               
#&gt;  [86] "Ligand_Perturbations_from_GEO_up"                 
#&gt;  [87] "MCF7_Perturbations_from_GEO_down"                 
#&gt;  [88] "MCF7_Perturbations_from_GEO_up"                   
#&gt;  [89] "MGI_Mammalian_Phenotype_2013"                     
#&gt;  [90] "MGI_Mammalian_Phenotype_2017"                     
#&gt;  [91] "MGI_Mammalian_Phenotype_Level_3"                  
#&gt;  [92] "MGI_Mammalian_Phenotype_Level_4"                  
#&gt;  [93] "MSigDB_Computational"                             
#&gt;  [94] "MSigDB_Oncogenic_Signatures"                      
#&gt;  [95] "Microbe_Perturbations_from_GEO_down"              
#&gt;  [96] "Microbe_Perturbations_from_GEO_up"                
#&gt;  [97] "Mouse_Gene_Atlas"                                 
#&gt;  [98] "NCI-60_Cancer_Cell_Lines"                         
#&gt;  [99] "NCI-Nature_2015"                                  
#&gt; [100] "NCI-Nature_2016"                                  
#&gt; [101] "NIH_Funded_PIs_2017_AutoRIF_ARCHS4_Predictions"   
#&gt; [102] "NIH_Funded_PIs_2017_GeneRIF_ARCHS4_Predictions"   
#&gt; [103] "NIH_Funded_PIs_2017_Human_AutoRIF"                
#&gt; [104] "NIH_Funded_PIs_2017_Human_GeneRIF"                
#&gt; [105] "NURSA_Human_Endogenous_Complexome"                
#&gt; [106] "OMIM_Disease"                                     
#&gt; [107] "OMIM_Expanded"                                    
#&gt; [108] "Old_CMAP_down"                                    
#&gt; [109] "Old_CMAP_up"                                      
#&gt; [110] "PPI_Hub_Proteins"                                 
#&gt; [111] "Panther_2015"                                     
#&gt; [112] "Panther_2016"                                     
#&gt; [113] "Pfam_InterPro_Domains"                            
#&gt; [114] "Phosphatase_Substrates_from_DEPOD"                
#&gt; [115] "RNA-Seq_Disease_Gene_and_Drug_Signatures_from_GEO"
#&gt; [116] "Rare_Diseases_AutoRIF_ARCHS4_Predictions"         
#&gt; [117] "Rare_Diseases_AutoRIF_Gene_Lists"                 
#&gt; [118] "Rare_Diseases_GeneRIF_ARCHS4_Predictions"         
#&gt; [119] "Rare_Diseases_GeneRIF_Gene_Lists"                 
#&gt; [120] "Reactome_2013"                                    
#&gt; [121] "Reactome_2015"                                    
#&gt; [122] "Reactome_2016"                                    
#&gt; [123] "SILAC_Phosphoproteomics"                          
#&gt; [124] "Single_Gene_Perturbations_from_GEO_down"          
#&gt; [125] "Single_Gene_Perturbations_from_GEO_up"            
#&gt; [126] "SubCell_BarCode"                                  
#&gt; [127] "SysMyo_Muscle_Gene_Sets"                          
#&gt; [128] "TF-LOF_Expression_from_GEO"                       
#&gt; [129] "TF_Perturbations_Followed_by_Expression"          
#&gt; [130] "TRANSFAC_and_JASPAR_PWMs"                         
#&gt; [131] "TRRUST_Transcription_Factors_2019"                
#&gt; [132] "TargetScan_microRNA"                              
#&gt; [133] "TargetScan_microRNA_2017"                         
#&gt; [134] "Tissue_Protein_Expression_from_Human_Proteome_Map"
#&gt; [135] "Tissue_Protein_Expression_from_ProteomicsDB"      
#&gt; [136] "Transcription_Factor_PPIs"                        
#&gt; [137] "VirusMINT"                                        
#&gt; [138] "Virus_Perturbations_from_GEO_down"                
#&gt; [139] "Virus_Perturbations_from_GEO_up"                  
#&gt; [140] "WikiPathways_2013"                                
#&gt; [141] "WikiPathways_2015"                                
#&gt; [142] "WikiPathways_2016"                                
#&gt; [143] "WikiPathways_2019_Human"                          
#&gt; [144] "WikiPathways_2019_Mouse"                          
#&gt; [145] "dbGaP"                                            
#&gt; [146] "huMAP"                                            
#&gt; [147] "miRTarBase_2017"                                  </div><div class='output co'>#&gt;   [1] "ARCHS4_Cell-lines"                                
#&gt;   [2] "ARCHS4_IDG_Coexp"                                 
#&gt;   [3] "ARCHS4_Kinases_Coexp"                             
#&gt;   [4] "ARCHS4_TFs_Coexp"                                 
#&gt;   [5] "ARCHS4_Tissues"                                   
#&gt;   [6] "Achilles_fitness_decrease"                        
#&gt;   [7] "Achilles_fitness_increase"                        
#&gt;   [8] "Aging_Perturbations_from_GEO_down"                
#&gt;   [9] "Aging_Perturbations_from_GEO_up"                  
#&gt;  [10] "Allen_Brain_Atlas_down"                           
#&gt;  [11] "Allen_Brain_Atlas_up"                             
#&gt;  [12] "BioCarta_2013"                                    
#&gt;  [13] "BioCarta_2015"                                    
#&gt;  [14] "BioCarta_2016"                                    
#&gt;  [15] "BioPlex_2017"                                     
#&gt;  [16] "CORUM"                                            
#&gt;  [17] "Cancer_Cell_Line_Encyclopedia"                    
#&gt;  [18] "ChEA_2013"                                        
#&gt;  [19] "ChEA_2015"                                        
#&gt;  [20] "ChEA_2016"                                        
#&gt;  [21] "Chromosome_Location"                              
#&gt;  [22] "Chromosome_Location_hg19"                         
#&gt;  [23] "DSigDB"                                           
#&gt;  [24] "Data_Acquisition_Method_Most_Popular_Genes"       
#&gt;  [25] "Disease_Perturbations_from_GEO_down"              
#&gt;  [26] "Disease_Perturbations_from_GEO_up"                
#&gt;  [27] "Disease_Signatures_from_GEO_down_2014"            
#&gt;  [28] "Disease_Signatures_from_GEO_up_2014"              
#&gt;  [29] "DrugMatrix"                                       
#&gt;  [30] "Drug_Perturbations_from_GEO_2014"                 
#&gt;  [31] "Drug_Perturbations_from_GEO_down"                 
#&gt;  [32] "Drug_Perturbations_from_GEO_up"                   
#&gt;  [33] "ENCODE_Histone_Modifications_2013"                
#&gt;  [34] "ENCODE_Histone_Modifications_2015"                
#&gt;  [35] "ENCODE_TF_ChIP-seq_2014"                          
#&gt;  [36] "ENCODE_TF_ChIP-seq_2015"                          
#&gt;  [37] "ENCODE_and_ChEA_Consensus_TFs_from_ChIP-X"        
#&gt;  [38] "ESCAPE"                                           
#&gt;  [39] "Enrichr_Libraries_Most_Popular_Genes"             
#&gt;  [40] "Enrichr_Submissions_TF-Gene_Coocurrence"          
#&gt;  [41] "Epigenomics_Roadmap_HM_ChIP-seq"                  
#&gt;  [42] "GO_Biological_Process_2013"                       
#&gt;  [43] "GO_Biological_Process_2015"                       
#&gt;  [44] "GO_Biological_Process_2017"                       
#&gt;  [45] "GO_Biological_Process_2017b"                      
#&gt;  [46] "GO_Biological_Process_2018"                       
#&gt;  [47] "GO_Cellular_Component_2013"                       
#&gt;  [48] "GO_Cellular_Component_2015"                       
#&gt;  [49] "GO_Cellular_Component_2017"                       
#&gt;  [50] "GO_Cellular_Component_2017b"                      
#&gt;  [51] "GO_Cellular_Component_2018"                       
#&gt;  [52] "GO_Molecular_Function_2013"                       
#&gt;  [53] "GO_Molecular_Function_2015"                       
#&gt;  [54] "GO_Molecular_Function_2017"                       
#&gt;  [55] "GO_Molecular_Function_2017b"                      
#&gt;  [56] "GO_Molecular_Function_2018"                       
#&gt;  [57] "GTEx_Tissue_Sample_Gene_Expression_Profiles_down" 
#&gt;  [58] "GTEx_Tissue_Sample_Gene_Expression_Profiles_up"   
#&gt;  [59] "GWAS_Catalog_2019"                                
#&gt;  [60] "GeneSigDB"                                        
#&gt;  [61] "Genes_Associated_with_NIH_Grants"                 
#&gt;  [62] "Genome_Browser_PWMs"                              
#&gt;  [63] "HMDB_Metabolites"                                 
#&gt;  [64] "HomoloGene"                                       
#&gt;  [65] "HumanCyc_2015"                                    
#&gt;  [66] "HumanCyc_2016"                                    
#&gt;  [67] "Human_Gene_Atlas"                                 
#&gt;  [68] "Human_Phenotype_Ontology"                         
#&gt;  [69] "Jensen_COMPARTMENTS"                              
#&gt;  [70] "Jensen_DISEASES"                                  
#&gt;  [71] "Jensen_TISSUES"                                   
#&gt;  [72] "KEA_2013"                                         
#&gt;  [73] "KEA_2015"                                         
#&gt;  [74] "KEGG_2013"                                        
#&gt;  [75] "KEGG_2015"                                        
#&gt;  [76] "KEGG_2016"                                        
#&gt;  [77] "Kinase_Perturbations_from_GEO_down"               
#&gt;  [78] "Kinase_Perturbations_from_GEO_up"                 
#&gt;  [79] "LINCS_L1000_Chem_Pert_down"                       
#&gt;  [80] "LINCS_L1000_Chem_Pert_up"                         
#&gt;  [81] "LINCS_L1000_Kinase_Perturbations_down"            
#&gt;  [82] "LINCS_L1000_Kinase_Perturbations_up"              
#&gt;  [83] "LINCS_L1000_Ligand_Perturbations_down"            
#&gt;  [84] "LINCS_L1000_Ligand_Perturbations_up"              
#&gt;  [85] "Ligand_Perturbations_from_GEO_down"               
#&gt;  [86] "Ligand_Perturbations_from_GEO_up"                 
#&gt;  [87] "MCF7_Perturbations_from_GEO_down"                 
#&gt;  [88] "MCF7_Perturbations_from_GEO_up"                   
#&gt;  [89] "MGI_Mammalian_Phenotype_2013"                     
#&gt;  [90] "MGI_Mammalian_Phenotype_2017"                     
#&gt;  [91] "MGI_Mammalian_Phenotype_Level_3"                  
#&gt;  [92] "MGI_Mammalian_Phenotype_Level_4"                  
#&gt;  [93] "MSigDB_Computational"                             
#&gt;  [94] "MSigDB_Oncogenic_Signatures"                      
#&gt;  [95] "Microbe_Perturbations_from_GEO_down"              
#&gt;  [96] "Microbe_Perturbations_from_GEO_up"                
#&gt;  [97] "Mouse_Gene_Atlas"                                 
#&gt;  [98] "NCI-60_Cancer_Cell_Lines"                         
#&gt;  [99] "NCI-Nature_2015"                                  
#&gt; [100] "NCI-Nature_2016"                                  
#&gt; [101] "NIH_Funded_PIs_2017_AutoRIF_ARCHS4_Predictions"   
#&gt; [102] "NIH_Funded_PIs_2017_GeneRIF_ARCHS4_Predictions"   
#&gt; [103] "NIH_Funded_PIs_2017_Human_AutoRIF"                
#&gt; [104] "NIH_Funded_PIs_2017_Human_GeneRIF"                
#&gt; [105] "NURSA_Human_Endogenous_Complexome"                
#&gt; [106] "OMIM_Disease"                                     
#&gt; [107] "OMIM_Expanded"                                    
#&gt; [108] "Old_CMAP_down"                                    
#&gt; [109] "Old_CMAP_up"                                      
#&gt; [110] "PPI_Hub_Proteins"                                 
#&gt; [111] "Panther_2015"                                     
#&gt; [112] "Panther_2016"                                     
#&gt; [113] "Pfam_InterPro_Domains"                            
#&gt; [114] "Phosphatase_Substrates_from_DEPOD"                
#&gt; [115] "RNA-Seq_Disease_Gene_and_Drug_Signatures_from_GEO"
#&gt; [116] "Rare_Diseases_AutoRIF_ARCHS4_Predictions"         
#&gt; [117] "Rare_Diseases_AutoRIF_Gene_Lists"                 
#&gt; [118] "Rare_Diseases_GeneRIF_ARCHS4_Predictions"         
#&gt; [119] "Rare_Diseases_GeneRIF_Gene_Lists"                 
#&gt; [120] "Reactome_2013"                                    
#&gt; [121] "Reactome_2015"                                    
#&gt; [122] "Reactome_2016"                                    
#&gt; [123] "SILAC_Phosphoproteomics"                          
#&gt; [124] "Single_Gene_Perturbations_from_GEO_down"          
#&gt; [125] "Single_Gene_Perturbations_from_GEO_up"            
#&gt; [126] "SubCell_BarCode"                                  
#&gt; [127] "SysMyo_Muscle_Gene_Sets"                          
#&gt; [128] "TF-LOF_Expression_from_GEO"                       
#&gt; [129] "TF_Perturbations_Followed_by_Expression"          
#&gt; [130] "TRANSFAC_and_JASPAR_PWMs"                         
#&gt; [131] "TRRUST_Transcription_Factors_2019"                
#&gt; [132] "TargetScan_microRNA"                              
#&gt; [133] "TargetScan_microRNA_2017"                         
#&gt; [134] "Tissue_Protein_Expression_from_Human_Proteome_Map"
#&gt; [135] "Tissue_Protein_Expression_from_ProteomicsDB"      
#&gt; [136] "Transcription_Factor_PPIs"                        
#&gt; [137] "VirusMINT"                                        
#&gt; [138] "Virus_Perturbations_from_GEO_down"                
#&gt; [139] "Virus_Perturbations_from_GEO_up"                  
#&gt; [140] "WikiPathways_2013"                                
#&gt; [141] "WikiPathways_2015"                                
#&gt; [142] "WikiPathways_2016"                                
#&gt; [143] "WikiPathways_2019_Human"                          
#&gt; [144] "WikiPathways_2019_Mouse"                          
#&gt; [145] "dbGaP"                                            
#&gt; [146] "huMAP"                                            
#&gt; [147] "miRTarBase_2017"                                  </div><div class='input'>
</div></pre>
  </div>
  <div class="col-md-3 hidden-xs hidden-sm" id="sidebar">
    <h2>Contents</h2>
    <ul class="nav nav-pills nav-stacked">
      <li><a href="#arguments">Arguments</a></li>
      
      <li><a href="#value">Value</a></li>
      
      <li><a href="#examples">Examples</a></li>
    </ul>

  </div>
</div>

      <footer>
      <div class="copyright">
  <p>Developed by <a href='https://github.com/anfederico'>Anthony Federico</a>, <a href='https://github.com/montilab'>Stefano Monti</a>.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.3.0.</p>
</div>
      </footer>
   </div>

  

  </body>
</html>

